## SH3BP4 promotes neuropilin-1 and a 5-integrin endocytosis and is inhibited by Akt

## Graphical Abstract

<!-- image -->

## Highlights

- d SH3BP4 promotes neuropilin-1 and a 5-integrin endocytosis and semaphorin-3a signaling
- d Semaphorin-3a activates SH3BP4 via PTEN to promote cell contraction by endocytosis
- d AKT phosphorylates SH3BP4 at S246, excluding SH3BP4 from clathrin-coated pits
- d 14-3-3 binding to S246 blocks SH3BP4 complex with Eps15, GIPC1 to curb endocytosis

<!-- image -->

## Authors

Christoph J. Burckhardt, John D. Minna, Gaudenz Danuser

## Correspondence

christoph.burckhardt@gmail.com (C.J.B.), gaudenz.danuser@ utsouthwestern.edu (G.D.)

## In brief

SH3BP4 is an endocytic adaptor. Burckhardt et al. report that SH3BP4 localization to clathrin-coated pits is regulated by Akt. Operating at the intersection of signaling and membrane trafficking, SH3BP4 mediates both internal and external cues to regulate the endocytosis of neuropilin and a 5-integrin receptors, thus modulating cell-matrix interactions.

<!-- image -->

<!-- image -->

## Article

## SH3BP4 promotes neuropilin-1 and a 5-integrin endocytosis and is inhibited by Akt

Christoph J. Burckhardt, 1,2, * John D. Minna, 3 and Gaudenz Danuser 1,2,4, *

1 Lyda Hill Department of Bioinformatics, UT Southwestern Medical Center, Dallas, TX 75390, USA

2 Department of Cell Biology, UT Southwestern Medical Center, Dallas, TX, USA

3 Hamon Center for Therapeutic Oncology Research, Simmons Comprehensive Cancer Center, Departments of Internal Medicine and

Pharmacology, University of Texas Southwestern Medical Center, Dallas, TX 75390-8593, USA

4 Lead contact

*Correspondence: christoph.burckhardt@gmail.com (C.J.B.), gaudenz.danuser@utsouthwestern.edu (G.D.) https://doi.org/10.1016/j.devcel.2021.03.009

## SUMMARY

Cells probe their surrounding matrix for attachment sites via integrins that are internalized by endocytosis. We find that SH3BP4 regulates integrin surface expression in a signaling-dependent manner via clathrincoated pits (CCPs). Dephosphorylated SH3BP4 at S246 is efficiently recruited to CCPs, while upon Akt phosphorylation, SH3BP4 is sequestered by 14-3-3 adaptors and excluded from CCPs. In the absence of Akt activity, SH3BP4 binds GIPC1 and targets neuropilin-1 and a 5/ b 1-integrin for endocytosis, leading to inhibition of cell spreading. Similarly, chemorepellent semaphorin-3a binds neuropilin-1 to activate PTEN, which antagonizes Akt and thus recruits SH3BP4 to CCPs to internalize both receptors and induce cell contraction. In PTEN mutant non-small cell lung cancer cells with high Akt activity, expression of non-phosphorylatable active SH3BP4-S246A restores semaphorin-3a induced cell contraction. Thus, SH3BP4 links Akt signaling to endocytosis of NRP1 and a 5/ b 1-integrins to modulate cell-matrix interactions in response to intrinsic and extrinsic cues.

## INTRODUCTION

SH3BP4(SH3binding protein 4, also known as transferrin receptor trafficking protein, TTP) was cloned from human corneal fibroblasts and was found to be ubiquitously expressed (Dunlevy et al., 1999). Subsequently, SH3BP4 was associated with cellular compartments such as the plasma membrane and the nucleus (Khanobdee et al., 2004), lysosomes (Kim et al., 2012), filopodia (O'Loughlin et al., 2018), the perinuclear region (Antas et al., 2019), endosomes (Francavilla et al., 2013) and clathrincoated pits (CCPs) (Tosoni et al., 2005), indicating that localization and function of SH3BP4 are still diffuse. The N-terminal portion of SH3BP4 contains binding motifs for clathrin, dynamin-2, and Eps15 and was shown to bind these proteins (Tosoni et al., 2005). This has suggested that SH3BP4 might be implicated in clathrin-mediated endocytosis (CME).

CME is the primary process for macromolecular uptake in most cells (Conner and Schmid, 2003). AP-2 adaptor complexes and clathrin triskelia assemble a protein coat at the plasma membrane, together with more than 30 accessory factors (Schmid and McMahon, 2007). Transmembrane receptors are sorted into CCPs via AP-2 and other adaptors (Traub, 2009). CME accessory factors regulate AP-2 complexes, generate and sense membrane curvature, stabilize or destabilize the clathrin coat, and orchestrate the recruitment of cargo and the GTPase dynamin (Mettlen et al., 2018). A multitude of mostly weak and transient protein-protein and protein-phospholipid interactions keep coats adaptive to changing membrane curvatures (McMahon and Boucrot, 2011). At the late stages, the mechanochemical GTPase dynamin assembles around the membrane neck for membrane fission, releasing the coated vesicle (Antonny et al., 2016). Rapid phosphoinositide conversion triggers vesicle uncoating (He et al., 2017) and the vesicle and cargo enter endosomal sorting (Zerial and McBride, 2001).

Among the many cargos internalized via CME are integrin cellmatrix attachment receptors (Moreno-Layseca et al., 2019). Integrins form heterodimeric complexes composed of a and b chains that exist in ''closed,'' inactive and active, and ''open'' conformations to signal in a bidirectional way across the plasma membrane(Hynes,2002). Integrins connect the extracellular matrix (ECM) via adhesions to the actomyosin cytoskeleton (Schwartz,2010).Talin andkindlin inducethe high-affinity integrin conformation that promotes ECM binding, referred to as insideout signaling (Sun et al., 2019). Reciprocally, the abundance of integrin binding sites on ECM proteins leads to the formation of adhesion complexes, known as outside-in signaling (Kim et al., 2003).

While integrin activation and assembly of adhesion complexes are well studied (Vicente-Manzanares et al., 2009), it is less clear how integrins are inactivated. Protein complexes capturing the cytoplasmic portions of integrins are inducing the inactive conformation through inside-out-signaling (Bouvard et al.,

<!-- image -->

Article

Figure 1. The SH3BP4 mutant S246A is targeted to clathrin-coated pits (CCPs)

<!-- image -->

(A) Western blot (WB) analysis of RPE SH3BP4 KO and SH3BP4-GFP reconstitution cells.

(B) Total internal reflection fluorescence (TIRF) microscopy of GFP fusions at CCPs. TdTomato-clathrin light chain a (Clca) was used as a CCP marker. Cytosolic GFP, cGFP.

- (C) Fluorescence intensity in the 40-60 s CCP cohort (N = 3 experiments, n = 40 cells per condition) for SH3BP4-GFP and for negative controls soluble GFP, membrane GFP, and positive controls Eps15-GFP, Dyn2-GFP. The tdTomato-Clca cohort from SH3BP4-GFP for comparison.
- (D) Percentage of GFP positive CCPs, from C, rank-sum tests of SH3BP4 versus all other constructs. Membrane-targeted GFP, mGFP.

<!-- image -->

## ll

2013). In addition, endocytosis is thought to play a major role in integrin homeostasis via the reduction of integrin exposure at the cell surface (Caswell et al., 2009). However, only a few interactions of bona fide CME accessory factors with integrins are established. A subset of a -integrin cytoplasmic tails interact with the m 2 subunit of the AP-2 complex (De Franceschi et al., 2016). The b -integrin tails bind through their NPxY motifs with protein-tyrosine binding domains in Dab2 (Teckchandani et al., 2009; Prunier and Howe, 2005) and Numb (Calderwood et al., 2003; Nishimura and Kaibuchi, 2007). Additionally, a number of non-CME proteins were reported to bind integrin tails (Caswell et al., 2009). Among them is GIPC1, a PDZ domain containing protein that targets neuropilin-1 (NRP1) and a 5 and a 6-integrins for endocytosis (Valdembri et al., 2009; Tani and Mercurio, 2001). However, it is not understood how GIPC1 connects with the CCP machinery. As with integrin activation, integrin endocytosis can also be triggered from outside e.g., by chemorepellents such as semaphorin-3a (Serini et al., 2003; Gioelli et al., 2018). The semaphorin ligands bind NRP1 receptors, which is thought to reduce Akt activity (Chadborn et al., 2006). Again, how these various signals impinge on the dynamics of endocytic integrin trafficking has remained elusive.

Here, we establish SH3BP4 as a central link between such signals and integrin endocytosis. We show that dephosphorylated SH3BP4 is efficiently accumulating at CCPs and enhancing NRP1 and a 5-integrin endocytosis, thereby reducing cell adhesion. We also show that SH3BP4 endocytic activity is inhibited by Akt phosphorylation, and we establish the mechanism of both activation and inactivation of SH3BP4. We propose that under conditions of reduced Akt signaling, cell attachment is weakened by an SH3BP4-dependent endocytic process whereby integrins are internalized. We show how the Akt-regulated SH3BP4tiesNRP1stimulation by semaphorin ligands to the control of integrin endocytosis. Together, these data resolve numerous open questions regarding the integration of cellintrinsic and extrinsic signals and adhesion regulation.

## RESULTS

## SH3BP4 mutant S246A is targeted to CCPs

We chose to study the subcellular localization of the protein product of SH3BP4 in human hTERT immortalized retinal pigment epithelium (RPE) cells (Jiang et al., 1999), since SH3BP4 expression was reported to be high in ocular tissues (Dunlevy et al., 1999). We used CRISPR/Cas9 to generate SH3BP4 knockout cells and expressed the SH3BP4-GFP fusion

Article by lentiviral transduction (Figure 1A). We applied live cell total internal reflection fluorescence (TIRF) microscopy to quantify the accumulation of SH3BP4-GFP at CCPs that were labeled by tdTomato-clathrin light chain a (Clca) (Figure 1B). The tdTomato-Clca signals were detected and tracked as previously described (Aguet et al., 2013) (Figures S1A-S1E) and the SH3BP4-GFP signal at CCPs was followed over time (Figures 1CandS1F). For comparison, cytosolic and membrane-targeted GFPwererecorded (Figure S1G). SH3BP4-GFP did not accumulate at CCPs (Figures 1B-1D and S1H-S1I), contrary to the bona fide CME accessory factors Eps15 and dynamin-2 (Dyn2) (Figure 1C) (Taylor et al., 2011). For both Eps15 and Dyn2, more CCPs were GFP positive (Figure 1D), had higher mean GFP intensity (Figure S1H), and showed more continuously positive CCPs than SH3BP4 (Figure S1I). This opened the possibility that the SH3BP4 presence at CCPs was regulated by additional mechanisms.

Indeed, by mass spectrometry, we found multiple phosphorylation sites, with serine-246 (S246) as the most phosphorylated residue (Figures 1E, S1J, and S1K; Table S1). To test how the phosphorylation status affected targeting of SH3BP4-GFP to CCPs, we mutated some of the most frequently observed phosphorylation sites (Table S1) to non-phosphorylatable alanine (A) or phosphomimic aspartic acid (D) residues and quantified SH3BP4-GFP accumulation at CCPs (Figures 1F and S1L). Most prominently, S246A and S246D exhibited enhanced colocalization with CCPs. This site was previously reported to be susceptible to Akt phosphorylation (Chiba et al., 2009; Reinartz et al., 2014) and fits an RxKRxxS Akt phosphorylation motif (Figure S1M; Table S2) (Manning and Cantley, 2007). Moreover, reports suggest that for Akt phosphorylation sites, aspartic and glutamic acid residues do not function as phosphomimic toward 14-3-3 adaptor binding (Yip-Schneider et al., 2000) (examples in Table S3; Figure 2A), explaining why both mutants exhibited the same phenotype. Next, we asked to what extent the recruitment of non-phosphorylatable SH3BP4-GFP S246A was dependent on CME peptide motifs (Tosoni et al., 2005). Specifically, we mutated the clathrin binding motif (LIDL), the Dyn2 specific SH3domain, and the Eps15-homology domain binding NPF motifs (SH3BP4-delta-CME, dCME). By reconstitution of the S246A/dCME mutants in the RPE SH3BP4 KO cells, we confirmed strong colocalization of the S246A construct with CCPs but reduced colocalization by S246A/dCME (Figures 1G and 1H). This suggested a contribution of the CME motifs to CCP targeting. Intensity profiles of individual CCPs indicated that the SH3BP4-GFP S246A localized to CCPs along their

(E) Number of SH3BP4 phospho-peptides by mass spectrometry (N = 4 experiments), for pS131 (triangles) and pS246 (squares), mean ± SEM are shown. Bottom: SH3BP4 domain architecture (Kokoszy /C19 nska et al., 2009).

(F) TIRF microscopy of SH3BP4 phospho-mutants to alanine (A) or aspartic acid (D), percentage of SH3BP4-GFP-positive CCPs (N = 3 experiments and n = 20 cells each).

(G) WB of SH3BP4 KO cells reconstituted with SH3BP4-GFP wt, S246A, and dCME/S246A mutants.

(H) TIRF microscopy of SH3BP4-GFP wt, S246A, and dCME/S246A mutants.

(I) Two example tracks of SH3BP4-GFP S246A at CCPs.

(J) Percentage of continuously SH3BP4-GFP positive CCPs for wt (N = 3 experiments with n = 62 cells in total), S246A (n = 60) and S246A/dCME (n = 59) constructs.

(K) 40-60 s cohorts for SH3BP4-GFP wt and mutants.

(L) Mean SH3BP4-GFP intensity at CCPs. Intensities in arbitrary units (AU., for C, I, K, and L). For intensity cohorts the mean ± SEM are shown (C and K). Scale bars, 5 m m (B and H). For statistics, rank-sum tests were used and data are presented as box and whisker plots (D, F, J, and L), ns, not significant, *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, ****p &lt; 0.0001.

Article

Figure 2. SH3BP4 targeting to CCPs is blocked by 14-3-3

<!-- image -->

(A) Immunoprecipitation (IP) of SH3BP4-GFP wt and S246A/D mutants for endogenous 14-3-3 ε (N = 6 experiments, representative data).

- (B) IP of SH3BP4-GFP wt and R18 mutants for endogenous 14-3-3 ε (N = 4).
- (C) Total internal reflection fluorescence microscopy of SH3BP4-GFP wt and R18 mutants co-expressed with tdTomata-Clca. Scale bar 5 m m.
- (D) Percentage of SH3BP4-GFP-positive CCPs for SH3BP4-GFP wt and R18 mutants (N = 3, n = 42 cells each). For statistical analysis, rank-sum test was used, data presented as box and whisker plots, ns, not significant, **p &lt; 0.01, ****p &lt; 0.0001.
- (E) IP of SH3BP4-GFP for endogenous 14-3-3 ε with Akt1-wt-HA overexpression (OX) (N = 5).
- (F) IP of 14-3-3 b -GFP for HA-SH3BP4 with Akt1-wt-HA OX (N = 5). For the IP experiments, representative data are shown and statistics of IP data are in Figure S2.

lifetime (Figure 1I). The S246A mutant also showed more continuously positive CCPs (Figure 1J) and higher mean SH3BP4-GFP intensity at CCPs (Figures 1K, 1L, and S1N). The efficient localization of S246A to CCPs was also apparent in a different RPE cell line and in a human microvascular endothelial cell line (Figures S1OandS1T), implying that this observation can be generalized. In summary, these data suggest that the phosphorylation status at S246 regulates the incorporation of SH3BP4 into CCPs.

## SH3BP4 targeting to CCPs is blocked by 14-3-3

Our data indicated that phosphorylated SH3BP4 was blocked from CCP accumulation, so we sought to elucidate the mechanism of SH3BP4 inhibition. It was previously reported that SH3BP4 binds 14-3-3 adaptor proteins via phosphorylated S246 (Chiba et al., 2009) and the amino acid sequence

RSKRSYpSL around pS246 overlaps with 14-3-3 consensus binding motifs (Yaffe et al., 1997) where the phospho-group is captured by 14-3-3 (Muslin et al., 1996). Indeed, all isoforms of 14-3-3 bound SH3BP4 (Figures S2A and S2B). To confirm the interaction of SH3BP4 with the 14-3-3 ε isoform, we overexpressed SH3BP4-GFP and probed for binding of endogenous 14-3-3 ε (Figures 2A, S2C, and S2D). Consistent with previous findings, the non-phosphorylatable S246A mutant did not bind 14-3-3 ε (Chiba et al., 2009). To further study the putative loss of function caused by the interaction with 14-3-3, we introduced into SH3BP4 a 20-amino-acid peptide R18, which constitutively binds 14-3-3 (Wang et al., 1999) (Table S4). The R18 peptide was introduced upstream of S246 where it replaced a stretch of less conservedaminoacidsinSH3BP4(FigureS2E).Asexpected, SH3BP4-GFP R18 pulled down endogenous 14-3-3 ε efficiently,

<!-- image -->

<!-- image -->

while the mutant R18-KK peptide did not (Figures 2B and S2F). The S246A/R18 construct with only one 14-3-3 binding site, showed reduced 14-3-3 ε binding in accordance with a model where multiple 14-3-3 binding sites enhanced the interaction with the dimeric 14-3-3 adaptors (Yaffe, 2002). This notion was further supported with an SH3BP4 construct containing two R18 sites that more strongly bound 14-3-3 ε (Figures S2G and S2H).

We then tested the effect of R18-mediated 14-3-3 binding on SH3BP4 targeting to CCPs. The S246/R18 construct, where 143-3 proteins could bind both a phosphorylated S246 and the R18 peptide, was efficiently excluded form CCPs (Figures 2C, 2D, and S2I). The construct S246A/R18 where 14-3-3 only bound once at the peptide sequence was found at CCPs as frequently as the wt protein, and the S246A/R18-KK mutant was recruited to CCPsasefficientlyastheS246Amutant,consistentwithabolished 14-3-3 binding. These data suggested that 14-3-3 binding to the SH3BP4pS246sitestronglyinhibitedtheincorporationintoCCPs.

Previous studies had identified Akt as the kinase responsible for phosphorylation of S246 (Chiba et al., 2009; Reinartz et al., 2014). Indeed, Akt1 overexpression enhanced 14-3-3 ε binding to SH3BP4-GFP (Figures 2E and S2J). Conversely, the 14-33 b -GFP fusion protein showed enhanced SH3BP4 binding under Akt overexpression (Figures 2F and S2K). Together, these data support a model in which SH3BP4 phosphorylation by Akt at S246 promotes 14-3-3 binding, which sequesters SH3BP4 preventing its recruitment to CCPs. This establishes SH3BP4 as an Akt substrate in CME.

## 14-3-3 interferes with SH3BP4 binding to Eps15

Next, we explored the nature of SH3BP4 targeting to CCPs and the mechanism of inhibition by 14-3-3. When combined with the S246A activating mutation, all of SH3BP4's single CME motif mutants reduced CCP targeting compared with the S246A mutation alone (Figures 3A and S3A). This suggested that for efficient incorporation of SH3BP4 into the CCP structure, concerted interactions with clathrin heavy chain (CHC), Eps15, and Dyn2 were needed. In vitro , Eps15 bound most SH3BP4, followed by Dyn2 and CHC (Figures 3B and S3B). The interaction of Eps15 and Dyn2 with SH3BP4 was reduced by the dCME mutation and binding was comparable with interactions with well-established CME partners of Eps15 and Dyn2 (Figures S3C-S3J), as reported before (Tosoni et al., 2005; Salcini et al., 1997). Comparison of SH3BP4-GFP S246A recruitment (Figure 1K) with profiles of Eps15 and Dyn2 (Figure 1C) suggested a better match with Eps15 (Mettlen et al., 2018; Taylor et al., 2011). Moreover, in presence of 14-3-3, the Eps15-SH3BP4 interaction was weakened (Figures 3C and S3K). Since the most proximal Eps15- homology domain interaction motif (NPF) is only eight amino acids upstream of S246, an interference by 14-3-3 is conceivable (Figure S2E). Consistent with this notion and our model of SH3BP4 as a substrate for Akt, overexpression of Akt reduced Eps15 binding to SH3BP4 (Figures 3D and S3L). We further tested if SH3BP4-GFP wt and S246A positive CCPs were simultaneously positive for Eps15. Indeed, some CCPs showed a synchronized accumulation of both SH3BP4-GFP S246A and Eps15-SNAP (Figures 3E, 3F, 3M, and S3A). On average, the amount of SH3BP4/Eps15 double-positive CCPs increased with S246A (Figures 3G and S3N-S3Q) despite the fact that SH3BP4 S246A expression levels were lower than for the wt protein in

Article these cells (Figure S3M). Together, these data indicate that Eps15 contributes to SH3BP4 recruitment to CCPs and that this interaction is antagonized by 14-3-3 binding when SH3BP4 is phosphorylated at S246 by Akt.

## SH3BP4 binds GIPC1

To better understand the function of SH3BP4 at CCPs, we performed mass spectrometry to identify binding partners (Figures 4A and S4A; Table S5). As expected, these analyses indicated 14-3-3 isoforms, together with a few other reproducible binding partners, such as GIPC1. GIPC family proteins, of which there are three isoforms, are 37-kDa adaptors with a PDZ domain that binds to three-amino-acid-long motifs found at the C terminus of particular transmembrane receptors (Katoh, 2013), myosin-6 (Bunn et al., 1999), and APPL proteins (Varsano et al., 2006; Lin et al., 2006) (Schenck et al., 2008). SH3BP4 has previously been identified as a GIPC1 binding partner by high throughput proteomics, but this interaction was not further explored (Ewing et al., 2007) (O'Loughlin et al., 2018). Indeed, when we used GIPC1 as a bait, SH3BP4 was one of the strongest hits (Figure S4A; Table S5). Furthermore, endogenous GIPC1 co-immunoprecipitated with endogenous SH3BP4 (Figures 4B and S4B). SH3BP4 binding was specific to GIPC isoform 1 (Figures 4C and S4C). GIPC and SH3BP4 interact directly as shown by pull-down experiments with purified, recombinant GIPC1 with SH3BP4 (Figures 4D, S4D, and S4E). GIPC1 with mutant PDZ domain (Giese et al., 2012) showed enhanced SH3BP4 interaction (Figures 4E and S4F), confirming that SH3BP4didnot interact via the GIPC PDZ domain. To determine the domain of GIPC1-mediating interaction with SH3BP4 a series of GIPC1 fragments were tested for their ability to pull down SH3BP4 (Figure 4F). The C-terminal region without the PDZ domain showed enhanced binding to SH3BP4, suggesting intramolecular inhibition could shield the GIPC1 C-term (Figures 4G and S4G). Next, a series of N-terminal and C-terminal SH3BP4 fragments were tested for GIPC1 binding (Figures 4H-4J and S4H) and a middle region of SH3BP4 containing the S246 site emerged as the most likely binding domain (fragmentq, Figures 4K and S4H). Therefore, we tested whether binding to GIPC is also regulated by SH3BP4 phosphorylation in cell lysates and found that S246A pulled down GIPC1 more efficiently than wt (Figures 4L and S4I). When the SH3BP4-GIPC1 interaction was challenged with 14-3-3 on the R18 mutant constructs, GIPC1 pull down was reduced, although to a lesser extent than that observed in Eps15 (Figures 4L and S4I). Together, these data establish a GIPC1 and SH3BP4 interaction, which is sensitive to competition by 14-3-3 adaptors.

## SH3BP4 regulates neuropilin-1 and a 5-integrin endocytosis

Next, we explored whether SH3BP4 is involved in the endocytosis of transmembrane proteins that are known to bind GIPC1. GIPC1 binds via its PDZ domain the cytoplasmic tail of neuropilin-1 (NRP1) (Cai and Reed, 1999), a 5 and a 6-integrin (Tani and Mercurio, 2001; Mourabit et al., 2002). The GIPC1 PDZ domain binds the highly conserved C-terminal SEA* motif of the NRP1 cytoplasmic tail (Figure S5A; Table S2). GIPC1 was reported to mediate NRP1 endocytosis in human endothelial cells (Valdembri et al., 2009), and this function is conserved in

Article

Figure 3. 14-3-3 interferes with SH3BP4 binding to Eps15

<!-- image -->

(A) Total internal reflection fluorescence (TIRF) microscopy of SH3BP4-GFP wt and clathrin-mediated endocytosis (CME) mutants at clathrin-coated pits (CCPs)

(N = 3 experiments, n = 45 cells per condition). Clathrin heavy-chain binding site (LIDL), dynamin-2 binding (SH3), and Eps15 binding sites (NPF, all three) mutated. All mutations combined in dCME.

(B) Immunoprecipitation (IP) of Eps15, Dyn2, and CHC-TD GFP fusion proteins for HA-SH3BP4 (N = 5).

- (C) IP of SH3BP4-GFP R18 mutants for HA-Eps15 and endogenous 14-3-3 ε (N = 6).
- (D) IP of SH3BP4-GFP for HA-Eps15 and endogenous 14-3-3 ε with Akt1-wt-HA overexpression (N = 6).

(E) TIRF microscopy of SH3BP4-GFP S246A with Eps15-SNAP and tdTomato-Clca as CCP marker. Scale bar: 5

m

m.

- (F) Example tracks of a CCP positive for both SH3BP4-GFP S246A and Eps15-SNAP, data from (E). Intensity in arbitrary units (AU).
- (G) Percentage of SH3BP4-GFP (wt or S246A) and/or Eps15-SNAP positive CCPs, data from (E) (N = 3 experiments with n = 45 cells in total). For IPs, representative data are shown and statistics are in Figure S3. For statistics rank-sum test was used and box and whisker plots are shown (A and G), ns, not significant, ***p &lt; 0.001, ****p &lt; 0.0001.

<!-- image -->

<!-- image -->

Article

<!-- image -->

## Figure 4. SH3BP4 binds GIPC1

(A) Numbers of peptides for SH3BP4-binding partners by mass spectrometry (N = 3 experiments), mean ± SEM are shown.

- (B) Immunoprecipitation (IP) of endogenous SH3BP4 versus endogenous GIPC1 in RPE cells (N = 2).

(C) IP of SH3BP4-GFP for GIPC isoforms (N = 4).

(D) Pull down of purified GST-SH3BP4 versus purified GIPC1 (N = 3).

- (E) IP of GFP-GIPC1 wt and PDZ domain mutant LGL/AAA for HA-SH3BP4 (N = 3).

(F and G) IP of GFP-GIPC1 fragments for HA-SH3BP4 (N = 3).

(H-K) IP of SH3BP4-GFP fragments for HA-GIPC1 (N = 3).

- (L) IP of SH3BP4-GFP R18 mutants for HA-GIPC1 and endogenous 14-3-3 ε (N = 5). For IPs, representative data are shown and statistics are in Figure S4.

Article

Figure 5. SH3BP4 mediates neuropilin-1 (NRP1) and a 5-integrin endocytosis

<!-- image -->

(A) Immunoprecipitation (IP) of GFP-NRP1 cytoplasmic tail, wt, and dSEA for HA-GIPC1 wt and dPDZ mutant (LGL/AAA) and HA-SH3BP4 (N = 5 experiments). Long (L) and short (S) exposure of the HA-GIPC1 IP blot are shown.

(B) IP of GFP-NRP1 cytoplasmic tail for HA-GIPC1 wt and dPDZ and HA-SH3BP4 wt and mutants (N = 6).

(C) IP of GFP-NRP1 wt and dSEA cytoplasmic tails for HA-GIPC1 and HA-SH3BP4 wt and R18 mutants (N = 5).

(D) IP of GFP-NRP1 wt and dSEA cytoplasmic tails for HA-GIPC1 and HA-SH3BP4 with overexpression of Akt1-wt-HA (N = 8).

(E) Micrographs for the uptake of NRP1. Scale bar, 10 m m.

<!-- image -->

<!-- image -->

Xenopus laevis where XGIPC was also found to regulate a 5/ b 1 integrin endocytosis (Spicer et al., 2010). Furthermore, NRP1 neither has a conserved NPxY nor a Yxx ɸ motif, which would promote interactions with the Dab2/Numb and AP-2 adaptors, respectively (De Franceschi et al., 2016). This suggests GIPC1 could play an important role in NRP1 internalization (Figure S4A; Table S5) (Pang et al., 2014). However, it is not clear how GIPC1 is targeted to CCPs. To explore whether SH3BP4 could form a complex with GIPC1 and NRP1 simultaneously, the cytoplasmic portion of NRP1 was fused to GFP and the fusion bound GIPC, indeed. Interestingly, binding was increased /C24 2-fold in the presence of SH3BP4 (Figures 5A and S5B), but SH3BP4 did not bind NRP1 in the absence of GIPC1 (Figure 5A). While the dCME mutation had no effect on the formation of the NRP1/GIPC1/ SH3BP4 complex, the S246A mutation enhanced the incorporation of both GIPC1 and SH3BP4 (Figures 5B and S5C), suggesting that SH3BP4 phosphorylation at S246 impaired GIPC1/ NRP1 binding. Further supporting these findings, SH3BP4-R18 was only poorly pulled down by GFP-NRP1 (Figures 5C and S5D). Moreover, GFP-NRP1 pulled down less GIPC1 and SH3BP4 under Akt overexpression (Figures 5D and S5E). Together, these data suggested that the interaction of GIPC1 with the NRP1 cytoplasmic portion was enhanced by SH3BP4, but inhibited by 14-3-3 adaptors, further highlighting the importance of the Akt phosphorylation.

To test whether SH3BP4 was mediating NRP1 endocytosis, weestablished an anti-NRP1 antibody-uptake assay in RPE cells that were serum starved for 12 h, to suppress Akt activity (Figures 5E and S5F). In RPE NRP1 KO cells no antibody was internalized, while reconstitution with GFP-NRP1 in the NRP1 KO cells significantly increased antibody internalization (Figures S5GandS5H). Knockdown of CHC by siRNA reduced NRP1 uptake, suggesting a CME-dependent uptake mechanism (Figures S5I and S5J). Both in RPE GIPC1 KO and in SH3BP4 KO cells (Figure 5F), NRP1 internalization was reduced (Figure 5G). When the SH3BP4 KO cells were reconstituted with SH3BP4 GFP mutants, both wt and S246A restored NRP1 uptake in starved cells, while dCME did not (Figure 5H). This supported a model where SH3BP4 together with GIPC1 were mediating NRP1 internalization by CME.

Like NRP1, the a 5-integrin cytoplasmic domain has a highly conserved C-terminal S[D/E]A* motif that binds the GIPC1 PDZ domain (Figure S5K; Table S2) (Tani and Mercurio, 2001). To test whether SH3BP4 was involved in a 5-integrin endocytosis, we measured the uptake of an antibody against a 5-integrin in RPE cells (Figures S5L and S5M). The a 5-integrin uptake was strongly reduced in GIPC1 KO cells and restored with HIS8xGIPC1 reconstitution (Figures S5N-S5P). Similarly, a 5-integrin uptake was strongly reduced in SH3BP4 KO cells and restored

(F) Western blot of RPE knockout (KO) cells.

(G) Quantification of NRP1 uptake in RPE KO cells (N = 3 experiments), arbitrary units (AU).

(H) Quantification of NRP1 uptake in RPE KO cells reconstituted with SH3BP4-GFP wt and mutants (N = 3).

(I) Quantification of a 5-integrin uptake in KO cells (N = 3).

(J) Quantification of a 5-integrin uptake in RPE cells overexpressing constitutively active Akt1 (GFP-Akt1-ca, N = 3).

(K-M) Uptake assay for total (K, N = 4, antibody NKI-SAM-1), inactive (L, N = 3, antibody mAb11), and active (M, N = 3, antibody SNAKA51) a 5-integrin in RPE wt and SH3BP4 knockout cells. Internalization was quantified as a fraction of the total surface a 5-integrin, time in minutes (min). Mean ± SD are shown (K, L, and M); Student's t test was used. For statistics (G, H, I, and J) rank-sum test was used; data presented as box and whiskers, ns, not significant, *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, ****p &lt; 0.0001. For IP experiments representative data is shown, statistics are in Figure S5. For image-based uptake assays, representative experiment is shown.

Article upon reconstitution with either SH3BP4-GFP wt or S246A but only partially rescued by the dCME mutant (Figures 5I and S5Q-S5S). Finally, in RPE cells overexpressing constitutively active Akt, a 5-integrin uptake was partially inhibited (Figures 5J and S5T). Knockdown of CHC reduced a 5-integrin internalization (Figures S5I and S5U). Finally, in a biochemical uptake assay, SH3BP4 mediated internalization was not specific toward active or inactive a 5-integrins (Figures 5K-5M and S6A-S6C). As previously reported, GIPC1 was not specific for a a 5-integrin subpopulation (Figures S6D-S6G), while NRP1 was required for the uptake of active a 5-integrin (Figures S6H-S6K) (Valdembri et al., 2009). In summary, these data show that SH3BP4 does form a complex with the NRP1 C-term and GIPC1 that is sensitive to Akt phosphorylation and that both NRP1 and a 5-integrin endocytosis are dependent on SH3BP4.

## SH3BP4 regulates a 5-integrin surface expression

Considering these findings, we hypothesized that active SH3BP4 could reduce a 5-integrin steady-state cell surface expression. Cells overexpressing SH3BP4-GFP wt and S246A exhibited reduced total a 5-integrin surface expression when analyzed by flow cytometry (Figure 6A). Next, the active a 5-integrin surface expression on fibronectin-coated glass was measured (Humphries, 2009). SH3BP4 KO cells spread out to cover 70% more area than wt cells and exhibited more active a 5-integrin (Figures 6B, 6C, and S6L). These effects were reversed by reconstitution of KO cells with wt SH3BP4-GFP. Importantly, the cell area was reduced by 52% in cells reconstituted with SH3BP4 S246A compared with cells reconstituted with wt SH3BP4-GFP (Figure 6C). Expression of SH3BP4 S246A also reduced active a 5-integrin (Figures 6B and 6C). To test whether the S246A inhibitory activity on spreading and active a 5-integrins depended on GIPC1 and/or NRP1, the corresponding KO cells were analyzed with overexpression of S246A (Figures 6D and S6M). Expression of S246A had no effect on GIPC1 KO cell spreading or surface expression of active a 5-integrins, suggesting that GIPC1 was required for the S246A activity. In contrast, in NRP1 KO cells, S246A expression reduced spreading as in wt cells, suggesting that SH3BP4 inhibition of spreading was not dependent on NRP1. Active a 5/ b 1-integrins bind soluble fibronectin dimers and mediate the assembly of fibronectin fibrils (Mao and Schwarzbauer, 2005). Fibronectin matrix secretion and remodeling is a core functionality of fibroblasts, but it is also observed for endothelial cells. Human RPE cells display both epithelial and mesenchymal characteristics (Gan et al., 2016), and they formed networks of fibronectin fibrils (Figure 6E). In the absence of GIPC1 or SH3BP4, fewer cells displayed fibronectin fibrils and the remaining fibrils were shorter (Figures 6E and 6F), while NRP1 KO cells remained competent

Article

Figure 6. SH3BP4 regulates a 5-integrin surface expression

<!-- image -->

(A) Surface expression of a 5-integrins by flow cytometry (N = 4 experiments).

(B) Total internal reflection fluorescence (TIRF) microscopy of fixed RPE cells after spreading for 20 min on fibronectin. SH3BP4 knockout (KO) and SH3BP4-GFP wt and S246A mutant reconstitution. Actin and immunofluorescence (IF) for paxillin and active a 5-integrin (N = 3, n = 30 cells each).

<!-- image -->

<!-- image -->

for fibril assembly. In SH3BP4 KO cells, the fibril formation was rescued by expression of SH3BP4-GFP wt or S246A mutant (Figure 6G). In human endothelial HMEC1 cells, SH3BP4 knockout also reduced fibronectin fibrillogenesis (Figures S6NS6P), suggesting that the process of matrix remodeling is dependent on SH3BP4 across cell types. Together, these results support a model where SH3BP4 coordinated with GIPC1 regulates a 5-integrin surface expression and fibronectin extracellular matrix remodeling.

## SH3BP4 mediates semaphorin-3a induced cell contraction

NRP1 has been reported as a co-receptor for a variety of transmembrane proteins and as a receptor for a series of ligands, including the chemorepellent semaphorin-3a (SEMA3A) (Lampropoulou and Ruhrberg, 2014). SEMA3A binds NRP1 (He and Tessier-Lavigne, 1997; Kolodkin et al., 1997) leading to complex formation with Plexin-A receptors (PlexinA1-4) (Takahashi et al., 1999; Tamagnone et al., 1999), which are GTPase activating proteins (GAPs) for Rap GTPases (Wang et al., 2012). Evidence is accumulating that endocytosis is required for SEMA3A-induced cell collapse (Jurney et al., 2002; Castellani et al., 2004; Fournier et al., 2000; Tojima et al., 2010; Gioelli et al., 2018). In addition, NRP1 was shown to increase active a 5/ b 1-integrin endocytosis in a GIPC1 dependent manner (Valdembri et al., 2009) and SEMA3A stimulated active a 5-integrin uptake (Gioelli et al., 2018). Interestingly, multiple reports suggest that semaphorin3 family proteins, including SEMA3A signaling, reduces Akt activity (Eickholt et al., 2002; Manns et al., 2012; Castro-Rivera et al., 2008; Chadborn et al., 2006). Moreover, overexpression of dominant-negative phosphatase and tensin homolog (PTEN) C124S reduced SEMA3A-induced growth cone collapse (Chadborn et al., 2006), while in non-small cell lung cancer (NSCLC) cells SEMA3B signaling was blocked by high Akt activity (Castro-Rivera et al., 2008). Finally, NRP1 cytoplasmic tail was suggested to complex and activate PTEN upon semaphorin ligand binding (Delgoffe et al., 2013). Together, these data suggest that SEMA3A signaling reduces Akt activity and, conversely, that high Akt activity can antagonize SEMA3A induced cell collapse.

ToconfirmtheeffectofSEMA3AonAktactivity,wetreatedRPE wt cells with SEMA3A for 1 h and found a reduction in Ser473 phosphorylation (Figures 7A, 7B, and S7A). SEMA3A-induced cell collapse was inhibited in cells with clathrin heavy-chain knockdown, confirming that the process was CME dependent (Figure S7B). Next, we set out to test the hypothesis that SH3BP4 was required for SEMA3A signaling. Serum-starved human RPE cells and HMEC1 cells were sensitive to SEMA3Ainduced contraction, while in SH3BP4 KO cells, cell contraction was reduced (Figures 7C, 7D, and S7C). In RPE cells, SH3BP4GFP wt and S246A rescued SEMA3A signaling, while dCME did

Article not, suggesting that SH3BP4 was involved in a CME-dependent way (Figure 7D). RPE cells lacking NRP1 or GIPC1 were resistant to SEMA3A signaling (Figures S7D and S7E). Together, this suggested that GIPC1 and SH3BP4 were involved in SEMA3A signaling in a CME-dependent way. When serum-starved RPE cells expressing wild-type SH3BP4-GFP were observed by TIRF microscopy during SEMA3A signaling (Figure 7E), cells contracted over the 60 min of treatment (Figures 7F and 7G), preferentially from the cell edge that exhibited the highest SH3BP4-GFP localization (Figures S7F-S7N). In response to SEMA3A signaling, SH3BP4-GFP intensities at tdTomato-Clca labeled CCPs increased compared with untreated cells, indicating enhanced recruitment of SH3BP4-GFP (Figure 7H). SH3BP4-GFP dCME was not recruited to CCPs by SEMA3A (Figure S7O). The SEMA3A-induced accumulation of SH3BP4-GFP to CCPs was also absent in NRP1 and GIPC1 KO cells (Figures S7P and S7Q). When constitutively active Akt (Akt-ca) was overexpressed, RPEcellsbecameresistantto SEMA3A-inducedcontraction(Figures 7I and S5T) and Akt-ca blocked SH3BP4-GFP accumulation at CCPs (Figure 7J). Finally, SEMA3A enhanced NRP1 uptake but not total a 5-integrin uptake, in line with previous reports suggesting that SEMA3Aenhancestheuptakeofactive a 5-integrin (Gioelli et al., 2018) (Figures 7K and S7R). RPE cells lacking ARHGEF12 or MYO6,twoGIPC1bindingpartners(FigureS4A;TableS5),hadno effect on SH3BP4 mediated endocytosis of NRP1 or total a 5-integrinsandremainedSEMA3Asensitive,indicatingthatthesefactors were not relevant for SH3BP4-mediated endocytosis or signaling (Figures S7S-S7Z). Together, these data suggest that SEMA3A signaling depends on SH3BP4-mediated endocytosis. Moreover, SEMA3A promotes SH3BP4 recruitment to CCPs to enhance endocytosis and cell contraction, and this process is blocked by high Akt activity.

## Semaphorin-3a signaling in lung cancer cells depends on active SH3BP4

During tumorigenesis and metastasis, cancer cells adapt to ever-changing microenvironments. At various stages during the transformation, cancer cells experience chemo-attractive and chemorepellent signals from the surrounding tissues (Capparuccia and Tamagnone, 2009). To persist, the development of resistance mechanisms toward repellent signals is advantageous. In human lung cancer, the 3p21 locus is frequently deleted (Sekido et al., 1996; Senchenko et al., 2004) and the semaphorin-3 family gene SEMA3B was identified as a tumor suppressor in this region (Tomizawa et al., 2001; Castro-Rivera et al., 2004; Ji et al., 2005; Kuroki et al., 2003). In NSCLC cell lines, SEMA3B suppressed Akt activity, whereas cells with high Akt activity were resistant to semaphorin signaling (Castro-Rivera et al., 2008). We hypothesized that SH3BP4 could play a role in this balance of SEMA3A signaling. To study the role of the SH3BP4 endocytic mechanism in SEMA3A signaling

(C) Quantification of cell area, paxillin, and active a 5-integrin area from (B).

(D) Quantification of spreading in RPE KO cells overexpressing SH3BP4-GFP S246A (N = 3, n = 30).

(E) TIRF microscopy of RPE wt and SH3BP4 KO cells after 3 h on glass, fibronectin fibrils stained. Total a 5-integrin staining (antibody NKI-SAM-1) for cell outlines.

(F) Quantification of fibronectin fibrils (FF).

(G) FF in RPE SH3BP4 KO and SH3BP4-GFP wt and S246A reconstitution cells. The scale bar is 5 m m(B and E). Representative data are shown (C and D). For statistics Student's t test was used and data are presented as mean ±SEM (A, F, and G); or rank-sum test was used and data are presented with box and whisker plots (C and D), ns, not significant, *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, ****p &lt; 0.0001.

## Article

Figure 7. SH3BP4 mediates semaphorin-3a signaling

<!-- image -->

- (A) Western blot of semaphorin-3a (SEMA3A) versus phosphorylation of Akt Ser-473 (pS473), RPE wt cells.
- (B) Quantification of (A), mean ± SEM are shown and Student's t test was used (N = 4 experiments). (C) SEMA3A per RPE wt and SH3BP4 KO cell areas.
- (D) Quantification of (C) (N = 3 experiments, n = 100 cells).
- (E) Total internal reflection fluorescence microscopy of RPE wt cells overexpressing (OX) SH3BP4-GFP wt treated for 60 min with SEMA3A.
- (F) Cell outlines from (E) color-coded, 0-60 min.
- (G) Area of cell in (E) 0-60 min post SEMA3A.
- (H) Mean SH3BP4-GFP intensity at CCPs, control (N = 6 experiments, n = 92 cells, pooled) and upon SEMA3A (n = 91), intensities in arbitrary units (AU)
- (I) Effect of SEMA3A on cell area in RPE cells OX constitutively active Akt (GFP-Akt1-ca, N = 3, n = 100).
- (J) SH3BP4-GFP wt intensity at CCPs in RPE cells OX GFP-Akt1-ca versus control cells (N = 4 experiments, n = 73 cells pooled), upon SEMA3A treatment (n = 71).
- (K) Neuropilin-1 (NRP1) uptake upon stimulation with SEMA3A in RPE wt and SH3BP4 KO cells (N = 2). Representative data are shown (D, I, and K). Scale bar, 5 m m(CandE). For statistics, rank-sum test was used and data are presented as box and whisker plots (D, H, I, J, and K), ns, not significant, *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, ****p &lt; 0.0001.

in NSCLCs (Figure S8A), we screened a series of NSCLC cell lines for protein expression, Akt activity, and SEMA3A sensitivity (Figures 8A-8C and S8B). As expected, cell lines with low PTEN expression showed increased Akt phosphorylation at Ser473 (pS473) and high pS473, in turn, correlated with resistance to

SEMA3A induced cell contraction (Figure 8C). One such cell line was H157 that was confirmed to express a homozygous PTEN G251C missense mutant (Osoegawa et al., 2017) (Figures S8C and S8D) reported to lack phosphatase activity (Han et al., 2000). In H157, expression of GFP-PTEN wt restored sensitivity

<!-- image -->

<!-- image -->

Article

Figure 8. Semaphorin-3a signaling depends on PTEN

<!-- image -->

(A) Western blot (WB) analysis of non-small cell lung cancer (NSCLC) cells protein expression (N = 5 experiments, representative data, and statistics in supplemental information). Mutations for TP53, KRAS/NRAS (RAS), PTEN, and overexpression (OX) for MYC.

- (B) Quantification of PTEN and Akt phospho-Ser473 (pS473) intensities from WBs in (A), arbitrary units (AU)
- (C) NSCLC cell sensitivity to semaphorin-3a (SEMA3A) induced contraction (2-5 repeats per cell). p values above 0.05 indicate SEMA3A resistance.

(D) Effect of SEMA3A on, area of H157 wt and OX GFP-PTEN (N = 3 experiments, n = 100 cells).

- (E) Total internal reflection fluorescence (TIRF) microscopy of H157 wt cells expressing SH3BP4-GFP wt and S246A and TdTomato-clathrin light chain a (Clca). Scale bar, 5 m m.

(F) Quantification of (E) with SH3BP4-GFP signal at clathrin-coated pits (CCPs) (N = 3 experiments, wt n = 48 cells, S246A n = 49).

## Article

to SEMA3A (Figures 8D and S8E). When SH3BP4-GFP wt and S246A were expressed in H157 the mutant protein was efficiently targeted to CCPs as observed for RPE and endothelial cells (Figures 8E, 8F, S8F and S8G). To address how the reduction of Akt activity would affect SH3BP4-GFP recruitment to CCPs, SNAP-PTEN wt was overexpressed in H157 lowering Akt pS473 levels (Figure S8H). When analyzed by TIRF microscopy, SH3BP4-GFP accumulation at CCPs was elevated in the presence of exogenous PTEN expression (Figures 8G and S8I). This was consistent with the idea that reduced Akt activity would lead to increased SH3BP4 accumulation at CCPs. Finally, to address whether active non-phosphorylatable SH3BP4 S246A could restore SEMA3A sensitivity in H157, we generated SH3BP4 KO cells and reconstituted the cells with SH3BP4-GFP (Figure S8J). When the cells were challenged with SEMA3A for 1 h, the S246A mutant cells contracted (Figure 8H). Together, these data confirm the critical role of SH3BP4 in SEMA3A signaling und supports a model where high Akt activity leads to SEMA3A resistance by phosphorylating SH3BP4 at S246. These results suggest that in cancer cells with low PTEN and high Akt activities, SH3BP4 is inactivated and cells become resistant to semaphorin signaling because of impaired SH3BP4-mediated endocytosis.

## DISCUSSION

This study establishes the CME accessory factor SH3BP4 as a central link between Akt signaling and cell-matrix adhesion regulation via endocytosis of NRP1 and a 5-integrins (Figure S8K). When dephosphorylated at S246, SH3BP4 accumulates at CCPs to target via GIPC1 cell adhesion receptors for endocytosis. This reduces integrin surface expression and compromises cell-matrix attachment. Conversely, upon Akt phosphorylation of S246, SH3BP4 is sequestered by 14-3-3 adaptor proteins and excluded from CCPs. This inhibits SH3BP4's endocytic function establishing Akt as a regulator of a 5-integrin surface availability.

SH3BP4 not only translates via endocytosis the global Akt state to integrin surface expression but also functions as an integrator of signals from cell surface receptors that regulate integrin availability. Here, we show how the semaphorin SEMA3A via its receptor NRP1 activates SH3BP4 and promotes cell contraction, effects that are well established but have yet to be mechanistically explained (Serini et al., 2003). Our data now indicate that acute exposure of cells to SEMA3A enhances SH3BP4 recruitment to CCPs and that SH3BP4 knockout reduces SEMA3A induced cell contraction (Figure 7). SEMA3A induced cell contraction is thought to depend mostly on NRP1 signaling (Gioelli et al., 2018). Besides NRP1, SEMA3A also binds plexinA, which leads to the activation of the cytoplasmic GAP domain toward the Rap1 GTPase (Wang et al., 2012; Wang et al., 2013; Gioelli et al., 2018). SEMA3A reduces the activity of Rap1 and its various downstream effectors (Bos et al., 2001). This has been reported to inhibit cell migration but it does not trigger cell

<!-- image -->

contraction (Gioelli et al., 2018). Our data support a model where SEMA3A signaling activates PTEN to reduce Akt activity.

Throughout tumorigenesis, cancer cells receive a plethora of signals from their microenvironment, including from chemorepellents like semaphorins. We found that hyperphosphorylation of SH3BP4 at S246 by Akt contributes to SEMA3A resistance in NSCLCs. Conversely, reconstitution with the non-phosphorylatable mutant SH3BP4 sensitized cells for SEMA3A. These data offer mechanistic insight to earlier reports of semaphorin resistance in cells with high Akt expression. In NSCLCs, PTEN mutations were found in 1%-3% of lung adenocarcinoma patients and in up to 15%-22% of squamous cell carcinoma patients (Campbell et al., 2016; Cancer Genome Atlas Research, 2012, 2014, Sanchez-Vega et al., 2018). This suggests that a considerable fraction of NSCLCs could invoke a semaphorin resistance mechanism through the loss of PTEN together with enhanced Akt activity and inhibition of SH3BP4-mediated endocytosis. While hyperactive Akt may be a frequent semaphorin resistance mechanism, other pathways may likely co-exist.

Most of the current models of CME describe a constitutive protein assembly process at the plasma membrane orchestrated by the adaptor AP-2 complex and clathrin together with a range of accessory factors with overlapping functionality (Schmid and McMahon, 2007). However, various receptors are not constitutively loaded into CCPs. Thus, incorporation must be tightly controlled, yet mechanisms for such regulation have been elusive. Here, we describe how phosphorylation in response to cellintrinsic or extrinsic signaling via Akt selectively determines the incorporation of the accessory factor SH3BP4 into CCPs upon which NRP1 and a 5-integrins are internalized by CME. Most certainly, SH3BP4isonlyoneamongseveralCCP-associatedfactorsthattransiently engagewiththeCMEmachineryinresponseto signaling inputs. Our experiments now dissect the regulation for one of this vastly understudied class of CME adaptors with potentially central roles as an integrator of signaling and trafficking.

## Limitations of the study

Interestingly, NRP1 endocytosis was previously shown to be sensitive to the nutritional state of cells, wherein, particularly, insulin signaling strongly inhibited the uptake of NRP1-binding ligands (Pang et al., 2014). Insulin binds the insulin receptor, followed by tyrosine-kinase activation, recruitment of IRS-1/2 adapters, and finally activation of PI3K and Akt (Haeusler et al., 2018). In independent, unrelated studies SH3BP4 S246 phosphorylation was reported to be enhanced by insulin signaling measured by mass spectrometry (Humphrey et al., 2013, 2015). These observations line-up with our data showing SH3BP4 as a central switch between PI3K/Akt signaling and endocytosis. However, Pang et al. found that their NRP1 peptide ligand attached to silver nanoparticles entered cells independent of the CME machinery but identified internalized particles in vesicles resembling macropinosomes (Pang et al., 2014). Although our data indicate that NRP1 is internalized in an SH3BP4- and

(G) TIRF microscopy of H157 cells OX SH3BP4-GFP wt, with OX of SNAP-PTEN wt. Analysis of SH3BP4-GFP at CCPs (N = 4) for control (n = 60 cells) and SNAP-PTEN OX (n = 61).

(H) Effect of SEMA3A on cell area in H157 wt and SH3BP4 KO cells reconstituted with SH3BP4-GFP wt and S246A (N = 3, n = 100). For statistics mean ± SEM is shown (B and C) or rank-sum test and box and whisker plots were used, a representative experiment is shown (D, F, G, and H), ns, not significant, *p &lt; 0.05, ****p &lt; 0.0001.

<!-- image -->

CME-dependent way, we cannot exclude the possibility that NRP1follows alternative endocytic routes. The mode of internalization of NRP1 may depend on the nature of the ligand and the metabolic and signaling state of the cell.

## STAR + METHODS

Detailed methods are provided in the online version of this paper and include the following:

- d KEY RESOURCES TABLE
- d RESOURCE AVAILABILITY
- B Lead contact
- B Materials availability
- B Data and code availability
- d EXPERIMENTAL MODEL AND SUBJECT DETAILS
- B Cells and culture
- d METHOD DETAILS
- B DNA constructs
- B mRNA extraction, cDNA generation and sequencing
- B siRNA knockdown of clathrin heavy chain
- B CRISPR/Cas9 knockout cells
- B Lentiviral expression
- B Expression and purification of Semaphorin-3a
- B Western blotting and antibodies
- B Protein expression and purification
- B Immunoprecipitation and pull-down experiments
- B Mass spectrometry
- B Live cell microscopy and analysis
- B Cell based assays
- B Analysis of a 5-integrin surface expression by flow cytometry
- B Bioinformatics analysis
- d QUANTIFICATION AND STATISTICAL ANALYSIS
- B Statistical analysis

## SUPPLEMENTAL INFORMATION

Supplemental information can be found online at https://doi.org/10.1016/j. devcel.2021.03.009.

## ACKNOWLEDGMENTS

We thank Sandra Schmid for guidance on endocytosis experiments and for critically reading the manuscript. We thank members of the Danuser lab for helpful discussions. We thank Ross Tomaino from the Taplin Mass Spectrometry Facility at the Department of Cell Biology Cell Biology at Harvard Medical School for expert analysis. We thank Luc Girard for genomic analyses of lung cancer lines. We thank Nicolas Loof for support with cell sorting/flow cytometry analysis for this project on instruments in the Moody Foundation Flow Cytometry Facility at UT Southwestern Medical Center. This project was supported by the Swiss National Science Foundation (SNF, PA00P3\_139694) and the Novartis Research Foundation (both to C.J.B.). Funding in the Danuser lab for this work was made available through NIH grants GM067230, GM073165, and GM136428. This work was supported by NIH Lung SPORE grant P50 CA070907 (J.D.M.).

## AUTHOR CONTRIBUTIONS

C.J.B. conceived and designed the study and executed all experiments and analyses with overall scientific observation by G.D.; C.J.B. wrote the manuscript with substantial edits by G.D.; J.D.M. provided reagents, scientific guidance, and critical insights.

## DECLARATION OF INTERESTS

The authors declare no competing interests. G.D. is a member of the editorial board of Developmental Cell .

Received: June 3, 2020

Revised: December 23, 2020

Accepted: February 27, 2021

Published: March 23, 2021

## REFERENCES

Adams, R.H., Lohrum, M., Klostermann, A., Betz, H., and P € uschel, A.W. (1997). The chemorepulsive activity of secreted semaphorins is regulated by furindependent proteolytic processing. EMBO J. 16 , 6077-6086.

Aguet, F., Antonescu, C.N., Mettlen, M., Schmid, S.L., and Danuser, G. (2013). Advances in analysis of low signal-to-noise images link dynamin and AP2 to the functions of an endocytic checkpoint. Dev. Cell 26 , 279-291.

Altschuler, Y., Barbas, S.M., Terlecky, L.J., Tang, K., Hardy, S., Mostov, K.E., and Schmid, S.L. (1998). Redundant and distinct functions for dynamin-1 and dynamin-2 isoforms. J. Cell Biol. 143 , 1871-1881.

Antas, P., Novellasdemunt, L., Kucharska, A., Massie, I., Carvalho, J., Oukrif, D., Nye, E., Novelli, M., and Li, V.S.W. (2019). SH3BP4 regulates intestinal stem cells and tumorigenesis by modulating beta-catenin nuclear localization. Cell Rep. 26 , 2266-2273 e4.

Antonny, B., Burd, C., De Camilli, P., Chen, E., Daumke, O., Faelber, K., Ford, M., Frolov, V.A., Frost, A., Hinshaw, J.E., et al. (2016). Membrane fission by dynamin: what we know and what we need to know. EMBO J. 35 , 2270-2284.

Artimo, P., Jonnalagedda, M., Arnold, K., Baratin, D., Csardi, G., De Castro, E., Duvaud, S., Flegel, V., Fortier, A., Gasteiger, E., et al. (2012). Expasy: SIB bioinformatics resource portal. Nucleic Acids Res 40 , W597-W603.

Beausoleil, S.A., Ville ´ n, J., Gerber, S.A., Rush, J., and Gygi, S.P. (2006). A probability-based approach for high-throughput protein phosphorylation analysis and site localization. Nat. Biotechnol. 24 , 1285-1292.

Bos, J.L., De Rooij, J., and Reedquist, K.A. (2001). Rap1 signalling: adhering to new models. Nat. Rev. Mol. Cell Biol. 2 , 369-377.

Bouaouina, M., Harburger, D.S., and Calderwood, D.A. (2012). Talin and signaling through integrins. Methods Mol. Biol. 757 , 325-347.

Bouvard, D., Pouwels, J., De Franceschi, N., and Ivaska, J. (2013). Integrin inactivators: balancing cellular functions in vitro and in vivo. Nat. Rev. Mol. Cell Biol. 14 , 430-442.

Bunn, R.C., Jensen, M.A., and Reed, B.C. (1999). Protein interactions with the glucose transporter binding protein GLUT1CBP that provide a link between GLUT1 and the cytoskeleton. Mol. Biol. Cell 10 , 819-832.

Cai, H., and Reed, R.R. (1999). Cloning and characterization of neuropilin-1-interacting protein: a PSD-95/Dlg/ZO-1 domain-containing protein that interacts with the cytoplasmic domain of neuropilin-1. J. Neurosci. 19 , 6519-6527.

Cai, L., Lin, S., Girard, L., Zhou, Y., Yang, L., Ci, B., Zhou, Q., Luo, D., Yao, B., Tang, H., et al. (2019). LCE: an open web portal to explore gene expression and clinical associations in lung cancer. Oncogene 38 , 2551-2564.

Calderwood, D.A., Fujioka, Y., De Pereda, J.M., Garcı ´a-Alvarez, B., Nakamoto, T., Margolis, B., Mcglade, C.J., Liddington, R.C., and Ginsberg, M.H. (2003). Integrin beta cytoplasmic domain interactions with phosphotyrosine-binding domains: a structural prototype for diversity in integrin signaling. Proc. Natl. Acad. Sci. USA 100 , 2272-2277.

Campbell, J.D., Alexandrov, A., Kim, J., Wala, J., Berger, A.H., Pedamallu, C.S., Shukla, S.A., Guo, G., Brooks, A.N., Murray, B.A., et al. (2016). Distinct patterns of somatic genome alterations in lung adenocarcinomas and squamous cell carcinomas. Nat Genet 48 , 607-616.

Cancer genome atlas research network (2012). Comprehensive genomic characterization of squamous cell lung cancers. Nature 489 , 519-525.

Cancer genome atlas research network (2014). Comprehensive molecular profiling of lung adenocarcinoma. Nature 511 , 543-550.

Article

## Article

Capparuccia, L., and Tamagnone, L. (2009). Semaphorin signaling in cancer cells and in cells of the tumor microenvironment-two sides of a coin. J. Cell Sci. 122 , 1723-1736.

Castellani, V., Falk, J., and Rougon, G. (2004). Semaphorin3A-induced receptor endocytosis during axon guidance responses is mediated by L1 Cam. Mol. Cell. Neurosci. 26 , 89-100.

Castro-Rivera, E., Ran, S., Brekken, R.A., and Minna, J.D. (2008). Semaphorin 3Binhibits the phosphatidylinositol 3-kinase/Akt pathway through neuropilin-1 in lung and breast cancer cells. Cancer Res. 68 , 8295-8303.

Castro-Rivera, E., Ran, S., Thorpe, P., and Minna, J.D. (2004). Semaphorin 3B (SEMA3B)induces apoptosis in lung and breast cancer, whereas VEGF165 antagonizes this effect. Proc. Natl. Acad. Sci. USA 101 , 11432-11437.

Caswell, P.T., Vadrevu, S., and Norman, J.C. (2009). Integrins: masters and slaves of endocytic transport. Nat. Rev. Mol. Cell Biol. 10 , 843-853.

Cermak, T., Doyle, E.L., Christian, M., Wang, L., Zhang, Y., Schmidt, C., Baller, J.A., Somia, N.V., Bogdanove, A.J., and Voytas, D.F. (2011). Efficient design and assembly of custom TALEN and other TAL effector-based constructs for DNA targeting. Nucleic Acids Res 39 , e82.

Chadborn, N.H., Ahmed, A.I., Holt, M.R., Prinjha, R., Dunn, G.A., Jones, G.E., and Eickholt, B.J. (2006). PTEN couples Sema3A signalling to growth cone collapse. J. Cell Sci. 119 , 951-957.

Chiba, S., Tokuhara, M., Morita, E.H., and Abe, S. (2009). TTP at Ser245 phosphorylation by AKT is required for binding to 14-3-3. J. Biochem. 145 , 403-409.

Christianson, T.W., Sikorski, R.S., Dante, M., Shero, J.H., and Hieter, P. (1992). Multifunctional yeast high-copy-number shuttle vectors. Gene 110 , 119-122.

Clark, K., Pankov, R., Travis, M.A., Askari, J.A., Mould, A.P., Craig, S.E., Newham, P., Yamada, K.M., and Humphries, M.J. (2005). A specific alpha5beta1-integrin conformation promotes directional integrin translocation and fibronectin matrix formation. J. Cell Sci. 118 , 291-300.

Conner, S.D., and Schmid, S.L. (2003). Regulated portals of entry into the cell. Nature 422 , 37-44.

Cunningham, F., Achuthan, P., Akanni, W., Allen, J., Amode, M.R., Armean, I.M., Bennett, R., Bhai, J., Billis, K., Boddu, S., et al. (2019). Ensembl 2019. Nucleic Acids Res 47 , D745-D751.

De Franceschi, N., Arjonen, A., Elkhatib, N., Denessiouk, K., Wrobel, A.G., Wilson, T.A., Pouwels, J., Montagnac, G., Owen, D.J., and Ivaska, J. (2016). Selective integrin endocytosis is driven by interactions between the integrin alpha-chain and AP2. Nat. Struct. Mol. Biol. 23 , 172-179.

Delgoffe, G.M., Woo, S.R., Turnis, M.E., Gravano, D.M., Guy, C., Overacre, A.E., Bettini, M.L., Vogel, P., Finkelstein, D., Bonnevier, J., et al. (2013). Stability and function of regulatory T cells is maintained by a neuropilin-1-semaphorin-4a axis. Nature 501 , 252-256.

Dunlevy, J.R., Berryhill, B.L., Vergnes, J.P., Sundarraj, N., and Hassell, J.R. (1999). Cloning, chromosomal localization, and characterization of cDNA from a novel gene, SH3BP4, expressed by human corneal fibroblasts. Genomics 62 , 519-524.

Dunn, K.C., Aotaki-Keen, A.E., Putkey, F.R., and Hjelmeland, L.M. (1996). ARPE-19, a human retinal pigment epithelial cell line with differentiated properties. Exp. Eye Res. 62 , 155-169.

Edelstein, A., Amodaj, N., Hoover, K., Vale, R., and Stuurman, N. (2010). Computer control of microscopes using m Manager. Curr. Protoc. Mol. Biol. 92 , 14.20.1-14.20.17.

Eickholt, B.J., Walsh, F.S., and Doherty, P. (2002). An inactive pool of GSK-3 at the leading edge of growth cones is implicated in semaphorin 3A signaling. J. Cell Biol. 157 , 211-217.

El Mourabit, H., Poinat, P., Koster, J., Sondermann, H., Wixler, V., Wegener, E., Laplantine, E., Geerts, D., Georges-Labouesse, E., Sonnenberg, A., and Aumailley, M. (2002). The PDZ domain of TIP-2/GIPC interacts with the C-terminus of the integrin alpha5 and alpha6 subunits. Matrix Biol. 21 , 207-214.

Eng, J.K., Mccormack, A.L., and Yates, J.R. (1994). An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. J. Am. Soc. Mass Spectrom. 5 , 976-989.

<!-- image -->

Ewing, R.M., Chu, P., Elisma, F., Li, H., Taylor, P., Climie, S., McBroomCerajewski, L., Robinson, M.D., O'Connor, L., Li, M., et al. (2007). Large-scale mapping of human protein-protein interactions by mass spectrometry. Mol. Syst. Biol. 3 , 89.

Fournier, A.E., Nakamura, F., Kawamoto, S., Goshima, Y., Kalb, R.G., and Strittmatter, S.M. (2000). Semaphorin3A enhances endocytosis at sites of receptor-F-actin colocalization during growth cone collapse. J. Cell Biol. 149 , 411-422.

Francavilla, C., Rigbolt, K.T., Emdal, K.B., Carraro, G., Vernet, E., BekkerJensen, D.B., Streicher, W., Wikstro ¨ m, M., Sundstro ¨ m, M., Bellusci, S., et al. (2013). Functional proteomics defines the molecular switch underlying FGF receptor trafficking and cellular outputs. Mol. Cell 51 , 707-722.

Gan, Z., Ding, L., Burckhardt, C.J., Lowery, J., Zaritsky, A., Sitterley, K., Mota, A., Costigliola, N., Starker, C.G., Voytas, D.F., et al. (2016). Vimentin intermediate filaments template microtubule networks to enhance persistence in cell polarity and directed migration. Cell Syst. 3 252-263.e8.

Gazdar, A.F., Girard, L., Lockwood, W.W., Lam, W.L., and Minna, J.D. (2010). Lung cancer cell lines as tools for biomedical discovery and research. J. Natl. Cancer Inst. 102 , 1310-1321.

Giese, A.P., Ezan, J., Wang, L., Lasvaux, L., Lembo, F., Mazzocco, C., Richard, E., Reboul, J., Borg, J.P., Kelley, M.W., et al. (2012). Gipc1 has a dual role in Vangl2 trafficking and hair bundle integrity in the inner ear. Development 139 , 3775-3785.

Gietz, R.D., and Schiestl, R.H. (2007). Frozen competent yeast cells that can be transformed with high efficiency using the LiAc/SS carrier DNA/PEG method. Nat. Protoc. 2 , 1-4.

Gioelli, N., Maione, F., Camillo, C., Ghitti, M., Valdembri, D., Morello, N., Darche, M., Zentilin, L., Cagnoni, G., Qiu, Y., et al. (2018). A rationally designed NRP1-independent superagonist SEMA3A mutant is an effective anticancer agent. Sci. Transl. Med. 10 , eaah4807.

Haeusler, R.A., Mcgraw, T.E., and Accili, D. (2018). Biochemical and cellular properties of insulin receptor signalling. Nat. Rev. Mol. Cell Biol. 19 , 31-44.

Han, S.Y., Kato, H., Kato, S., Suzuki, T., Shibata, H., Ishii, S., Shiiba, K., Matsuno, S., Kanamaru, R., and Ishioka, C. (2000). Functional evaluation of PTENmissense mutations using in vitro phosphoinositide phosphatase assay. Cancer Res. 60 , 3147-3151.

Hardy, S., Kitamura, M., Harris-Stansil, T., Dai, Y., and Phipps, M.L. (1997). Construction of adenovirus vectors through Cre-lox recombination. J. Virol. 71 , 1842-1849.

He, K., Marsland, R., III, Upadhyayula, S., Song, E., Dang, S., Capraro, B.R., Wang, W., Skillern, W., Gaudin, R., Ma, M., and Kirchhausen, T. (2017). Dynamics of phosphoinositide conversion in clathrin-mediated endocytic traffic. Nature 552 , 410-414.

He, Z., and Tessier-Lavigne, M. (1997). Neuropilin is a receptor for the axonal chemorepellent semaphorin III. Cell 90 , 739-751.

Hornbeck, P.V., Zhang, B., Murray, B., Kornhauser, J.M., Latham, V., and Skrzypek, E. (2015). PhosphoSitePlus, 2014: mutations, PTMs and recalibrations. Nucleic Acids Res 43 , D512-D520.

Humphrey, S.J., Azimifar, S.B., and Mann, M. (2015). High-throughput phosphoproteomics reveals in vivo insulin signaling dynamics. Nat. Biotechnol. 33 , 990-995.

Humphrey, S.J., Yang, G., Yang, P., Fazakerley, D.J., Sto ¨ ckli, J., Yang, J.Y., and James, D.E. (2013). Dynamic adipocyte phosphoproteome reveals that Akt directly regulates mTORC2. Cell Metab. 17 , 1009-1020.

Humphries, M.J. (2009). Cell adhesion assays. Methods Mol. Biol. 522 , 203-210.

Hynes, R.O. (2002). Integrins: bidirectional, allosteric signaling machines. Cell 110 , 673-687.

Jaqaman, K., Loerke, D., Mettlen, M., Kuwata, H., Grinstein, S., Schmid, S.L., and Danuser, G. (2008). Robust single-particle tracking in live-cell time-lapse sequences. Nat. Methods 5 , 695-702.

Ji, L., Minna, J.D., and Roth, J.A. (2005). 3p21.3 tumor suppressor cluster: prospects for translational applications. Future Oncol. 1 , 79-92.

## ll

Jiang, X.R., Jimenez, G., Chang, E., Frolkis, M., Kusler, B., Sage, M., Beeche, M., Bodnar, A.G., Wahl, G.M., Tlsty, T.D., and Chiu, C.-P. (1999). Telomerase expression in human somatic cells does not induce changes associated with a transformed phenotype. Nat. Genet. 21 , 111-114.

Jurney, W.M., Gallo, G., Letourneau, P.C., and McLoon, S.C. (2002). Rac1mediated endocytosis during ephrin-A2- and semaphorin 3A-induced growth cone collapse. J. Neurosci. 22 , 6019-6028.

Katoh, M. (2013). Functional proteomics, human genetics and cancer biology of GIPC family members. Exp. Mol. Med. 45 , e26.

Khanobdee, K., Kolberg, J.B., and Dunlevy, J.R. (2004). Nuclear and plasma membrane localization of SH3BP4 in retinal pigment epithelial cells. Mol. Vis. 10 , 933-942.

Kim, M., Carman, C.V., and Springer, T.A. (2003). Bidirectional transmembrane signaling by cytoplasmic domain separation in integrins. Science 301 , 1720-1725.

Kim, Y.M., Stone, M., Hwang, T.H., Kim, Y.G., Dunlevy, J.R., Griffin, T.J., and Kim, D.H. (2012). SH3BP4 is a negative regulator of amino acid-rag GTPasemTORC1 signaling. Mol. Cell 46 , 833-846.

Kokoszy /C19 nska, K., Kry /C19 nski, J., Rychlewski, L., and Wyrwicz, L.S. (2009). Unexpected domain composition of MACC1 links MET signaling and apoptosis. Acta Biochim. Pol. 56 , 317-323.

Kolodkin, A.L., Levengood, D.V., Rowe, E.G., Tai, Y.T., Giger, R.J., and Ginty, D.D. (1997). Neuropilin is a semaphorin III receptor. Cell 90 , 753-762.

Kuroki, T., Trapasso, F., Yendamuri, S., Matsuyama, A., Alder, H., Williams, N.N., Kaiser, L.R., and Croce, C.M. (2003). Allelic loss on chromosome 3p21.3 and promoter hypermethylation of semaphorin 3B in non-small cell lung cancer. Cancer Res. 63 , 3352-3355.

Lampropoulou, A., and Ruhrberg, C. (2014). Neuropilin regulation of angiogenesis. Biochem. Soc. Trans. 42 , 1623-1628.

Lin, D.C., Quevedo, C., Brewer, N.E., Bell, A., Testa, J.R., Grimes, M.L., Miller, F.D., and Kaplan, D.R. (2006). APPL1 associates with TrkA and GIPC1 and is required for nerve growth factor-mediated signal transduction. Mol. Cell. Biol. 26 , 8928-8941.

Liu, Y., Fisher, D.A., and Storm, D.R. (1994). Intracellular sorting of neuromodulin (GAP-43) mutants modified in the membrane targeting domain. J. Neurosci. 14 , 5807-5817.

Loerke, D., Mettlen, M., Schmid, S.L., and Danuser, G. (2011). Measuring the hierarchy of molecular events during clathrin-mediated endocytosis. Traffic 12 , 815-825.

Lu, G., Zhang, Q., Huang, Y., Song, J., Tomaino, R., Ehrenberger, T., Lim, E., Liu, W., Bronson, R.T., Bowden, M., et al. (2014). Phosphorylation of ETS1 by Src family kinases prevents its recognition by the COP1 tumor suppressor. Cancer Cell 26 , 222-234.

Manning, B.D., and Cantley, L.C. (2007). AKT/PKB signaling: navigating downstream. Cell 129 , 1261-1274.

Manns, R.P., Cook, G.M., Holt, C.E., and Keynes, R.J. (2012). Differing semaphorin 3A concentrations trigger distinct signaling mechanisms in growth cone collapse. J. Neurosci. 32 , 8554-8559.

Mao, Y., and Schwarzbauer, J.E. (2005). Fibronectin fibrillogenesis, a cellmediated matrix assembly process. Matrix Biol. 24 , 389-399.

Mcmahon, H.T., and Boucrot, E. (2011). Molecular mechanism and physiological functions of clathrin-mediated endocytosis. Nat. Rev. Mol. Cell Biol. 12 , 517-533.

Mettlen, M., Chen, P.H., Srinivasan, S., Danuser, G., and Schmid, S.L. (2018). Regulation of Clathrin-mediated endocytosis. Annu. Rev. Biochem. 87 , 871-896.

Mettlen, M., Loerke, D., Yarar, D., Danuser, G., and Schmid, S.L. (2010). Cargo- and adaptor-specific mechanisms regulate clathrin-mediated endocytosis. J. Cell Biol. 188 , 919-933.

Moreno-Layseca, P., Icha, J., Hamidi, H., and Ivaska, J. (2019). Integrin trafficking in cells and tissues. Nat. Cell Biol. 21 , 122-132.

Article

Muslin, A.J., Tanner, J.W., Allen, P.M., and Shaw, A.S. (1996). Interaction of 14-3-3 with signaling proteins is mediated by the recognition of phosphoserine. Cell 84 , 889-897.

Nishimura, T., and Kaibuchi, K. (2007). Numb controls integrin endocytosis for directional cell migration with aPKC and PAR-3. Dev. Cell 13 , 15-28.

O'Loughlin, T., Masters, T.A., and Buss, F. (2018). The MYO6 interactome reveals adaptor complexes coordinating early endosome and cytoskeletal dynamics. EMBO Rep. 19 , e44884.

Osoegawa, A., Gills, J.J., Kawabata, S., and Dennis, P.A. (2017). Rapamycin sensitizes cancer cells to growth inhibition by the PARP inhibitor olaparib. Oncotarget 8 , 87044-87053.

Oughtred, R., Stark, C., Breitkreutz, B.J., Rust, J., Boucher, L., Chang, C., Kolas, N., O'Donnell, L., Leung, G., Mcadam, R., et al. (2019). The BioGRID interaction database: 2019 update. Nucleic Acids Res 47 , D529-D541.

Pang, H.B., Braun, G.B., Friman, T., Aza-Blanc, P., Ruidiaz, M.E., Sugahara, K.N., Teesalu, T., and Ruoslahti, E. (2014). An endocytosis pathway initiated through neuropilin-1 and regulated by nutrient availability. Nat. Commun. 5 , 4904.

Park, J.S., Burckhardt, C.J., Lazcano, R., Solis, L.M., Isogai, T., Li, L., Chen, C.S., Gao, B., Minna, J.D., Bachoo, R., et al. (2020). Mechanical regulation of glycolysis via cytoskeleton architecture. Nature 578 , 621-626.

Peng, J., and Gygi, S.P. (2001). Proteomics: the move to mixtures. J. Mass Spectrom. 36 , 1083-1091.

Prunier, C., and Howe, P.H. (2005). Disabled-2 (Dab2) is required for transforming growth factor beta-induced epithelial to mesenchymal transition (EMT). J. Biol. Chem. 280 , 17540-17548.

Ran, F.A., Hsu, P.D., Wright, J., Agarwala, V., Scott, D.A., and Zhang, F. (2013). Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. 8 , 2281-2308.

Reinartz, M., Raupach, A., Kaisers, W., and Go ¨ decke, A. (2014). AKT1 and AKT2 induce distinct phosphorylation patterns in HL-1 cardiac myocytes. J. Proteome Res. 13 , 4232-4245.

Rizk, A., Paul, G., Incardona, P., Bugarski, M., Mansouri, M., Niemann, A., Ziegler, U., Berger, P., and Sbalzarini, I.F. (2014). Segmentation and quantification of subcellular structures in fluorescence microscopy images using Squassh. Nat. Protoc. 9 , 586-596.

Salcini, A.E., Confalonieri, S., Doria, M., Santolini, E., Tassi, E., Minenkova, O., Cesareni, G., Pelicci, P.G., and Di Fiore, P.P. (1997). Binding specificity and in vivo targets of the EH domain, a novel protein-protein interaction module. Genes Dev. 11 , 2239-2249.

Sanchez-Vega, F., Mina, M., Armenia, J., Chatila, W.K., Luna, A., La, K.C., Dimitriadoy, S., Liu, D.L., Kantheti, H.S., Saghafinia, S., et al. (2018). Oncogenic signaling pathways in the cancer genome atlas. Cell 173 , 321337 e10.

Schenck, A., Goto-Silva, L., Collinet, C., Rhinn, M., Giner, A., Habermann, B., Brand, M., and Zerial, M. (2008). The endosomal protein Appl1 mediates Akt substrate specificity and cell survival in vertebrate development. Cell 133 , 486-497.

Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods 9 , 676-682.

Schmid, E.M., and Mcmahon, H.T. (2007). Integrating molecular and network biology to decode endocytosis. Nature 448 , 883-888.

Schwartz, M.A. (2010). Integrins and extracellular matrix in mechanotransduction. Cold Spring Harb. Perspect. Biol. 2 , a005066.

Sekido, Y., Bader, S., Latif, F., Chen, J.Y., Duh, F.M., Wei, M.H., Albanesi, J.P., Lee, C.C., Lerman, M.I., and Minna, J.D. (1996). Human semaphorins A(V) and IV reside in the 3p21.3 small cell lung cancer deletion region and demonstrate distinct expression patterns. Proc. Natl. Acad. Sci. USA 93 , 4120-4125.

Senchenko, V.N., Liu, J., Loginov, W., Bazov, I., Angeloni, D., Seryogin, Y., Ermilova, V., Kazubskaya, T., Garkavtseva, R., Zabarovska, V.I., et al. (2004). Discovery of frequent homozygous deletions in chromosome 3p21.3 LUCA and AP20 regions in renal, lung and breast carcinomas. Oncogene 23 , 5719-5728.

## Article

Serini, G., Valdembri, D., Zanivan, S., Morterra, G., Burkhardt, C., Caccavari, F., Zammataro, L., Primo, L., Tamagnone, L., Logan, M., et al. (2003). Class 3 semaphorins control vascular morphogenesis by inhibiting integrin function. Nature 424 , 391-397.

Shevchenko, A., Wilm, M., Vorm, O., and Mann, M. (1996). Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels. Anal. Chem. 68 , 850-858.

Skene, J.H., and Vira ´ g, I. (1989). Posttranslational membrane attachment and dynamic fatty acylation of a neuronal growth cone protein, GAP-43. J. Cell Biol. 108 , 613-624.

Spicer, E., Suckert, C., Al-Attar, H., and Marsden, M. (2010). Integrin alpha5beta1 function is regulated by XGIPC/kermit2 mediated endocytosis during Xenopus laevis gastrulation. PLoS One 5 , e10665.

Srinivasan, S., Burckhardt, C.J., Bhave, M., Chen, Z., Chen, P.H., Wang, X., Danuser, G., and Schmid, S.L. (2018). A noncanonical role for dynamin-1 in regulating early stages of clathrin-mediated endocytosis in non-neuronal cells. PLoS Biol. 16 , e2005377.

Sun, Z., Costell, M., and F € assler, R. (2019). Integrin activation by talin, kindlin and mechanical forces. Nat. Cell Biol. 21 , 25-31.

Takahashi, T., Fournier, A., Nakamura, F., Wang, L.H., Murakami, Y., Kalb, R.G., Fujisawa, H., and Strittmatter, S.M. (1999). Plexin-neuropilin-1 complexes form functional semaphorin-3A receptors. Cell 99 , 59-69.

Tamagnone, L., Artigiani, S., Chen, H., He, Z., Ming, G.I., Song, H., Chedotal, A., Winberg, M.L., Goodman, C.S., Poo, M., et al. (1999). Plexins are a large family of receptors for transmembrane, secreted, and GPI-anchored semaphorins in vertebrates. Cell 99 , 71-80.

Tani, T.T., and Mercurio, A.M. (2001). PDZ interaction sites in integrin alpha subunits. T14853, TIP/GIPC binds to a type I recognition sequence in alpha 6A/alpha 5 and a novel sequence in alpha 6B. J. Biol. Chem. 276 , 36535-36542.

Taylor, M.J., Perrais, D., and Merrifield, C.J. (2011). A high precision survey of the molecular dynamics of Mammalian clathrin-mediated endocytosis. PLoS Biol. 9 , e1000604.

Teckchandani, A., Toida, N., Goodchild, J., Henderson, C., Watts, J., Wollscheid, B., and Cooper, J.A. (2009). Quantitative proteomics identifies a Dab2/integrin module regulating cell migration. J. Cell Biol. 186 , 99-111.

Tojima, T., Itofusa, R., and Kamiguchi, H. (2010). Asymmetric clathrin-mediated endocytosis drives repulsive growth cone guidance. Neuron 66 , 370-377.

Tomizawa, Y., Sekido, Y., Kondo, M., Gao, B., Yokota, J., Roche, J., Drabkin,

H., Lerman, M.I., Gazdar, A.F., and Minna, J.D. (2001). Inhibition of lung cancer cell growth and induction of apoptosis after reexpression of 3p21.3 candidate

tumor suppressor

13954-13959.

Tosoni, D., Puri, C., Confalonieri, S., Salcini, A.E., De Camilli, P., Tacchetti, C., and Di Fiore, P.P. (2005). TTP specifically regulates the internalization of the transferrin receptor. Cell 123 , 875-888.

gene

SEMA3B.

Proc.

Natl.

Acad.

Sci.

USA

98

,

<!-- image -->

Traub, L.M. (2009). Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat. Rev. Mol. Cell Biol. 10 , 583-596.

Tuladhar, R., Yeu, Y., Piazza, J.T., Tan, Z., Clemenceau, J.R., Wu, X., Barrett, Q., Herbert, J., Mathews, D.H., Kim, J., et al. (2019). CRISPR/Cas9-based mutagenesis frequently provokes on-target mRNA misregulation. bioRxiv https://www.biorxiv.org/content/10.1101/583138v1.full.

UniProt Consortium. (2019). UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res 47 , D506-D515.

Valdembri, D., Caswell, P.T., Anderson, K.I., Schwarz, J.P., Ko ¨ nig, I., Astanina, E., Caccavari, F., Norman, J.C., Humphries, M.J., Bussolino, F., and Serini, G. (2009). Neuropilin-1/GIPC1 signaling regulates alpha5beta1 integrin traffic and function in endothelial cells. PLoS Biol. 7 , e25.

Varsano, T., Dong, M.Q., Niesman, I., Gacula, H., Lou, X., Ma, T., Testa, J.R., Yates, J.R., and Farquhar, M.G. (2006). GIPC is recruited by APPL to peripheral TrkA endosomes and regulates TrkA trafficking and signaling. Mol. Cell. Biol. 26 , 8942-8952.

Vicente-Manzanares, M., Choi, C.K., and Horwitz, A.R. (2009). Integrins in cell migration-the actin connection. J. Cell Sci. 122 , 199-206.

Wang, B., Yang, H., Liu, Y.C., Jelinek, T., Zhang, L., Ruoslahti, E., and Fu, H. (1999). Isolation of high-affinity peptide antagonists of 14-3-3 proteins by phage display. Biochemistry 38 , 12499-12504.

Wang, Y., He, H., Srivastava, N., Vikarunnessa, S., Chen, Y.B., Jiang, J., Cowan, C.W., and Zhang, X. (2012). Plexins are GTPase-activating proteins for Rap and are activated by induced dimerization. Sci. Signal. 5 , ra6.

Wang, Y., Pascoe, H.G., Brautigam, C.A., He, H., and Zhang, X. (2013). Structural basis for activation and non-canonical catalysis of the Rap GTPase activating protein domain of plexin. eLife 2 , e01279.

Weksler, B.B., Subileau, E.A., Perrie ` re, N., Charneau, P., Holloway, K., Leveque, M., Tricoire-Leignel, H., Nicotra, A., Bourdoulous, S., Turowski, P., et al. (2005). Blood-brain barrier-specific properties of a human adult brain endothelial cell line. FASEB J. 19 , 1872-1874.

Yaffe, M.B. (2002). How do 14-3-3 proteins work?- gatekeeper phosphorylation and the molecular anvil hypothesis. FEBS Lett. 513 , 53-57.

Yaffe, M.B., Rittinger, K., Volinia, S., Caron, P.R., Aitken, A., Leffers, H., Gamblin, S.J., Smerdon, S.J., and Cantley, L.C. (1997). The structural basis for 14-3-3:phosphopeptide binding specificity. Cell 91 , 961-971.

Yip-Schneider, M.T., Miao, W., Lin, A., Barnard, D.S., Tzivion, G., and Marshall, M.S. (2000). Regulation of the Raf-1 kinase domain by phosphorylation and 14-3-3 association. Biochem. J. 351 , 151-159.

Zacharias, D.A., Violin, J.D., Newton, A.C., and Tsien, R.Y. (2002). Partitioning of lipid-modified monomeric GFPs into membrane microdomains of live cells. Science 296 , 913-916.

Zerial, M., and Mcbride, H. (2001). Rab proteins as membrane organizers. Nat. Rev. Mol. Cell Biol. 2 , 107-117.

<!-- image -->

## STAR + METHODS

## KEY RESOURCES TABLE

| REAGENT or RESOURCE                                          | SOURCE                    | IDENTIFIER                          |
|--------------------------------------------------------------|---------------------------|-------------------------------------|
| Antibodies                                                   |                           |                                     |
| Rabbit polyclonal anti-pan-Akt                               | Cell Signaling Technology | Cat #4691; RRID: AB_915783          |
| Rabbit polyclonal anti-phospho-Akt- Ser473                   | Cell Signaling Technology | Cat #4060; RRID: AB_2305337         |
| Rabbit polyclonal anti-phospho-Akt-Thr308                    | Cell Signaling Technology | Cat #2965; RRID: AB_10695743        |
| Mouse monoclonal anti green fluorescent protein (GFP)        | Roche                     | Cat #11814460001; RRID: AB_390913   |
| Rabbit polyclonal anti influenza virus hemaglutinin (HA)-tag | Sigma                     | Cat SAB4300603; RRID: AB_10620829   |
| Rabbit polyclonal anti paxillin                              | Abcam                     | Cat# ab32084; RRID: AB_779033       |
| Mouse monoclonal anti-active alpha-5 integrin                | Millipore                 | Cat# MABT201                        |
| Rabbit polyclonal anti-GIPC1                                 | Proteintech               | Cat# 14822-1-AP; RRID: AB_2263269   |
| Mouse monoclonal anti-SH3BP4                                 | Santa Cruz Biotechnology  | Cat# sc393730                       |
| Rabbit polyclonal anti- SH3BP4                               | Abcam                     | Cat# ab84853; RRID: AB_1861451      |
| Mouse monoclonal anti- neuropilin-1                          | BioLegend                 | Cat# 354502; RRID: AB_2564475       |
| Rabbit polyclonal anti- neuropilin-1                         | Abcam                     | Cat# ab81321; RRID: AB_1640739      |
| Mouse monoclonal anti-alpha-5 integrin                       | BioLegend                 | Cat# 328002; RRID: AB_893363        |
| Mouse monoclonal anti-alpha-5 integrin                       | Millipore                 | Cat# MABT822                        |
| Mouse monoclonal anti- 14-3-3 epsilon                        | Santa Cruz Biotechnology  | Cat# sc23957; RRID: AB_626619       |
| Rabbit polyclonal anti- Pten                                 | Cell Signaling Technology | Cat# 9188; RRID: AB_2253290         |
| Rabbit polyclonal anti-E-cadherin                            | Cell Signaling Technology | Cat# 3195; RRID: AB_2291471         |
| Rabbit polyclonal anti- Gapdh                                | Cell Signaling Technology | Cat# 5174; RRID: AB_10622025        |
| Rabbit polyclonal anti- Vimentin                             | Santa Cruz Biotechnology  | Cat# sc7557; RRID: AB_793998        |
| Mouse monoclonal anti-actin                                  | Sigma                     | Cat# A1978; RRID: AB_476692         |
| Rabbit polyclonal anti- myosin-6                             | Santa Cruz Biotechnology  | Cat# sc50461; RRID: AB_2148617      |
| Rabbit polyclonal anti- ARHGEF12/Larg                        | Santa Cruz Biotechnology  | Cat# sc25638; RRID: AB_2059903      |
| Rabbit polyclonal anti- alpha-5 integrin                     | Abcam                     | Cat# ab150361; RRID: AB_2631309     |
| Goat polyclonal anti- dynamin-2                              | Santa Cruz Biotechnology  | Cat# sc6400; RRID: AB_639943        |
| Rabbit polyclonal anti- Eps15                                | Schmid lab, UTSW          | NA                                  |
| Mouse monoclonal anti- clathrin heavy chain                  | BioXCell                  | Cat# 5262/0514                      |
| Rabbit polyclonal anti- fibronectin                          | Abcam                     | Cat# ab2413; RRID: AB_2262874       |
| Mouse monoclonal anti- vimentin                              | Sigma                     | Cat# V6630; RRID: AB_477627         |
| Rabbit polyclonal anti- semaphorin-3A                        | Abcam                     | Cat# ab23393; RRID: AB_447408       |
| Mouse monoclonal anti- penta-HIS tag                         | Qiagen                    | Cat# 34660; RRID: AB_2619735        |
| Rabbit polyclonal anti- SNAP tag                             | New England Biolabs       | Cat# P9319                          |
| Mouse monoclonal anti-mouse- AlexaFluor-647                  | Thermo Scientific         | Cat# A31571; RRID: AB_162542        |
| Mouse monoclonal anti-rabbit- AlexaFluor-568                 | Thermo Scientific         | Cat# A10042; RRID: AB_2534017       |
| Goat polyclonal anti-mouse-HRP                               | Jackson ImmunoResearch    | Cat# 115-035-003; RRID: AB_10015289 |
| Goat polyclonal anti-rabbit-HRP                              | Jackson ImmunoResearch    | Cat# 111-035-003; RRID: AB_2313567  |
| Donkey polyclonal anti-goat-HRP                              | Abcam                     | Cat# ab97110; RRID: AB_10679463     |
| Additional information on antibodies is listed in Table S9   |                           | N/A                                 |

( Continued on next page )

Article

## Article

| Continued                                       |                                                         | IDENTIFIER                                 |
|-------------------------------------------------|---------------------------------------------------------|--------------------------------------------|
| Bacterial and virus strains                     |                                                         |                                            |
| pAd-tet SP-HIS6x-Semaphorin-3a, R551A, R555A    | This work                                               | N/A                                        |
| Chemicals, peptides, and recombinant proteins   |                                                         |                                            |
| Purified HIS6x-Semaphorin-3a, R551A, R555A      | This work                                               | N/A                                        |
| Fibronectin                                     | Sigma                                                   | Cat# F1141                                 |
| Critical commercial assays                      |                                                         |                                            |
| RNAeasy mini kit                                | Qiagen                                                  | Cat# 74106                                 |
| DNA extraction kit                              | IBI Scientific                                          | Cat# IB47061                               |
| e-Myco Kit                                      | Boca Scientific                                         | Cat# 25235                                 |
| EndoGRO-MV Media Kit                            | Millipore                                               | Cat# SCME004                               |
| BCA assay kit                                   | Thermo Scientific                                       | Cat# 23225                                 |
| Experimental models: cell lines                 |                                                         |                                            |
| Human retinal pigment epithelium: hTERT RPE     | ATCC                                                    | Cat# CRL-4000, RRID: CVCL_4388             |
| Human retinal pigment epithelium: ARPE-19       | ATCC                                                    | Cat# CRL-2502, RRID: CVCL_6338             |
| Human brain microvascular endothelial: hCMEC/D3 | Millipore                                               | Cat# SCC066, RRID: CVCL_U985               |
| Human dermal microvascular endothelial: HMEC1   | ATCC                                                    | Cat# CRL-3243, RRID: CVCL_0307             |
| Human embryonic kidney: HEK Lenti- X 293T       | Takara                                                  | Cat# 632180                                |
| Experimental models: organisms/strains          |                                                         |                                            |
| hTERT RPE SH3BP4 KO cells                       | This work                                               | NA                                         |
| hTERT RPE GIPC1 KO cells                        | This work                                               | NA                                         |
| hTERT RPE NRP1 KO cells                         | This work                                               | NA                                         |
| Oligonucleotides                                |                                                         |                                            |
| siRNA CLTC SASI_Hs01_00208103                   | Sigma                                                   | Cat# 3014005638-000020                     |
| siRNA CLTC SASI_Hs01_00208103_AS                | Sigma                                                   | Cat# 3014005638-000030                     |
| siRNA CLTC SASI_Hs_02_00338467                  | Sigma                                                   | Cat# 3014005638-000050                     |
| siRNA CLTC SASI_Hs_02_00338467_AS               | Sigma                                                   | Cat# 3014005638-000060                     |
| siRNA is listed in Table S10                    |                                                         | N/A                                        |
| Oligonucleotides are listed in Table S7         |                                                         | N/A                                        |
| Recombinant DNA                                 |                                                         |                                            |
| pLVX puro SH3BP4-GFP wt                         | This work                                               | N/A                                        |
| pLVX puro SH3BP4-GFP S246A                      | This work                                               | N/A                                        |
| pLVX puro SH3BP4-GFP dCME                       | This work                                               | N/A                                        |
| pLVX puro SH3BP4-GFP dCME/S246A                 | This work                                               | N/A                                        |
| pSpCas9(BB)-2A-GFP (PX458)                      | Ran et al., 2013.                                       | Addgene ID: 48138, RRID: Addgene_48138     |
| cDNA is listed in Table S8                      |                                                         | N/A                                        |
| Software and algorithms                         |                                                         |                                            |
| cmeAnalysis                                     | Danuser lab, UT Southwestern Medical Center, Dallas, TX | https://github.com/DanuserLab/ cmeAnalysis |
| Matlab                                          | The Mathworks, Inc                                      | RRID:SCR_001622                            |
| Fiji                                            | NIH                                                     | RRID:SCR_002285                            |
| Squash package for ImageJ/Fiji                  | Sbalzarini lab, Center for Systems Biology              | http://mosaic.mpi-cbg.de/?q=downloads/     |
|                                                 | Dresden, Dreseden, Germany                              | imageJ                                     |

(

Continued on next page

)

<!-- image -->

<!-- image -->

| Continued                         |                                                          |                                                |
|-----------------------------------|----------------------------------------------------------|------------------------------------------------|
| REAGENT or RESOURCE CRISPR design | SOURCE Crispinator.com, UT Southwestern Medical          | IDENTIFIER http://crispinator.com/             |
| Expasy                            | Swiss Institute of Bioinformatics, Lausanne, Switzerland | https://www.expasy.org/                        |
| Lung Cancer Explorer              | UT Southwestern Medical Center,                          | http://lce.biohpc.swmed.edu/lungcancer/        |
| Uniprot                           | Dallas, TX                                               |                                                |
| Phosphosite                       |                                                          | https://www.uniprot.org/                       |
| plus                              | Cell Signaling Technologies                              | https://www.phosphosite.org/homeAction. action |
| The Biogrid 4.2                   |                                                          | https://thebiogrid.org/                        |

## RESOURCE AVAILABILITY

## Lead contact

Requests for reagents, data, and software should be directed to and will be fulfilled by the lead contact, Gaudenz Danuser (Gaudenz. Danuser@utsouthwestern.edu)

## Materials availability

Materials are available upon reasonable request.

## Data and code availability

Data is available upon reasonable request. The cmeAnalysis software was previously deposited on Github (https://github.com/ DanuserLab/cmeAnalysis).

## EXPERIMENTAL MODEL AND SUBJECT DETAILS

## Cells and culture

The human retinal pigment epithelial cells (RPE) immortalized with human Telomerase (hTERT) (Jiang et al., 1999) were obtained from American Type Culture Collection (hTERT RPE, sex: female, ATCC, Manassas, VA, CRL-4000) and grown in Dulbecco's Modified Eagle Medium DMEM/F-12 (1:1, Gibco, 11330-032), 10 % fetal bovine serum (FBS, Sigma F0926) and antibiotic-antimycotic (AA, Gibco, containing 10,000 units/mL penicillin, 10,000 m g/mL streptomycin 25 m g/mL amphotericin B, 15240062). Human retinal pigment epithelium ARPE-19/HPV-16 cells were obtained from ATCC (CRL-2502, sex: male). ARPE-19/HPV-16 cells are ARPE19 cells transformed with human papillomavirus-16 DNA by transfection (ATCC). The parental ARPE-19 cell line arose spontaneously in culture (Dunn et al., 1996). Cells were maintained in DMEM/F-12, 10 % FBS and AA. Human brain microvascular endothelial cells hCMEC/D3,that were originally immortalized with hTERT and SV40 large T antigen (Weksler et al., 2005) were obtained from Millipore (Millipore, SCC066, sex: female). Cells were grown in EndoGRO-MV Media Kit (Millipore, SCME004) according to manufacturer's instructions. Human embryonic kidney (HEK) cells Lenti-X 293T (Takara, 632180) were grown in DMEM (Gibco, 11995-065), supplemented with 10 % FBS and AA. Human dermal microvascular endothelial cells HMEC1 were obtained from ATCC (CRL-3243, sex: male) and grown in MCDB 131 medium (Gibco, 10372019) supplemented with microvascular growth supplement (Gibco, S00525). The non-small cell lung cancer (NSCLC) cell A549 (sex: male), H1299 (male), H2009 (female), H157 (male), HCC193 (female), HCC44 (female), H1473 (NA), H1819 (female) and H2052 (male) were obtained from the Hamon Cancer Center Collection (UT Southwestern Medical Center) and described previously (Gazdar et al., 2010). Cell line identity was confirmed by DNA fingerprinting (PowerPlex 1.2 Kit, Promega) and mycoplasma-free status was verified by PCR (e-Myco Kit, Boca Scientific, 25235). Cells were grown in Roswell Park Memorial Institute (RPMI) 1640 medium (Gibco, 11875-093) supplemented with 5 % FBS and AA. All cells were cultured in humidified incubators (Nuaire) with 5 % CO2 at 37 /C14 Celsius (37 /C14 C). For starvation experiments, cells were cultured in serum free and low glucose DMEM (Gibco, 11885-084) with 1 g/L D-glucose and AA. For passaging, cells were washed in phosphate buffered saline (PBS), pH 7.4, composed of 137 mM NaCl (Research Products International, RPI, S23020), 2.7 mM KCl (Fisher Scientific, BP366), 10 mM Na2HPO4 (RPI, S23100), 1.8 mM KH2PO4 (Fisher Scientific, BP362), incubated in 0.5 % trypsin-EDTA solution (GIBCO, 15400-054) until cells detached and resuspended in full medium. Cell concentrations were determined using an automated cell counter (Cellometer Auto 1000, Nexcelom, Lawrence, MA).

## METHOD DETAILS

## DNA constructs

For expression of the SH3BP4-GFP fusion constructs, full length proteins, and truncated proteins, human SH3BP4 cDNA (Thermo Scientific, pCMV-SPORT6 clone ID 6138465) was amplified using forward and reverse priming DNA oligonucleotides (IDT DNA,

## Article

Coralville, IA, for all oligonucleotides see Table S6) with EcoRI and KpnI restriction sites by polymerase chain reaction (PCR) using PFUultra II Fusion HS DNA polymerase (Agilent, Santa Clara, CA, 600674-51 ), dNTP mix (BioBasic, DD0057) in a thermal cycler (BioRad, Hercules, CA, C1000 Touch, 1851148). PCR products were purified on 1 % agarose gels (Fisher Scientific, BP160) in Tris-acetic acid-EDTA (TAE) buffer containing 40 mM Tris (Fisher Scientific, BP152), 20 mM glacial acetic acid (Fisher, BP2401SI-212) and 1 mM Ethylenediaminetetraacetic acid, di-Sodium salt dihydrate (EDTA, Fisher Scientific, BP120) with ethidium bromide (Bio-Rad, 1610433) by electrophoresis (Bio-Rad, 1704406). Gels were imaged on a GBOX imaging system with a Synoptics 4.2 MP camera and Genesys software (all Syngene, Frederick, MD). DNA bands were excised and solubilized in PB/QG buffer composed of 3M guaninidine-thiocyanate (Sigma, G9277), 10 mM Tris-HCl, pH 6.6, 5 % ethanol (Pharmaco, 111000190). DNA was purified on mini spin columns (Sydlab, Natick, MA, MB082) and eluted in ultrapure water (Millipore Synergy UV, Biopak filter CDUFBI001). The pEGFP-N1 vector (Clontech, Mountain View, CA, 6085-1) was first modified at position alanine-206 of EGFP by site directed mutagenesis (as described below, for oligonucleotides see Table S6) to introduce an alanine to lysine (A206K) mutation to reduce GFP dimerization (Zacharias et al., 2002). Plasmid and PCR DNA were digested with EcoRI-HF (New England Biolabs, NEB Ipswich, MA, R3101) and KpnI-HF (NEB, R3142) in cutsmart buffer (NEB, B7204) at 37 /C14 C according to manufacturer protocols. Plasmid DNA was dephosphorylated with calf intestine phosphatase (CIP, NEB, M0290) at 37 /C14 C and purified on a 1 % agarose gel as described above. Plasmid and PCR DNA were ligated with T4 ligase (NEB, M0202) at room temperature for 10 min followed by prolonged incubation at 16 /C14 Cin the thermal cycler. Ligation products were transformed into XL1- blue E.coli cells (Agilent, 200236) by a 10 min incubation on ice, followed by 90 seconds heat shock at 42 /C14 C in a water bath (Thermo Scientific) before cells were chilled again on ice. Cells were grown out in Luria-Bertani broth (LB broth Miller, Millipore, 71753-5) medium for 30 min at 37 /C14 C, spun down, resuspended in LB medium and plated on LB-agarose plates containing 10 m g/ml Kanamycin sulfate (Fisher Scientific, BP906-5) for selection. The next day, colonies were collected and grown in LB medium in the presence of 10 m g/ml Kanamycin. Cultures were harvested and plasmid DNAwasextracted using standard DNA extraction protocols and spin columns (Sydlab). The DNA was analyzed by restriction digest and constructs containing the insert of the expected size were sequenced (Genewiz, South Plainfield, NJ). To introduce amino acid changes in the pEGFP-N1 SH3BP4-GFP construct at motifs relevant for clathrin mediated endocytosis and at phosphorylation sites, standard site directed mutagenesis protocols were applied, using PFU Ultra II DNA polymerase and oligonucleotides containing the respective altered nucleotides (IDT DNA, for oligonucleotides see Table S6). Briefly, oligonucleotides were annealed in a temperature gradient ranging from 55-68 /C14 C, DNA was amplified at 68 /C14 Cfor one min per kilobase of DNA, for 18 cycles with a final extension at 72 /C14 Cfor up to 20 min. The input plasmid DNA was digested with DpnI (NEB, R0176) for 3 h at 37 /C14 Cin the thermal cycler. PCR products were purified over spin columns in PB/QG buffer and transformed into XL1-blue cells and processed as described above. The resulting constructs were sequenced (Genewiz). To obtain constructs with multiple mutations, nucleotide changes were introduced sequentially. The same procedure was applied to generate the GFP-GIPC1 LGL/AAA PDZ domain mutant construct and the GFP-PTEN G251C mutant construct (for oligonucleotides see Table S6).

For transient expression experiments in HEK cells, including all immunoprecipitation experiments, GFP fusion constructs were established in pEGFP vectors (Clontech). For the GFP-GIPC1 expression constructs, the cDNA for human GIPC1 (PlasmidID Harvard, HsCD00040225) was inserted into pEGFP-C1 (Clontech) using XhoI and EcoRI (NEB). For pEGFP-N1 Dyn2-GFP human Dyn2 (Srinivasan et al., 2018) was inserted using EcoRI and KpnI restriction sites. For pEGFP-N1 CLTC-TD-GFP, human CLTC terminal domain (TD) the N-terminal 579 amino acids of human CLTC (plasmidID Harvard: HsCD00339048) were amplified by PCR and subcloned with XhoI and HindIII (NEB, R0104). For pEGFP-C1 NRP1 cytoplasmic tail constructs the 44 C-terminal amino acid cytoplasmic portion of human NRP1 (Thermo, Ultimate ORF, clone IOH81223) were inserted using XhoI and EcoRI (both NEB) in frame with GFP. For the dSEA mutant, the last three amino acids were omitted. For all 14-3-3-GFP isoform fusion constructs PCR products were subcloned into pEGFP-N1 as described above (for cDNA see Table S7, for oligonucleotides and restriction enzymes see Table S6).

For biochemistry experiments FLAG-tag and influenza virus hemagglutinin (HA) tag were first introduced in pcDNA 3.1 vector. In brief, two overlapping oligonucleotides were annealed and expanded using PFU Ultra DNA polymerase (for Oligonucleotides see supplementary Table S6), agarose gel purified and extracted (small DNA extraction kit, IBI Scientific, IB47061), digested with HindIII and KpnI (NEB) and ligated into pcDNA 3.1 as described above. Next, cDNAs (all human) for SH3BP4, GIPC1, GIPC2 (Thermo Ultimate ORF lite, IOH27252), EPS15 (Thermo, IOH46773), YWHAE (PlasmidID Harvard, HsCD0005020) were inserted in frame with FLAG-HA tags into the pcDNA 3.1 FLAG-HA construct as described above, for oligonucleotides and restriction enzymes, see Table S6. For human GIPC3 continuous cDNA could not be obtained, two independent clones (Dharmacon, MHS6278-211687736, clone ID: 40125863) exhibited an approximately 20 bp deletion in the GC rich 5 0 part. Instead, for the 5' 240 bp region, single-stranded DNA designed with reduced GC content in the 5 0 114 bp and 30 bp for DNA assembly was ordered (gBlock, IDT DNA, see Table S6). The remaining part was amplified with 30 bp overhangs and assembled by homologous recombination in yeast as described below and the resulting construct was verified by Sanger sequencing (Genewiz). For transient overexpression of Akt1-wt-HA, DNA was amplified (DNA from 896 pcDNA3 T7 Akt1 was a gift from William Sellers, Addgene plasmid # 9003; http://n2t.net/addgene:9003; RRID:Addgene\_9003) with a reverse primer to insert a HA tag and subcloned into pcDNA 3.1 using EcoRI and XhoI (NEB) as described above. To establish stable expression of SH3BP4-GFP-fusion in human cells, constructs were subcloned into the pLVX lentiviral vector (Clontech, neo: Cat# 632181, puro: 632183) where expression is driven by human cytomegalovirus immediate early promotor (hCMV) and neomycin or puromycin resistance genes follow after an internal ribosomal entry site. For subcloning SnaBI (NEB, R0130) and NotI-HF (NEB, R3189) were used. The same approach was used for Dyn2-GFP, Eps15-GFP, Eps15SNAP, GFP-NRP1, cytosolic GFP, GAP-GFP, GFP-PTEN, SNAP-PTEN and constitutively active Akt1-SNAP expression. To assemble the SNAP-PTEN and Eps15-SNAP constructs, the SNAP-tag was amplified from pSNAP-f (NEB, N9183S). For the TIRF

<!-- image -->

## ll

## Article

clathrin coated pit localization assay a membrane targeted GFP was designed as a negative control. To localize GFP to the plasma membrane the first 20 amino acids of neuroglobulin/GAP43 were introduced into pEGFP-N1 in frame with GFP. The N-terminal peptide was previously found sufficient for membrane and Golgi targeting (Liu et al., 1994). The peptide is palmitoylated at Cys3 and Cys4 and anchored to the lipid bilayer (Skene and Virag, 1989). For rescue experiments in the RPE GIPC1 knockout cells an N-terminally 10x Histidine tagged- (HIS)-GIPC1 was generated by using oligonucleotides encoding the His-tag for PCR and subcloning the product into pLVX puro with EcoRI and XbaI (NEB). To rescue NRP1 function in the RPE NRP1 KO cells, a lentiviral GFP-NRP1 full-length fusion constructs was assembled in yeast by homologous recombination as described below. The NRP1 N-terminal 21 amino acid signal peptide was followed by GFP, which in turn was spaced from NRP1 by a six amino acid glycine-serine linker (GGSGGS). The three pieces (for oligonucleotides see Table S6) were assembled in pRS424 shuttle vector by homologous recombination in yeast as described below. The resulting fusion piece was transferred to pLVX puro lentiviral vector for stable expression. After infection, RPE cells were flow sorted and analyzed by western blotting (see below). To enhance Akt signaling a lentiviral expression construct was made in two steps, first myr-dPH-Akt1, a membrane targeted constitutively active version was amplified by PCR (1017 pcDNA3 Myr Akt1 deltaPH was a gift from William Sellers, Addgene plasmid # 9009; http://n2t.net/addgene:9009; RRID:Addgene\_9009) and introduced into pEGFP-N1. Next, the GFP fusion construct was subcloned into pLVX-puro as described above. For pEGFP-N1 GIPC1GFP constructs, EcoRI and BamHI sites were used. For expression of the GFP-PTEN fusion, the PTEN cDNA (Thermo, IOH56926) was amplified using primers with XhoI and BamHI restriction sites and inserted into pEGFP-C1 as described above. Subsequently, the GFP-PTEN insert was subcloned into pLVX puro. For expression of recombinant GST-SH3BP4-HIS8x in Ecoli, where SH3BP4 is GST-tagged at the N-term and contains an 8x histidine tag at the C-term, oligonucleotides introducing restriction sites and His-tag were used and the PCR product was introduced into pGEX-6P-1 (GE Healthcare) with EcoRI and NotI (NEB). The analogous strategy was employed to generate a pGEX-6P-1 GST-GIPC1-HIS8x expression vector.

The constructs for GIPC3, SP-HIS6x-SEMA3A, Eps15-SNAP, constitutively active Akt1-SNAP, SNAP-PTEN and the SH3BP4 R18 mutant constructs were assembled by homologous recombination in yeast S.cerevisiae. To this end, oligonucleotides with 30 bp overhangs were designed and DNA fragments were amplified by PCR and purified on agarose gels as described above. The pRS424 plasmid DNA containing the TRP1 marker (Christianson et al., 1992) was digested with KpnI-HF, NotI-HF and SmaI (NEB, R0141) for linearization and purified by agarose gel electrophoresis and gel extraction. For transformation, linear pRS424 and the DNA fragments were mixed with 260 m l Poly-ethylene glycol (PEG, average Mn 4000, Sigma, 81240, 50 % w/v in sterile water), 36 m l 1 M Lithium-acetate (in water, Sigma 517992) salmon sperm DNA carrier (Invitrogen, 15632-011) and competent yeast YPH500 (Cermak et al., 2011), that were prepared according to established protocols (Gietz and Schiestl, 2007). Cells were incubated for 22 min at 42 /C14 C in the water bath, spun down, and plated on tryptophan deficient yeast synthetic drop out plates. Synthetic drop out plates were prepared from 6.7 g Yeast Nitrogen Base (w/o amino acids, Difco, Becton Dickinson, 291940), 2 g of amino acid mix without Tryptophan (manually mixed, all amino acids were obtained from Sigma), 15 g agar (bacteriological type A, Affymetrix, 10906), 2 % D-glucose (Dextrose, Fisher Scientific, D16). Plates were incubated at 30 /C14 C until colonies formed. To obtain plasmid DNA, cells were harvested from plates, digested with zymolase and extracted according to manufacturer's instructions (Zymoprep Yeast Plasmid Miniprep II, Zymo Research, D2004). The eluted plasmid DNA was transformed into XL1-blue competent cells (NEB) and grown on ampicillin (100 m g/ml, Sigma, A-9518) selective LB-agar plates. Colonies were expanded, plasmid DNA was extracted and analyzed by restriction enzyme digestion and Sanger sequencing as described above. To obtain an SH3BP4 mutant that mimics S246 phosphorylation and 14-3-3 binding, the 20 amino acid long 14-3-3 binding peptide R18 (Wang et al., 1999) was introduced upstream of the last NPF motif and the S246 phosphorylation site. It replaced an un-conserved region of SH3BP4 with considerable amino acid sequence heterogeneity. As a negative control the double lysine mutant version of the R18 peptide was used (Wang et al., 1999).

## mRNA extraction, cDNA generation and sequencing

One million H157, HCC193 and H1819 NSCLC cells were harvested by trypsinization and mRNA was extracted using the RNAeasy mini kit (Qiagen, 74106) following the manufacturer's instructions. The cDNA was generated using SuperScript IV (Thermo Scientific, 18091050). In brief, 1 m g mRNA, oligo-d(T)18 primer (NEB, S1316S) and dNTPs (Bio Basic) were heated to 65 /C14 C for 5 min and annealed on ice for 3 min. SuperScript IV master mix was added and the reverse transcriptase reaction was run for 10 min at 52C before the enzyme was inactivated at 80 /C14 C for 10 min. The PTEN cDNA was amplified by PCR yielding a 1.2 kb product (Figure S8C, for oligonucleotides see Table S6) on a 1 % agarose gel. The band was extracted and purified as described above and processed for Sanger sequencing (Genewiz, for oligonucleotides used for PTEN sequencing see Table S6). For H157 the G251C mutation was confirmed while for HCC193 and H1819 cells no mutations were found in PTEN.

## siRNA knockdown of clathrin heavy chain

To assess the effect of clathrin mediated endocytosis on neuropilin-1 and a 5-integrin endocytosis a knockdown of clathrin heavy chain was performed. To this end 120 K wild type RPE cells were seeded in 6 well plates. The following day cells were transfected with 30 pmol each of two siRNAs targeting clathrin heavy chain (CHC, Sigma, see Table S8 for sequences) or equivalent amounts of AllStars negative control siRNA (Qiagen, 1027281) using Lipofectamine RNAiMAX (Thermo, 13778-150) in Opti-MEM medium (Gibco, 31985-070) as recommended by the manufacturer. The transfection was repeated on day 2. On day 3, 10 K cells were seeded for the

Article uptake and SEMA3A assays and starved overnight in low glucose serum free medium as described below. The assays were performed on day four post seeding as described below. Cells were harvested for knockdown analysis by western blotting as described below.

## CRISPR/Cas9 knockout cells

For CRISPR/Cas9 experiments, 200-500 bp of the first or second exon were used to find optimal guide-RNA targeting sequences on the web site of the Zhang lab at Massachusetts Institute of Technology (MIT, www.crispr.mit.edu (Ran et al., 2013)) or optimal targeting sequences were retrieved from the Lum lab at UT Southwestern Medical Center website (www.crispinator.com (Tuladhar et al., 2019)). Oligonucleotides (IDT DNA) with overhangs compatible with BbsI restriction enzyme were designed for insertion into pX458 pSpCas9(BB)-2A-GFP (Cas9 wt) or pSpCAS9n(BB)-2A-puro (Cas9 nickase, Addgene # 48138, #48139, respectively, the plasmids were a gift from Feng Zhang, MIT), as described in detail (Ran et al., 2013) (for oligonucleotide see Table S6). In brief, forward and reverse oligonucleotides were mixed and phosphorylated with T4 polynucleotide kinase (T4 PNK, M0201, NEB) for 30 min at 37 /C14 C in T4 ligase buffer (NEB, M0202). For duplex formation, the reaction mix was heated to 95 /C14 C for 5 min and slowly cooled downto room temperature in the thermal cycler (Bio-Rad). The pSpCas9 vectors were digested with BbsI-HF (NEB, R3539), dephosphorylated with CIP, purified on a 1 % agarose gel and extracted from the gel using extraction procedures and spin columns as described above. The oligonucleotide duplexes and digested pSpCas9 were ligated with T4 DNA ligase as described and unligated linear DNA was digested by addition of Plasmid Safe ATP dependent Nuclease (Epicentre, E3101K), 0.5 mM ATP (Biobasic, AB0311) and 1x plasmid safe nuclease buffer (Epicentre) for 30 min at 37 /C14 Cand heat inactivated at 70 /C14 Cfor 10 min. Reaction products were transformed into XL1-blue as described above. Resulting clones were analyzed by Sanger sequencing (Genewiz). Plasmid DNA was prepared using silica-based spin columns (Econo-Spin, Epoch Lifescience, 2040) according to standard protocols. For gene targeting, cells were transfected by electroporation using the NEON system (Thermo Scientific, MPK5000). Cells were washed once with phosphate buffered saline (PBS), incubated with 0.5 % trypsin-EDTA solution (GIBCO, 15400-054) until they detached, collected in culture medium and harvested by centrifugation. Cells were washed once with PBS and resuspended at 5*10 6 cells/ml. To each 100 m l of cell suspension, 8 m g Cas9-wt plasmid DNA was added. For Cas9-nickase, the cell suspension was mixed with 0.1 m g/ 100 m l pEGFP-C1 (Clontech) plasmid as a transfection marker. Sequentially, multiple volumes of 100 m l of cell suspension were pulsed twice at 1400 V for 20 ms and resuspended in culture medium. The next day, the medium was refreshed, and cells were expanded for fluorescence activated cell sorting (FACS). For single cell sorting cells were detached with trypsin, harvested in culture medium, washed once with PBS, resuspended in PBS and cell aggregates were removed by subjecting the suspension to a 40 m m nylon cell strainer (Falcon, 352340). For single cell sorting, 96 well plates (Perkin Elmer, Waltham, MA, Spectra Plate-96 TC, 6005650) were coated with gelatin solution (Attachment Factor, Gibco, S-006-100) for 30 min at 37 /C14 C. Next, single GFP positive cells were sorted into 96 well plates on a FACS Aria II SORP flow cytometer (BD Biosciences, UT Southwestern Children's Medical Center Research Institute Moody Foundation Flow Cytometry facility) using the 488 nm laser settings. Cell colonies were inspected on an inverted microscope (Nikon, Eclipse TS100) with a 4x objective (Nikon, Plan Fluor 4x, NA 0.13) and expanded. Protein expression was assessed by western blotting as described below.

## Lentiviral expression

In RPE SH3BP4 knockout cells, expression was reconstituted by SH3BP4-GFP fusion proteins. To establish stable expression of SH3BP4-GFP wt and mutant fusion constructs and expression of Eps15-GFP, Dyn2-GFP, GFP-NRP1 fluorescent reporter constructs in human cells a lentiviral transduction system was used. To this end, pLVX based constructs were transfected into HEK 293Tcells together with packaging vectors psPAX2 (HIV-1 gag and pol) and pMD2.G (VSV-G, both plasmids obtained from Addgene, #12260 and #12259, respectively, plasmids were a gift from Didier Trono, EPFL Lausanne, Switzerland). For HEK cell transfection, standard polyethylenimine (PEI, linear, MW 25 K, Polysciences, 23966) protocols were used. In brief, 30 m l PEI (1 mg/ml in H2O, pH 7.0) and 10 m g plasmid DNA each in 0.5 ml serum free DMEM medium were mixed and incubated for 15 min at room temperature before the transfection mix was added to the cells. Virus was harvested two days post transfection and filtered through 0.45 m m mixed cellulose esters membrane syringe filters (Fisher Scientific, MCE, 09-720-005,) and added to the cells. The next day the medium was exchanged, and cells were selected with puromycin dihydrochloride (Gibco, A1113802) or Geneticin (G418 sulfate, Gibco, 10131-035) or left unselected. After a couple of days of expansion, fluorescent cells were sorted on a FACS Aria II SORP flow cytometer equipped with a 488 nm and 633 nm laser (BD, Franklin Lakes, NJ) and expanded for analysis. Cells were further characterized by western blotting and microscopy as described below.

## Expression and purification of Semaphorin-3a

For expression and purification of active human semaphorin-3a protein (SEMA3A, cDNA from Dharmacon, 40034963), a histidine tag (HIS6x) was introduced at the N-term of the mature protein in frame with the signal peptide. To obtain higher yields of uncleaved protein a furin protease cleavage site was removed by introducing the R551A, R555A mutations (RRAA) (Adams et al., 1997) with overlapping primers by DNA assembly in yeast as described below (for oligonucleotides see Table S6). The His-tag semaphorin-3a fusion was subcloned into the shuttle vector pAd-tet-T3T7 using HindIII (NEB, R0104) and NotI-HF. The pAd-tet-T3T7 is a modification of pAdlox (Hardy et al., 1997) where the CMV promoter was replaced with a tetracycline responsive promoter as described previously (Altschuler et al., 1998). It contains a J packaging sequence, a single loxP site for cre recombination, a multiple cloning site followed by a poly-adenylation signal. Helper Adenovirus DNA was purified as described (Hardy et al., 1997). HEK 293 Cre-4 (Merck) that

<!-- image -->

## ll

Article express Cre-recombinase, were transfected with purified helper adenovirus DNA and pAd-tet-T3T7-HIS6x-SEMA3A plasmid using established calcium-phosphate protocols. After cytopathic effects appeared, virus was extracted using three freeze-thaw cycles and passaged one more round on Cre-4 cells followed by a third passage on regular HEK 293T cells to achieve high titer virus (P3 virus)(Hardy et al., 1997). Virus was extracted by three freeze thaw cycles, cell debris was spun down and supernatant was harvested, snap frozen in liquid nitrogen and stored at -80 /C14 C.

## Western blotting and antibodies

For sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) 1 mm thick gels were manually casted using the miniprotean system (Bio-Rad, 1658001FC) using standard protocols. In brief, the separation gel contained 375 mM Tris-HCl pH 8.8, 10 % acrylamide (from 40 % Acrylamide/Bis Solution, 37.5:1, Bio-Rad, 1610148), 0.1 % SDS (Thermo Scientific, 28365), and polymerization was catalyzed with tetramethylethylenediamine (TEMED, Bio-Rad, 161-0800) and ammonium persulfate (Thermo Scientific, 17874). Stacking gels were made of 5 % acrylamide, 125 mM Tris-HCl pH 6.8, and were polymerized as for the separation gels. Gels contained 10 or 15 wells, depending on the number of samples. Samples were run at 100 V for at least 90 min or until appropriate protein separation was observed (Power Pac Basic, Bio-Rad, 1645050). Gels were run in electrode buffer that contained 25 mM Tris, 190 mM glycine (Fisher Scientific, BP381-5) and 0.1 % SDS. Proteins were transferred to methanol (Pharmco-Aaper, 339000000CSGL) activated PVDF membranes (0.45 m m, Thermo Scientific, 88518), packed between Whatman paper (3MM-CHR GE Healthcare, 3030-917) in transfer buffer containing 25 mM Tris, 192 mM glycine (Fisher Scientific, BP381) and 10 % ethanol for 1 h at 400 mAmp in 4 /C14 C. Subsequently, membranes were blocked in Tris buffered saline with tween (TBST) composed of 150 mM NaCl, 25 mM Tris, 2.7 mM KCl, adjusted to pH7.4, 0.05 % Tween-20 (Fisher Scientific, BP337), containing 10 % milk. For protein detection, antibodies specific to SH3BP4 (mouse, Santa Cruz Biotechnology (SCBT), sc393730), SH3BP4 (mouse, 2992C1a, Abcam, ab84853), GIPC1 (rabbit, Proteintech, 14822-1-AP), neuropilin-1 (rabbit, Abcam, ab81321), 14-3-3ε (mouse, 8C3, SCBT, sc23957), myosin-6 (rabbit, SCBT,H-215, sc50461), ARHGEF12/Larg (rabbit, H-70, SCBT, sc25638), a 5-integrin (rabbit, EPR7854, Abcam, ab150361), Akt (pan, rabbit, C67E7, Cell Signaling Technology (CST), 4691), phospho-Akt Ser473 (pS473, rabbit, D9E, CST, 4060), phospho-Akt Thr308 (rabbit, C31E5E, CST, 2965), actin (mouse, Sigma, AC-15, A1978), dynamin-2 (goat, SCBT, sc6400), Eps15 (rabbit, Schmid laboratory, UTSW), vimentin (mouse, V9, Sigma, V6630), vimentin (rabbit, SCBT, C-20, sc7557), SNAP tag (rabbit, NEB, P9310) semaphorin-3a (rabbit, Abcam, ab23393), green fluorescent protein, GFP (mouse, Roche, 11814460001), influenza virus hemagglutinin HA-tag (rabbit, Sigma, SAB4300603), penta-Histidine tag (mouse, Qiagen, 34660), PTEN (rabbit, CST, D4.3, 9188), E-cadherin (rabbit, CST, 24E10, 3195), GAPDH (rabbit, CST, D16H11, 5174), clathrin heavy chain (mouse, manufactured by BioXCell, Lebanon, NH, clone TD.1) were used (see Table S6 for all antibodies used and corresponding research resource ID, RRID). Membranes were incubated with antibodies in TBST containing 5 % milk or 5 % bovine serum albumin (BSA, Equitech-Bio, BAH65) over night at 4 /C14 C, rocking. Membranes were washed three times for at least 20 min in TBST and incubated with secondary antibodies conjugated with horseradish peroxidase, goat anti mouse (Jackson ImmunoResearch, West Grove, PA, 115-035-003) goat anti rabbit (Jackson, 111-035-003) and donkey anti goat (Abcam, ab97110) for 1 h at room temperature before membranes were washed again three times with TBST. Membranes were incubated in custom made enhanced chemiluminescence (ECL) solution containing 100 mM Tris pH 8.5, 2.5 mM p-Coumaric acid (Sigma, C9008), 1.2 mM Luminol (Sigma, A8511), 0.01 % H2O2 (Alfa Aesar, L14000), or for weak signals with SuperSignal West Femto (Thermo Scientific, 34095) and imaged on a GBOX imaging system (Syngene, as described above). For image inversion, cropping, and intensity adjustments, Fiji/ImageJ (1.53c and earlier versions, NIH) software was used (Schindelin et al., 2012). Protein band intensities were quantified with Image studio light (Li-Cor, Lincoln, NE) software, statistical analysis rank sum test or Student's t test was used and plotting of summary data was done in Matlab (Mathworks, Natick, MA, R2018a).

Toassess protein expression in non-small cell lung cancer cells, 3 million cells were seeded in 10 cm dishes and harvested the next day using cell scrapers as described above. Cells were lysed in buffer containing 50 mM Tris (Fisher Scientific), pH 7.5, 150 mM NaCl (RPI), 0.5 % NP40 (Fluka) and HALT Protease and Phosphatase inhibitor cocktail (Thermo Scientific, 1861281). Samples were processed for western blotting and analysis as described above. Each blot contained a control lysate from RPE wt cells for normalization across all cell lines and all experimental repeats. The protein band intensities were normalized with actin or GAPDH loading controls and then normalized with the RPE sample. The phospho-Akt pS573 band intensities were additionally normalized with total Akt. Experiments were repeated at least five times, and mean +/- standard error (SEM) are shown.

## Protein expression and purification

For purification of HIS6x-SEMA3A (RRAA), confluent dishes of HEK 293T were infected with adenovirus expressing semaphorin-3a and tTA helper virus (see above) and the medium was harvested the next day. To prevent unspecific protein binding to the Ni-NTA resin (Invitrogen, R901-15), the NaCl concentration was adjusted to 500 mM and 10 mM imidazole (Acros organics, 12202) were added, together with 1 mM Phenylmethanesulfonyl fluoride to reduce proteolysis (PMSF, Sigma, 517992). The protein was bound to Ni-NTA agarose overnight, rocking at 4 /C14 C. The beads were harvested, applied to a Poly-prep chromatography column (BioRad, 731-1550) and washed at 4 /C14 C with PBS containing 500 mM NaCl and 25 mM Imidazole. The HIS6x-SEMA3A was eluted with increasing concentrations of imidazole and 500ul fractions were collected. For analysis, 16 m l of the samples were mixed with 6x sample buffer containing 375 mM Tris-HCl pH 6.8, 9 % Sodium dodecyl sulfate (SDS, Thermo Scientific, 28365), 50 % Glycerol (Fisher Scientific, BP229) and 0.03 % Bromophenol blue (Bio-Rad, 161-0404) under non-reducing condition to preserve semaphorin-3a dimers. The samples were heated for 5 min at 95 /C14 C and analyzed by polyacrylamide gel electrophoresis as described

## Article

above. Gels were stained with Aquastain blue (Bulldog Bio, AS001000) and destained in tap water and the fractions containing SEMA3A were pooled and dialyzed three times in PBS for 30 min at 4 /C14 C each using Slide-A-lyzer dialysis cassettes (10 kDa cutoff, Thermo, 66380). The protein was collected, protein concentration was determined using a BCA assay (Thermo Scientific, 23225) and a microplate reader (Synergy-H1, BioTek, Winooski, VT). The protein concentration was adjusted, samples were aliquoted, snap frozen in liquid nitrogen and stored at -80 /C14 C until use.

For expression of glutathione S-transferase (GST) fusion proteins for SH3BP4 and GIPC1 the corresponding pGEX-6P-1 constructs were transformed into BL21 cells (DE3, NEB, C2527I) as described above. Small cultures were grown in LB medium rocking overnight at 37 /C14 C. Large cultures were started and OD600 was observed regularly and when it reached 0.8-1, protein expression was induced with 1 mM Isopropyl-beta-D-thiogalactoside (IPTG, Goldbio, I2481), and cultures were grown overnight at 22 /C14 C. Cells were harvested and subjected to three freeze-thaw cycles in liquid nitrogen. Next, cells were resuspended in ice cold PBS containing 1 mM PMSF, oComplete protease inhibitor cocktail (Roche), 1 mM dithiothreitol (DTT, Goldbio), 1 % Triton-X100 (Fisher Scientific, BP151). Cells were sonicated at 40 % amplitude for 3 times 10 seconds (Fisher Scientific, model FB505) at 4 /C14 C and rocked in the cold for 10 min to solubilize proteins. Lysates were spun at 12'000 rpm for 10 min at 4 /C14 C (Beckman Coulter) and supernatants were harvested. Glutathione-agarose (ABT, 4B-GLU-100) was washed with PBS, added to the cell lysates and rocked overnight at 4 /C14 C. Next, the beads were washed three times in PBS at 4 /C14 C and eluted with 5 mM reduced L-glutathione (Sigma, G4251) in 50 mM Tris-HCl, pH 8.0. The eluted proteins were analyzed by polyacrylamide gel electrophoresis and Coomassie staining as described above. As a second purification step, NaCl concentration was adjusted to 500 mM, 10 mM imidazole, and Ni-NTA-resin were added and bound over night, rocking at 4 /C14 C. The next day, the resin was collected on a column, washed, and eluted as described above. Then, eluted GST-GIPC1-HIS8x fractions were pooled and dialyzed in 50 mM Tris-HCl pH 7.0, 150 mM NaCl, 1 mM EDTA, 1 mM dithiothreitol for three times 20 min at 4 /C14 C. Next, the protein was digested with precision protease (GE life sciences, 27084301) at 4 /C14 C overnight. To sequester the protease and cleaved GST, the protein solution was incubated with glutathione-agarose as described above and the supernatant was collected. The GIPC1-HIS8x was rebound to Ni-NTA resin, eluted in a small volume, analyzed, and stored as described above until use.

## Immunoprecipitation and pull-down experiments

For protein-protein interaction studies by immunoprecipitation (IP), unless stated otherwise, HEK cells were used. Cells were transfected with plasmids encoding GFP and HA-tagged fusion constructs using PEI methods as described above. For experiments involving overexpression of Akt1-wt-HA kinase constructs, cells were starved for 12 h in serum free low glucose DMEM (Gibco, 11885-084, 1 g/L D-Glucose), containing AA. The next day, cells were washed once with PBS, harvested with cell culture spatulas (Techno Plastic Products, TPP, 99010) on ice, spun down in a table top centrifuge at 4 /C14 C. The supernatant was removed and 1 ml lysis buffer containing 50 mM Tris (Fisher Scientific), pH 7.5, 150 mM NaCl (RPI), 0.5 % NP40 (Fluka) and oComplete protease inhibitor cocktail (Roche, 11697498001) and phosphatase inhibitors Sodium fluoride (1 mM, Fisher Scientific, S299), Sodium orthovanadate (1 mM, Fisher Scientific, S454), Sodium pyrophosphate decahydrate (1 mM, Sigma, 221368), b -Glycerophosphate disodium salt hydrate (1 mM, Sigma, G5422) were added. Cells were sonicated once for 10 seconds at 20 % amplitude (Fisher Scientific) andfurther incubated at 4 /C14 Crotating for 30 min. Samples were spun down at maximum speed at 4 /C14 Candsupernatant was collected. Protein-G-agarose (ABT, 4RRPG-5) was washed with lysis buffer once and spun down. To each sample 0.8 m g of mouse anti-GFP antibody (Roche, 11814460001) and 30 m l of protein-G-agarose beads were added and samples were incubated at 4 /C14 C rotating overnight. Beads were harvested by centrifugation at 700rpm and 4 /C14 C, washed three times with lysis buffer and resuspended in 6x sample buffer as described above containing 10 % b -mercaptoethanol (BME, Fisher Scientific, BP176) and heated at 95 /C14 C for 5 min. Both the immunoprecipitation and supernatant control samples were analyzed by SDS-PAGE and western blotting as described above. For the supernatant 1/50 th of the sample was loaded. Protein band intensities for bait and preys were quantified with Image studio light (Li-cor) and results were normalized with the signal from the GFP-fusion protein. The data were analyzed and plotted with Matlab. For experiments with five or more repeats, the Mann-Whitney-Wilcoxon rank sum test was used for statistical analysis. For experiments with three or four repeats Student's t test was used for statistical analysis. For immunoprecipitation experiments with three or more repeats, three independent experiments were performed on independent days. For experiments with more than three repeats, multiple repeats per day were included.

To test in-vitro binding of recombinant full-length GST-SH3BP4-HIS8x and GIPC1-HIS8x, purified GST and GST-SH3BP4-HIS8x were re-bound to glutathione-agarose in buffer containing 50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1 % NP40, protease inhibitor cocktail (HALT cocktail, Thermo Scientific, 78442), 5 mM DTT, in the presence of GIPC1-HIS8x over night at 4 /C14 C. The supernatant was collected and the beads were washed 3 times with binding buffer. Next, the beads were boiled for 5 min in 6x sample buffer with 10 % BMEandloaded for polyacrylamide gel electrophoresis as described above. Both pellet and supernatant fractions were analyzed by Coomassie staining (Bulldog Bio) and western blotting.

## Mass spectrometry

To identify SH3BP4 binding partners, FLAG-influenza-virus hemagglutinin (HA)-tagged SH3BP4 was overexpressed in HEK cells using standard Calcium phosphate and polyethylenimine (PEI) methods as described above. Cells were harvested the next day and washed once with ice-cold PBS. Cells were lysed in buffer containing 50 mM Tris-HCl, pH 7.5; 150 mM NaCl, 1 % NP40, protease inhibitor cocktail as described above at 4 /C14 Cfor at least 30 min. Lysates were sonicated for three times 10 seconds at 20 % amplitude and lysates were pelleted. Supernatants were recovered and cleared through spin filters (0.45 m M pore size, PVDF, Millipore

<!-- image -->

## ll

## Article

#UFC40HV00) at 3000 g for 1 min at 4 /C14 C. Filtered cell lysates were pooled and mouse anti-HA tag antibody coated agarose beads were added (Sigma, #A2095) and incubated overnight rotating at 4 /C14 C. Then samples were washed seven times with lysis buffer and five more times with cold PBS. Proteins were eluted with 250 m g/ml HA peptide in PBS (Sigma, I2149) for three times. From the pooled eluted protein, 10 % were subjected to polyacrylamide gel electrophoreses (PAGE) and silver staining (Thermo scientific, 24612) to verify expression and pull-down of the protein of interest. Peptides were then precipitated with 20 % trichloroacetic acid (TCA, Sigma) on ice for 30 min. Samples were spun in a tabletop centrifuge (Eppendorf) at max speed for 25 min at 4 /C14 C. Next, pellets were washed once with 10 % TCA and spun down again. The samples were washed three rounds with 1 ml of -20 /C14 Ccoldacetone (Sigma) and spun for 25 min at 4 /C14 C. The supernatant was aspirated and samples were air dried at room temperature and frozen at -80 /C14 C until use for mass spectrometry analysis.

For phosphoprotein analysis, FLAG-HA-SH3BP4 was expressed as described above and cells were harvested in lysis buffer containing phosphatase inhibitors 1 mM Sodium, 1 mM Sodium orthovanadate 1 mM Sodium pyrophosphate decahydrate 1 mM b -Glycerophosphate disodium salt hydrate as outlined above. After peptide elution samples were boiled 5 min with corresponding volume of 6x sample buffer (with BME) and loaded for PAGE. The gel was stained using Coomassie stain and destain solutions (Bio-Rad, 1610435) and washed in water. The stained SH3BP4 band was cut and excised gel bands were reduced into approximately 1 mm 3 pieces. Gel pieces were then subjected to a modified in-gel trypsin digestion procedure (Shevchenko et al., 1996). Gel pieces were washed and dehydrated with acetonitrile for 10 min followed by removal of acetonitrile. Pieces were then completely dried in a speed-vac. Rehydration of the gel pieces was with 50 mM ammonium bicarbonate solution containing 12.5 ng/ m l modified sequencing-grade trypsin (Promega, Madison, WI) at 4 /C14 C. After 45 min, the excess trypsin solution was removed and replaced with 50 mM ammonium bicarbonate solution to just cover the gel pieces. Samples were then placed in a 37 /C14 Croom overnight. Peptides were later extracted by removing the ammonium bicarbonate solution, followed by one wash with a solution containing 50 % acetonitrile and 1 % formic acid. The extracts were then dried in a speed-vac ( /C24 1 h). The samples were stored at 4 /C14 C until analysis. On the day of analysis, the samples were reconstituted in 5-10 m l of HPLC solvent A (2.5 % acetonitrile, 0.1 % formic acid). A nanoscale reverse-phase HPLC capillary column was created by packing 2.6 m mC18 spherical silica beads into a fused silica capillary (100 m minner diameter x /C24 30 cm length) with a flame-drawn tip (Peng and Gygi, 2001). After equilibrating the column each sample was loaded via a Famos auto sampler (LC Packings, San Francisco CA) onto the column. A gradient was formed and peptides were eluted with increasing concentrations of solvent B (97.5 % acetonitrile, 0.1 % formic acid). As peptides eluted, they were subjected to electrospray ionization and then entered into an LTQ Orbitrap Velos Pro ion-trap mass spectrometer (Thermo Fisher Scientific, Waltham, MA). Peptides were detected, isolated, and fragmented to produce a tandem mass spectrum of specific fragment ions for each peptide. Peptide sequences (and hence protein identity) were determined by matching protein databases with the acquired fragmentation pattern by the software program, Sequest (Thermo Fisher Scientific, Waltham, MA)(Eng et al., 1994). All databases include a reversed version of all the sequences and the data were filtered to between a one and two percent peptide false discovery rate.

To remove contaminating false positive proteins, control samples were generated. The pcDNA FLAG-HA empty vector was expressed in HEK cells and samples were processed the same way as described above for SH3BP4. The experiment was repeated twice and the resulting list of recurrent contaminants was subtracted from the data. Next, proteins were included in the final list that exhibited at least three peptides in three independent experiments (see Table S5). For Figure 4A only proteins identified with at least ten peptides in each of three independent experiments were displayed to enhance readability of the figure. Phospho-peptides were evaluated using the Ascor algorithm where localization was considered confidently assigned with scores above 19 (Beausoleil et al., 2006). Listed are all phosphorylated SH3BP4 peptides that were detected in four independent experiments (Lu et al., 2014) (Table S1).

To quantify the endocytosis of a 5-integrins a cell surface biotinylation and uptake assay was used as previously described (Valdembri et al., 2009). Two million RPE cells were seeded per 10 cm dish and grown over night as described above. The next day, cells were starved in serum free low glucose DMEM medium (Gibco, 11885-084) for 9 h. The conditioned medium was collected and cells were washed once in ice cold PBS and biotin- labeled in 10 ml of PBS containing 0.2 mg/ml EZ link sulfo-NHS-SS-biotin (Thermo, 21331) or HOOK-sulfo-NHS-SS-Biotin (G-Biosciences, BG-09) for 30 min at 4 /C14 C in the cold room. Cells were washed once with ice cold PBS. The conditioned medium was added back and cells were placed at 37 /C14 C in a cell culture incubator for receptor internalization. At the indicated timepoints, cells were washed twice with ice cold PBS and stored in PBS. Except for the total surface protein control sample, the PBS was removed and non-internalized biotin was cleaved for 1 h at 4 /C14 C in the cold room by adding 10 ml of cleavage buffer containing 20 mM sodium 2-mercaptoethanesulfonate (MesNA, Sigma, M1511) in 50 mM Tris-HCl (Fisher Scientific, pH 8.6) and 100 mM NaCl (RPI). The cleavage reaction was quenched by 400 m l of 0.5 M Iodoacetamide (Sigma, I1149) in PBS for 10 min at 4 /C14 C. Cells were washed twice in PBS, cells were harvested by scraping and spun down in a table top centrifuge at 4 /C14 C. Cells were lysed in 1 ml of buffer containing 50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1 % NP40, protease inhibitor cocktail (Roche), 1 mMSodiumOrthovanadate (Fisher Scientific, ICN15966425) and 2 mM MgCl2 (Fisher Scientific) for 1 h at 4 /C14 Crotating. Cell lysates were spun down and supernatants were collected. For the western blot analysis of the supernatant, before addition of antibody, 50 m l of supernatant was collected and stored at 4 /C14 C. To the remaining cell lysate, for immune-precipitation of a 5-integrin, antibodies against total a 5-integrin (Biolegend, 328002, NKI-SAM-1, mouse, 2 m g per sample), inactive a 5-integrin (Millipore, MABT822, mAb11, rat, 2 m g per sample) or active a 5-integrin (Millipore, SNAKA51, MABT201, mouse, 4 m g per sample) and 30 m l of ProteinG-agarose (ABT) were added and samples were rotated overnight at 4 /C14 C. The next day, beads were collected and washed three times with lysis buffer. The liquid was removed from the beads and 20 m l of 6x sample buffer (w/o BME) were added and samples were heated at 95 /C14 C for 5 min. Samples were loaded on 10 % polyacrylamide gels and blotted on PVDF membrane and blocked

Article in 10 % dry milk solution in TBST as described above. Biotin was probed with streptavidin-POD conjugate (horseradish peroxidase, Roche, 11089153001), at 1:1000 dilution in TBST with 5 % milk for 1h. The blot was washed 3 times in TBST and exposed with Supersignal West femto reagent enhanced chemiluminescence reagent (Thermo Scientific). The band intensities were quantified with Image studio light (Li-cor) and results were plotted with a customized script in Matlab. Mean and standard deviations (SD) from at least three experimental repeats are shown. The results from wild type and knockout cells were compared using Student's t test. To assess the protein levels in the immunoprecipate and the supernatant, the blots were probed and processed as described above according to standard western blotting procedures.

To assess the effect of semaphorin-3a on Akt phosphorylation, 500 K RPE wt or SH3BP4 KO cells were seeded in 10 cm culture dishes and starved overnight in low glucose serum free medium (Gibco). To the conditioned medium, 10 m g/ml purified HIS6xSEMA3A was added and incubated for 1 h at 37 /C14 C. Cells were washed once with PBS and collected with a cell scraper and spun down in a table top centrifuge. Cells were resuspended in 50 m l of lysis buffer with 50 mM Tris (Fisher Scientific), pH 7.5, 150 mM NaCl (RPI), 0.5 % NP40 (Fluka) and HALT Protease and Phosphatase inhibitor cocktail (Thermo Scientific). Cell lysates were spun down at top speed and 6x sample buffer containing 10 % BME was added to samples before they were processed for western blotting for Akt phosphorylated at Ser473 (rabbit, D9E, Cell Signaling Technology, 4060). Blots were imaged and analyzed as described above using Image studio light (Li-Cor). The pS473 signal was normalized with the total Akt signal. Data are shown as mean and standard error and samples were compared using Student's t test.

## Live cell microscopy and analysis

For total internal reflection fluorescence (TIRF) microscopy experiments where protein targeting to clathrin coated pits (CCPs) was measured, a clathrin light chain-a(Clca)-tdTomato fusion was used as marker for CCPs. Cells were infected with adenovirus expressing tdTomato-Clca fusion protein and tTA helper adenovirus as previously described (Hardy et al., 1997, Altschuler et al., 1998) 24 h before imaging. Cover glasses were mounted on custom built chambers for microscopy. For experiments with semaphorin-3a (SEMA3A) treatment, 10 m g purified HIS6x-SEMA3A was added to 1 ml conditioned medium and cells were imaged during the first hour of treatment. Cells were imaged on a TI-eclipse inverted microscope with a perfect focus system (Nikon), a TIRF unit (Diskovery, Andor Technology, Concord, MA), an Apo TIRF 60x 1.49 NA objective (Nikon) and a 1.8x tube lens yielding 108x total magnification. The microscope contained 488 nm (150 mW), 561 nm (50 mW) and 642 nm (110 mW) solid state lasers, a dichroic filter (GFP/Rhodamine-x/Cy5 triple laser TIRF, 488/561/642, Chroma, C124324). The ultra-fast emission filter wheel (Applied Scientific Instrumentation, FW-1000), was equipped with emission band pass filters for 525 nm ± 25, 600 nm ± 25, 700 nm ± 37.5 (all Chroma) and images were acquired using a Zyla 4.2 sCMOS camera (Andor) with 6.5 micron pixel size. The system was controlled by Metamorph software (Molecular Devices, San Jose, CA). The TIRF angle of the incident laser light was software controlled (Diskovery Control v1.2.0.0, Andor Technology) and set to 80 nm penetration depth. The temperature was maintained at 37 /C14 C by a Bold Line Cage incubator (Okolab, Italy). Images were acquired every 2 seconds for 150- 211 frames (or 5-7 min total movie duration). For long-term imaging of the effects of SEMA3A on SH3BP4-GFP targeting to CCPs, images were acquired every 10 seconds for 60 min. Imaging was started after the addition of SEMA3A. For analysis of Eps15-SNAP targeting to CCPs, cells were stained with 3 m M SNAP-cell647-SiR dye (NEB, S9102S) for 20 min at 37 /C14 C in the incubator, according to manufacturer's instructions. To remove excess dye, cells were washed three times with PBS and incubated in fresh medium for at least 20 more min prior to imaging to further wash out membrane associated SNAP dye. Cells were then mounted for TIRF imaging as outlined above.

For the quantification of protein recruitment to bona-fide CCPs we used the cmeAnalysis software as previously described (Aguet et al., 2013, Srinivasan et al., 2018) (software available at https://github.com/DanuserLab/cmeAnalysis ). In brief, significant diffraction limited clathrin structures were detected by a Gaussian-fit approach for each timepoint and structures were linked over time using uTrack (Jaqaman et al., 2008), which is included in cmeAnalysis. The post-processing of the CCP tracks included gap closing, dissection of compound tracks and removal of bright cytoplasmic visitor structures as described (Aguet et al., 2013). Next, bona-fide CCPs were selected by manually setting an intensity threshold for the clathrin signal to be reached within the first 6 frames (12 seconds) of the CCP lifetime as described (Srinivasan et al., 2018). The threshold was set for the control conditions such that the lifetime distribution peaked around 30-40 seconds and the dominant contribution of short-lived structures with lifetimes shorter than 20 seconds was eliminated. This reduced the proportion of short lived (10-30 sec) and likely abortive CCPs that likely do not lead to productive endocytic events. The same intensity threshold was applied to pooled datasets (data acquired on multiple days) and across conditions within one experiment. To measure SH3BP4-GFP signal intensities at CCPs the ''master-slave'' approach was used as described previously (Aguet et al., 2013, Mettlen et al., 2010). Intensities in both channels were corrected by local background values (Aguet et al., 2013). This yielded SH3BP4-GFP intensities at individual CCPs along their entire lifetime. CCPs that were significantly SH3BP4-GFP positive in at least one single frame were selected for further analysis. Next, CCPs with similar duration were pooled in lifetime cohorts (Loerke et al., 2011), yielding average intensities per cohort. To facilitate the data interpretation, only the 40-60 second cohort was printed in the figure panels. Mean and standard error (SEM) are shown for the intensity cohorts. To compare different experimental conditions, mean SH3BP4-GFP intensities for each movie were calculated as described (Srinivasan et al., 2018). Differences between conditions were assessed by rank-sum tests (see above). For some experiments, the fraction of continuously positive CCPs was analyzed, where the GFP signal was required to be significantly positive in all frames along a CCP track. For both mean GFP intensity and continuously positive CCPs the data from all cohorts were analyzed. In order to benchmark the software, constructs that were not expected to be efficiently targeted to CCPs (cytosolic GFP, membrane-GFP) were used as negative controls and bona fide CME accessory factors Eps15-GFP and Dyn2-GFP were used as positive controls for CCP targeting. Each experiment

<!-- image -->

## ll

## Article

was repeated at least three times with approximately 10-20 movies per condition per day, with on average one cell per movie. Data acquired independently on at least three days was pooled for the analysis. This yielded around 30-60 movies per condition. Control data were acquired on all days to account for day-to-day and cell-to-cell variability. Individual CCP intensity traces were plotted from the results generated by cmeAnalysis software as described earlier (Srinivasan et al., 2018). Rank-sum statistical test was used to compare conditions.

## Cell based assays

For the antibody uptake assay, cells were seeded on 18 mm round cover glasses #1.5 (Warner Instruments, CS18R15) and starved overnight in low glucose and serum free DMEM medium (1 g/L D-glucose, Gibco, 11885-084). The next day cells were washed once with PBS containing 1 mM MgCl2 (Fisher Scientific, BP214), 1 mM CaCl2 (Fisher Scientific, BP510) and 0.2 % BSA (PBS3+) and incubated with 50 m g/ml neuropilin-1 (mouse, BioLegend, 354502) or a 5-integrin (mouse BioLegend, NKI-SAM-1, 328002) antibodies in PBS3+for 10 min in the cold (for neuropilin antibody only, to increase the amount of antibody taken up by the cells) followed by 30 min at 37 /C14 C. Cells were put on ice and washed once with PBS3+ followed by three acid washes (200 mM acetic acid, 200 mM NaCl, pH 2.3) and three more washes with PBS3+, all at 4 /C14 C. Cells were then fixed in 4 % PFA on ice for 10 min and another 10 min at room temperature. Samples were permeabilized with Triton X-100 and washed as described above and stained with secondary antimouse-AlexaFluor-647 (Thermo Scientific, A31571), phalloidin-AlexaFluor-488 (Thermo), and 4 0 ,6-Diamidino-2-phenylindole (DAPI, Sigma, D9542) for 15 min at 4 /C14 C. Samples were washed again and mounted in Fluoromount aqueous mounting medium (Sigma, F4680) and dried at room temperature. Stained cells were imaged on an inverted epifluorescent microscope (Nikon, eclipse-Ti-U) equipped with a Plan Apo l 100x, N.A. 1.45 objective (Nikon), a Lambda-XL light source and Lambda 10-B smart shutter (both Sutter Instruments), a Zyla sCMOS camera (Andor) and controlled by m Manager (NIH) (Edelstein et al., 2010). For each condition 30-35 images were acquired encompassing approximately 1-5 cells each. Plotted are average values per image (i.e. one datapoint per image). Antibody and DAPI images were segmented using Squash (ImageJ suite from the Ivo Sbalzarini lab, MPI-Dresden, Germany, http://mosaic.mpi-cbg.de/?q=downloads/imageJ ) (Rizk et al., 2014). The segmentation parameter ''regularization'' was set to 0.05 for all images and the parameter ''minimum object intensity'' was set to 0.25 for antibody signal, while it was set to 0.15 for the DAPI signal. The results of the segmentation were analyzed in Matlab. In brief, for each image, the number of cells was determined based on the number of segmented nuclei and the fluorescence intensity from the segmented antibody signal was summed and averaged per cell. For statistical analysis the Mann-Whitney-Wilcoxon rank sum test was used. The number of experimental repeats is indicated in the figure legend, one data from a representative experiment is shown.

To assess the spreading efficiency of RPE cells, 22 mm square #1.5 cover glasses (Corning, 2850-22) in 6-well plates were coated with 2 ml of 10 m g/ml fibronectin solution in PBS (Sigma, F1141) for 30 min at 37 /C14 C. Cells were seeded at 10 K per well in DMEM with 10 % FBS and AA for 20 min before they were fixed with 4 % paraformaldehyde (PFA, Sigma, P6148) in PBS for 10 min at room temperature. Samples were washed and cells permeabilized in PBS containing 0.5 % Triton-X100 (Fisher Scientific, BP151) for 5 min at room temperature. After three additional PBS washes, samples were stained with primary antibodies for paxillin (rabbit, Y113, Abcam, ab32084) and active a 5-integrin (clone SNAKA51, mouse, Millipore, MABT201) for 1 h at 4 /C14 Cin 1 % BSA in PBS. The antibody SNAKA51 recognizes the active conformation of a 5-integrins (Clark et al., 2005). After washing, cells were incubated with fluorescently tagged secondary antibodies donkey anti-mouse-AlexaFluor-647 (Thermo Scientific, A31571), donkey anti-rabbit-AlexaFluor-568 (Thermo Scientific, A10042) and phalloidin-AlexaFluor-488 (Thermo Scientific, A12381) in 1 % BSA in PBS. Cells were washed, fixed again with 4 % PFA solution, washed again, mounted for TIRF microscopy and imaged as described above. For area measurements, cells were outlined manually in the actin channel using ImageJ and data were analyzed with Matlab. Adhesions in the paxillin and active a 5-integrin images were segmented using Squash/ImageJ as described above. For the segmentation, the parameter ''regularization'' was set to 0.05, and ''minimum object intensity'' was set to 0.25 for both the paxillin and the activea 5integrin signal. Experiments were repeated at least three times, data from one representative experiment is shown. For each condition around 30-35 image stacks were acquired, each containing one cell (i.e. one data point representing one cell).

To quantify the fibronectin fibrils, RPE and endothelial HMEC1 cells were seeded at 10 K cells per 6 well plate on 22 mm square #1.5 cover glass (Corning) in full medium for 3 h. Cells were fixed with 4 % PFA (see above) and stained with antibodies against fibronectin (Abcam, ab2413, rabbit) and total a 5-integrin (Biolegend, 328002, clone NKI-SAM-1, mouse) for 1 h at 4 /C14 C. Cells were washed three times with PBS and stained with fluorescently tagged secondary antibodies donkey anti-mouse-AlexaFluor-647 (Thermo Scientific, A31571) and donkey anti-rabbit-AlexaFluor-568 (Thermo Scientific, A10042) for 15 min at 4 /C14 C. Cells were washed three times and post-fixed with 4 % PFA for 10 min before they were washed and mounted for TIRF microscopy. For each condition 50 images with one or more cells were acquired and the cells were manually classified for fibronectin fibrils. Experiments were repeated three times. Data are presented as mean and standard error and conditions were compared using Student's t test.

To test the role of SH3BP4 in SEMA3A signaling, the SEMA3A cell area contraction assay was adapted from Takahashi et al. (Takahashi et al., 1999). In brief, cells were seeded in 12 well plastic dishes (Falcon, 353225) coated with attachment factor solution (0.1 %gelatin, Gibco, S006100) and serum starved overnight as described above. The next day, 10 m g/ml purified HIS6x-SEMA3A was added to the conditioned medium and cells were incubated for 1 h at 37 /C14 C and fixed with 4 % PFA as described above. Fixed cells were imaged on a Nikon eclipse TS100 microscope with a 20x/ NA 0.40 Ph1 objective and a Retiga R3 CCD camera (Qimaging, Surrey, BC, Canada) with 4.54 m mpixel size, controlled by m Manager (NIH). For each condition 10-15 images were acquired, each containing approximately 10-20 cells. Cell outlines were manually generated in ImageJ (NIH) and analyzed and plotted in Matlab. For each condition 100 cell outlines were analyzed (i.e. one datapoint representing one cell). The Mann-Whitney-Wilcoxon rank sum

Article test was used for statistical analysis. Each experiment was repeated at least three times, data from one representative experiment are shown.

## Analysis of a 5-integrin surface expression by flow cytometry

For the analysis of integrin cell surface expression, previously published protocols were followed (Bouaouina et al., 2012). Cells were trypsinized and collected in full medium, spun down, and washed once in PBS. Cells were resuspended in PBS and the cell suspension was filtered through 40 m mnyloncell strainers (Falcon) and counted (Cellometer Auto 1000, Nexcelom). Cells were pelleted again at 200 g for 5 min and resuspended at 6 million cells per ml in Tyrode's buffer composed of 12 mM Sodium Bicarbonate (Fisher Scientific, BP328), 127 mM NaCl (RPI), 5 mM KCl (Fisher Scientific, P217), 0.5 mM NaH2PO4 (Fisher Scientific, S397), 1 mM MgCl2 (Fisher Scientific), D-glucose (Fisher Scientific), 10 mM HEPES (Thermo Fisher Scientific, 15630080). Cells were stained with function blocking a 5-integrin antibody to label all a 5-integrin (Biolegend, 328002, clone NKI-SAM-1, mouse) for 30 min at 30 /C14 C. Cells were washedwith 250 m l of Tyrodes buffer and spun down at 200 g for 5 min. Next, cells were incubated with donkey anti-mouse-AF647 for 15 min at 4 /C14 C, and again washed with Tyrodes buffer and spun down. Cells were resuspended in 1.5 ml ice cold PBS and stored on ice. Cells were analyzed on a LSR Fortessa (BD Biosciences) flow cytometer equipped with four lasers (405 nm, 488 nm, 561 nm and 633 nm). For each condition, 5-10 K cells were analyzed. The fluorescent intensities were analyzed using FlowJo (BD Biosciences). The experiment was repeated four times and mean and standard errors were plotted. Data were analyzed using Student's t-Test.

## Bioinformatics analysis

For SH3BP4, NRP1 and ITGA5 protein sequence alignment, data were downloaded from ENSEMBL (Cunningham et al., 2019) and the alignment was performed in Matlab using the multialign function (2018a, The Mathworks, Natick, MA). The SH3BP4 domain architecture was adapted from earlier reports (Kokoszy /C19 nska et al., 2009) and from information available in the Uniprot database (UniProt, 2019). For sequence analysis the web-based tools such as Lalign, Translate and others from Expasy were used (Artimo et al., 2012). For analysis of SH3BP4 phosphorylation sites phosphositePlus was used (Cell Signaling Technology) (Hornbeck et al., 2015). For the analysis of protein-protein interaction data the BioGRID database was consulted (Oughtred et al., 2019). To assess gene expression in the SH3BP4 pathway in lung cancer, the normalized data from the Lung Cancer Explorer database (http://lce. biohpc.swmed.edu/lungcancer/)(Cai et al., 2019) was used as described previously (Park et al., 2020). The database comprises 20 studies with 3,159 samples, where expression from normal and tumor tissue is normalized across all studies. Data for all studies that covered a particular gene were plotted and data are presented as box and whiskers plots where the median is highlighted and the box covers from the 25 th -75 th percentile and the whiskers cover all data except outliers.

## QUANTIFICATION AND STATISTICAL ANALYSIS

## Statistical analysis

For the statistical comparison of TIRF-microscopy data, the Mann-Whitney-Wilcoxon, also known as rank-sum test was used. TIRFmicroscopy, antibody-uptake assay, cell spreading assay and cell area assay data are presented as box and whiskers plots where the median is highlighted and the box covers from the 25 th -75 th percentile and the whiskers cover all data except outliers. For the analysis of immunoprecipitation results the rank-sum test was used when more than five repeats were available, while for experiments with up to four repeats, Student's t test was used. For immunoprecipitation experiments, mean ± standard error (SEM) are shown. For the TIRF microscopy intensity cohort vs time plots, the mean ± standard error of the intensity is shown. For biochemical uptake assays, the mean, and standard deviation (SD) is shown and Student's t test was used. For all panels, the statistical approach is indicated in the figure legends and p value cut offs are indicated: *p&lt;0.05, **p&lt;0.01, ***p&lt;0.001, ****p&lt;0.0001.

<!-- image -->