## SYSTEMS BIOLOGY

## Human PBMC scRNA-seq -based aging clocks reveal ribosome to inflammation balance as a single-cell aging hallmark and super longevity

Quantifying aging rate is important for evaluating age-associated decline and mortality. A blood single-cell RNA sequencing dataset for seven supercentenarians (SCs) was recently generated. Here, we generate a reference 28sample aging cohort to compute a single-cell level aging clock and to determine the biological age of SCs. Our clock model placed the SCs at a blood biological age to between 80.43 and 102.67 years. Compared to the model-expected aging trajectory, SCs display increased naive CD8 + T cells, decreased cytotoxic CD8 + T cells, memory CD4 + T cells, and megakaryocytes. As the most prominent molecular hallmarks at the single-cell level, SCs contain more cells and cell types with high ribosome level, which is associated with and, according to Bayesian network inference, contributes to a low inflammation state and slow aging of SCs. Inhibiting ribosomal activity or translation in monocytes validates such translation against inflammation balance revealed by our single-cell aging clock.

## INTRODUCTION

Aging is a complex process of functional decline of an organism, which is accompanied by many degenerative diseases. Aging clocks refer to linear or nonlinear models trained on omics data to predict age and aging rate, which, measured by the difference between chronological and predicted age (AgeDiff), can be used to quantify aging at different levels ( 1 ). For example, human peripheral blood transcriptome clocks have a mean absolute difference (MAD) between chronological age and predicted age of 7.8 years ( 2 ), while blood proteome -based prediction reaches a Pearson correlation coefficient (PCC) between predicted and actual age of 0.93 to 0.97 ( 3 ), and DNA methylome has reached MAD of 4.9 years in the whole blood ( 4 ) and 3.6 years in heterogeneous tissues ( 5 ). A recent 50 blood cytokine -based immune aging clock has a MAD of 15.2 years ( 6 ). We have established a three-dimensional human facial image -based linear model with MAD of 6.1 years ( 7 ) and a deep learning artificial intelligence (AI) model of 2.8 years ( 8 ). Fast or slow agers (AgeDiff &gt; MAD) display substantial agingrelated physiological decline or delay than well predicted individuals ( 2 -7 ). However, how much of the heterogeneity in aging rate is contributed by the heterogeneity at the single-cell level has not been examined. Although a few single-cell RNA sequencing (scRNA-seq) analyses comparing two points of age (young and old) have found many age-related single-cell level changes in human and monkey ovary ( 9 , 10 ), and various tissues in mice ( 11 -17 ), the two points data cannot establish an age and aging rate predictor to assess the aging rate or its heterogeneity at the single-cell level. Several singlecell clocks have been developed in mice including multi-tissue single-cell DNA methylation clocks ( 18 ) and brain single-cell aging clocks ( 19 ). While 45-sample human peripheral blood mononuclear cell (PBMC) scRNA-seq data across different ages are available ( 20 ), the quality and sparsity of the data do not allow reliable separation of known cell types or clear mapping of cell identities, in addition to the lack of precise age information.

Avery small group of individuals, the supercentenarians (SCs), &gt;110 years old, not only achieve super longevity but also have a disease-free healthy and active life even at very old age. Recently, a precious blood scRNA-seq for 7 SCs was generated ( 21 ); however, without a quantitative aging clock model, there is no proper control or reference to decipher SCs ' cellular and molecular features associated with their extreme longevity, as neither young nor old samples can be rationally used for comparison. Specifically, due to the lack of a natural aging cohort to establish single-cell aging clock, it was not possible to examine what single-cell aging features were delayed or reversed in these SCs, and what were the biological ages of the SCs compared to the common single-cell aging trajectories, or what allow them move into a different aging trajectory. To address these problems, here we generated a 17-sample human PBMC scRNA-seq dataset from volunteers in Shanghai, China, evenly distributed from age 28 to 77 years, which we termed the Shanghai East Hospital natural aging cohort (or SE cohort for short). On the basis of the cell type composition changes in SE cohort and another cohort aged from 72 to 100 years ( 22 ), we built a linear aging clock, then applied to SCs, and revealed the cellular and molecular hallmarks deviating SCs ' actual age from the model-predicted age at the single-cell and cell type level (Fig. 1A).

## RESULTS

## scRNA-seq reveals cell type composition in aging cohorts

To build a single-cell transcriptome -based aging rate predictor, we collected the PBMCs from 17 individuals (6 males and 11 females with similar age distributions) from 28 to 77 years of age, which we termed the Shanghai East Hospital natural aging cohort (or SE cohort for short), and subjected them to 10X Genomics scRNAseq. For each individual, an average of 8545 cells were sequenced and analyzed with an average of 110,364 reads per cell (Fig. 1A). We processed these data together with published PBMC scRNAseq datasets for further analysis. These include two Chinese cohorts, Chinese young cohort (CYCT) ( 23 ) and Wuhan cohort (WHCT) ( 24 ), and two Japanese cohorts, Japanese old cohort (JOCT) and SCs ( 21 ). Each of the cohorts contains five individuals. After removing 25,111 cells due to poor quality (cells with &gt;10% mitochondrial gene expression or &lt;200 genes detected), a total of 131,972 single cells (average of 7763 cells per sample) are retained for the SE datasets and a total of 23,796 cells for CYCT, 45,923 cells for WHCT, 19,252 cells for JOCT, and 38,417 cells for SC (fig. S1A). We then removed batch effect between SE datasets and other independent cohorts, and identified red blood cells (RBCs), megakaryocytes (Mega), and six major immune cell lineages -CD4 + T cells (TC), CD8 + T cells, natural killer cells (NK), monocytes (MC), and dendritic cells (DC), based on the expression of canonical lineage markers and cluster-specific marker genes, and visualized them by Uniform Manifold Approximation and Projection (UMAP) (Fig. 1B). The presence of cells from both SE datasets and other independent datasets in each cluster demonstrates that all clusters contained cells from multiple samples and thus are not segregated due to dataset-specific batch effects (fig. S1B).

## Immune cell subtypes are dissected by recluster

To separate immune cell subtypes, we separately reclustered the cells of NK and TC lineage, BC lineage, and MC and DC lineage. Using the most significantly up-regulated genes in each cluster, we identified 29 cell subtypes in total. The CD4 + T cells were subdivided into five classes: CCR7 high naive CD4 + T cells (CD4 Naive), CCR7 high NECTIN1 high naive CD4 + T cells (CD4 NaiveNECTIN1), CCR7 med CCR6 -memory CD4 + T cells (CD4 Tm), CCR6 + effector memory CD4 + T cells (CD4 Tem), and FOXP3 + regulatory T cells (CD4 Treg) (Fig. 1B and fig. S1, C and D). The CD8 + T cells were subdivided into four classes: CCR7 + naive CD8 + T cells (CD8 Naive), GZMK + effector memory CD8 + T cells (CD8 Tem), GZMB + GNLY + cytotoxic CD8 + TCs (CD8 CTL), and DLG5 high cytotoxic CD8 + TCs (CD8 CTL-DLG5) (Fig. 1B and fig. S1, C and D). The NKs were subdivided into six subtypes by the most significantly up-regulated genes in each cluster (fig. S1C). In addition, we identified six major B cell subsets: two groups of IL4R + IGHD + naive B cells (Naive B and Naive B-NRGN); two groups of CD27 + IGHG1 + memory B cells (Memory B-IL32 and Memory B-CRIP1); plasma cells or socalled antibody-secreting cells, expressing high level of immunoglobulin genes MZB1; and a subset of ITGAX + B cells defined as age-associated B cells (ABCs) (Fig. 1B and fig. S1, E and F). As for myeloid cells (including MCs and DCs), we identified six transcriptionally distinct subsets: CD14 high CD16 -classical monocytes (CD14-MCs), CD14 high CD16 -PPBP high classical monocytes (CD14-MC-PPBP), CD14 +/ -CD16 high nonclassical monocytes

(CD16-MCs), CD14 + CD16 +/ -intermediate monocytes (Intermed-MCs) (fig. S1, G and H), CD1c + conventional DCs (cDCs), and CLEC4C + plasmacytoid DCs (pDCs) (fig. S1, G and H). These subtypes of cells were then used for downstream analyses.

## Cell type proportion-based clock models reveal SCs to be much younger than their actual age

With increased age, various cell types change in proportions (Fig. 2A). We therefore tested using cell type proportions of the scRNA-seq data to build a ' clock ' to predict age. We first trained a partial least square regression (PLSR) linear model and used leave-one-out cross-validation (LOOCV) to assess the accuracy of the model. To compare prediction power across models and to prevent overfitting, all PLSR models in this study included only the top three components. We then calculated variable importance in projection (VIP) to rank the overall contribution of each variable to the PLSR model, and the variables with VIP score &gt; 1 are considered to be high explanatory variables ( 25 ). These include NKGZMH, CD8-CTL, CD8-Naive, CD4-Tm, CD4-Naive, Naive-B, NK-FCER1G, CD14-MC, and Memory-B-CRIP1 contributing significantly to the PLSR model (Fig. 2B). To extend the age spectrum of training data, using TOSICA ( 26 ), we further mapped the cell types of a published PBMC dataset generated by Guangdong Medical University that contains very old (72 to 100 years) samples (GM) ( 22 ) (fig. S2A) and trained a PLSR model. Although the GM dataset contains frail and old control samples, frail samples were reported to have elevated expression of the NEAT1 long noncoding RNA (lncRNA) in a fraction of CD14 monocytes ( 22 ), which are not a cell type contributing to our cell type composition -based aging clocks, and we do not find that they differ significantly in cell type compositions (fig. S2B) or in scRNA-seq clock-predicted cAgeDiff (fig. S2C); therefore, we include both set of samples as expanded training samples. On the basis of this extended dataset (SE + GM), we find that the three-component PLSR model based on only the top 7 cell types generates a model with PCC between predicted and chronological age of 0.88 and MAD of 8.36 (Fig. 2C). Through saturation analysis, we find that the 28 reference samples we used for clock building are sufficient to reach a stable high accuracy (fig. S2D). We also used the cell type proportions to train an elastic net (EN) regression linear model and used the weight to rank the overall contribution of each variable to the EN model (fig. S2E). The EN model based on the top 7 cell types had a similarly accurate predictive power than PLSR model, with PCC and MADof0.89 and 7.86, respectively, between predicted and chronological age in SE + GM cohorts (fig. S2F), whereas the accuracy of PLSR model is more similar between training and independent cohorts, suggestive of a slightly better generalizability. Five of the top 7 cell types overlap between PLSR and EN model (fig. S2G), hinting at their critical importance at age prediction by blood cell type compositions.

To further validate our models, we tested them on blood scRNAseq data of three independent healthy cohorts, five CYCT individuals with an age range from 29 to 58 years old, five WHCT individuals from 37 to 71 years, and five JOCT individuals with an age range from 50s to 80s (age accurate to decades instead of years; thus, an error exceeding 5 years is expected by definition). Our seven-cell type PLSR model had MAD of 10.06, 10.91, and 10.16 and PCC of 0.43, 0.69, and 0.28 to chronological agein these three cohorts, respectively (fig. S2H). Similar MADs were found with the younger than the youngest training sample, TOSICA-mapped umbilical cord blood samples (Cord) are predicted by our clock model to be -10.10 to -5.42 years old (Fig. 2D). Toward the other end of the age spectrum, SCs who are 20 years older than the oldest training sample are predicted to be 80.43 to 102.67 years old, on average 18.64 years younger than their chronological age. The age-corrected cell type EN model placed all SCs to below 110, as young as the average of 95.23 years old, and umbilical cord blood (Cord) to 7.12 to 12.68 years old (fig. S3B). We further confirmed that the age-corrected clock was not affected by batch-related factors, such as sex, datasets, and cell numbers (fig. S3, C to E).

To further test clock performance, we applied our age clocks to other independent disease datasets including a COVID-19 dataset and a systemic lupus erythematosus (SLE) dataset (fig. S2A). Our cell type composition -based PLSR model reveals a higher average biological age of individuals in disease states than age-matched controls as shown by cAgeDiff, which increases with the disease severity (Fig. 2, E to G). Perhaps due to the small sample size, only severe symptom COVID-19 patients are significantly older (Fig. 2E; t test, P = 0.040) and SLE symptoms managed and flare patients are highly significantly older than controls (Fig. 2G; t test, P = 8.77 × 10 -4 and 4.20 × 10 -3 ). cAgeDiff predicted by the EN models shows a consistent significant increase in diseases versus controls (fig. S3, F to H). These suggest that both the PLSR and EN cell type composition clock models can identify the increase in immune-related biological age of patients with these diseases.

Consistent with the model-predicted ' slow aging ' of the SCs (Fig. 2D and fig. S2H), among the seven cell types used in the PLSR and those in EN models, a total of nine cell types (fig. S2G), a highly significant negative correlation exists between two CD8 + T subtypes, CD8 + CTL and CD8 + naive T cells, which significantly increase and decrease with age, respectively, in both SE + GM and SC cohorts, but SCs largely overlap with the 70- to 100-year-old SE + GMs (Fig. 2H). This suggests that the polarization of CD8 + naive T toward CD8 + CTL is an aging hallmark at the blood cell population level.

## Age-related cell composition changes are delayed in SCs

Now that we have natural aging cohort-derived clock models, instead of artificially using any particular age group as controls, we use the cellular and molecular deviation of model-predicted versus their actual status in SCs to decipher features associated with their extreme longevity (Fig. 3A). We reverse-calculated the cell type compositions at age 110 years by PLSR clock models (Materials and Methods). Alternatively, based on the top 7 cell types in the PLSR and EN clock models (Fig. 2E), we defined the cell types whose proportions are significantly positively or negatively correlated with age by linear regression model (LR) (with RCC to age slope, P &lt; 0.05), without age-related systemic bias (fig. S4A). Then, the expected cell proportions at age 110 are calculated based on the PLSR model or the LR model, and if they are significantly higher or lower than the real proportions in SCs ( z score, P &lt; 0.05), the cell types are considered as ' SC delayed age-up cell types ' or ' SC delayed agedown cell types ' (Fig. 3A). We find that SCs show significantly delayed age-dependent decrease in CD8 + naive T cells, Naive B cells, Memory-B, and NK-GZMK ( z score, P &lt; 0.05), marginally significantly delayed age-dependent increase in CD4 + Tm( z score, P &lt; 0.1) compared to the PLSR model (fig. S4B), and delayed age-dependent decrease in CD8 + naive T cells and delayed age-dependent increase in CD4 + Tm and Naive B compared to the LR model (fig. S4C). In addition, the frequencies of cell types that contribute to the aging clocks and are delayed in SCs show no differences between the predicted values and real values of individuals from both training and independent cohorts, except in Cord and SC cohorts (fig. S4, D and E). When the proportions of age down-regulated CD8Naive cells and age up-regulated CD4 Tm cells are plotted across age, there are clear delayed deviations of SCs from the linear trend line over the 28 samples of 28 to 100 years (fig. S4F).

## Age-related single-cell gene expression changes are delayed in SCs

We used the sum of gene expressions of all cells belonging to each individual to train PLSR models as pseudo-bulk transcriptome clock model, which predicts the 28 training samples at MAD of 5.04 and PCC of 0.97 to chronological age (Fig. 3B) and predicts SCs to a mean age of 82.49 years (fig. S4G). We used the sum of gene expressions of all cells belonging to each cell type for each individual to train PLSR models as the pseudo-bulk transcriptome clock for the cell type separately with MAD from 4.80 (ABC) to 9.60 (Plasma) years (Fig. 3C). These clocks, more dramatically than the cell type proportion-based clocks, predicted the SC median transcriptome ages to 51.62 to 63.19 years before age correction (fig. S4H) to 65.01 to 75.83 after age correction (Fig. 3C). Then, based on these PLSR clock models, or alternatively defining top contributing genes with VIP &gt; 1 (table S1) as age-related, and predicted their expression levels at 110 using LR models in each cell type, we then defined ' SC delayed age-up genes ' (up-regulated with age and delayed in SCs) and ' SC delayed age-down genes ' (downregulated with age and delayed in SCs) similar to cell type analysis described above (Fig. 3A). Using the LR models, which show no systematic bias against age (fig. S5, A and B, and table S2), we find 2749 and 495 age-up and age-down genes, respectively (table S3), and 1023 age-up and 149 age-down genes significantly delayed in SCs. As validation of the model, we find no significant differences between real and predicted gene expressions of genes that contribute to the aging clocks and are delayed in SCs in the training cohort, as well as in other independent cohorts, except in SC cohorts (fig. S5C). Transcription factor (TF) target enrichment analysis shows that age-up and age-down genes are enriched for targets of FOXR2, ZNF318, PSMB5, PER1, ZZZ3, and NERF (fig. S5, D and F). Among them, FOXR2, ZNF318, PSMB5, and ZZZ3 targets are enriched in SC delayed age-down genes and nuclear factor κB (NFκB) targets are enriched in SC delayed age-up genes (fig. S5, F and G). Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses reveal that age-up genes enrich for NFκB/ tumor necrosis factor (TNF)/TLR/NOD/FoxO signaling pathway, virus infection, apoptosis, T cell receptor signaling pathway, cytokine, and PD-1 pathway (fig. S6, A and B), and among them, NFκB/TNF/TLR/NOD/FoxO signaling pathway, virus infection, apoptosis, cytokine, and mitogen-activated protein kinase (MAPK) pathway are significantly delayed in SCs (Fig. 3D), while age-down genes most commonly enrich for ribosome genes (fig. S6, C and D), which are significantly delayed in most cell types of SCs (Fig. 3E and table S4).

Compared to the PLSR model -expected values, there are 111 age-up genes significantly delayed in the SCs, which are related to MAPK signaling pathway, rapamycin (mTOR) signaling pathway, NFκB signaling pathway, and virus infection (fig. S6E), whereas

1261 age-down genes are significantly delayed in SCs. Again, ribosome- and translation-related genes are enriched in most cell types at the very top of the list (fig. S6F). The delay of these ribosome genes decline in SCs cannot be attributed to batch effect in gene expression because, as a group, average ribosome gene expressions in each cell type do not differ among different cohorts [analysis of variance (ANOVA), P = 0.53; fig. S7A], while the age down-regulated ribosome genes identified in SEs also decrease with age in other cohorts (fig. S7B).

## Inflammatory cell-cell interactions are repressed in SCs

Altered intercellular communication is an integrative hallmark of aging ( 27 ). On the basis of annotated ligand-receptor databases, we calculated the numbers of ligand-receptor pairs in every pair of cell types using CellphoneDB ( 28 ). We found that many agerelated receptor-ligand pairs are up-regulated during aging among various cell types (PCC, P &lt; 0.05). These pairs are mostly related to inflammation, cytokines, and antigen presentation (Fig. 3F). Similar to delayed age-up and age-down cell types and genes, we defined delayed age-up and age-down ligand-receptor pairs in SCs. The age-related down-regulated aMb2 integrin signaling ligand-receptor pairs are generally delayed in SCs for most cell types, and ageup inflammation-related TNF signaling ligand-receptor pairs are also delayed, especially in pDC (Fig. 3G).

## Ribosome/translation counteracts inflammation at singlecell level

On the basis of the above results, we wondered whether age-related decline in ribosome levels and age-related increase of inflammation levels are two linked events of the single-cell aging. To quantitatively assess the inflammatory state of each cell, we calculated the inflammatory score for each cell based on inflammatory-related gene list (table S5) as described previously ( 29 ). The median expression levels of ribosome genes are used to indicate the level of ribosomes in each cell. In both SE and SC cohort, there is a significant negative correlation (PCC = -0.22, P &lt; 2.2 × 10 -16 in SE, PCC = -0.23, P &lt; 2.2 × 10 -16 in SC, PCC = -0.30, P &lt; 2.2 × 10 -16 combined) between ribosome level and inflammation score across all single cells of all cell types (Fig. 4A). Furthermore, across different cell types, there is also a significant negative correlation between the average ribosome levels and average inflammation scores (RCC = -0.58, P = 0.0021 in SE, RCC = -0.59, P = 0.0020 in SC, RCC = -0.58, P = 1.0 × 10 -5 combined) (Fig. 4B). On the basis of this ribosome level versus inflammation score distribution, cell types can be separated into highinflammatory and low-ribosome (HI-LR) cell types, which include Intermed-MC, CD14-MC, CD16-MC, CD14-MC-PPBP, and megakaryocytes, and low-inflammatory and high-ribosome (LIHR) cell types, which are the rest of the cell types (Fig. 4B). In all but one cell type, the total ribosome gene expression level decreases with age in SEs, whereas in HI-LR cell types, SCs show a similar level to young SEs (Fig. 4C). Consistently, principal components analysis (PCA) analysis based on ribosomal and inflammatory gene expressions in each single cell within the same cell type shows that for most cell types, for example, CD4-Treg and CD14-MC, single cells fall into two major cellular states, corresponding to high and low ribosome expression, respectively (fig. S8A). Consistent with the ribosome expression level, the proportion of cells in the high ribosome state decreases with age for most cell types (RCC to age range from -0.64 to 0.43 with a median of -0.23; Fig. 4D and table S6).

Remarkably, SCs consistently have more cells in the high ribosome state than SE in the HI-LR cell types (Fig. 4D). These indicate that HI-LR cell types ' high ribosome expression, or more precisely more cells in high ribosome state, is a common feature of super longevity at the single-cell level in the blood.

We also validated the changes of ribosome expression and inflammation scores in COVID-19 and SLE cohorts. Consistent with the balance between ribosome expression and inflammation, with the disease progression in COVID-19 and SLE, the inflammatory score consistently increased and the expression level of ribosomes decreased in all cell types and HI-LR cells, with the changes generally more pronounced in SLE than in COVID-19 (fig. S9).

Furthermore, we find significantly different cell-cell communication between cells in the high ribosome versus in the low ribosome within most cell types (Fig. 4E). In the low-ribosome state, the top 100 ligand-receptor pairs are mostly between CD8-CTL and the rest of the cell types, including MHCII, S100A8/A9TLR4, CCL5-CCR5, and TNF-related pairs, while these are all significantly lower in high-ribosome state ( t test, P &lt; 0.01; Fig. 4E and fig. S8B). The most commonly down-regulated ligand-receptor pair in high versus low ribosome state is the interaction between ribosomal protein S19 (RPS19) and its membrane receptor C5AR1 (fig. S8B), which has been reported to suppress immune response ( 30 ).

Finally, to understand the causal relationships among ribosome level, inflammation level, age, and aging rate in SE + GM data, we inferred a Bayesian network (BN) based on all single cells in HI-LR and LI-HR cell types, respectively, in the aging rate reference SE + GM data. Here, aging rate is measured by age-corrected AgeDiff, which is the difference between the predicted age and the chronological age based on the cell composition PLSR model and corrected by age. The BN model infers that AgeDiff is dependent on the level of ribosome and inflammation in the HI-LR cell types, and the level of ribosome is in turn dependent on age. The model also infers that a decrease in ribosome level increases in the level of inflammation in the HI-LR cells. This is not the case in LI-HR cell types, where age decreases inflammation score, and inflammation score does not contribute to aging rate (Fig. 4F). These suggest that ribosome abundance, which is often a surrogate of translation and growth rate, may suppress the inflammatory cytokine secretion of the HILRcells; in other words, high level of inflammatory cytokine expression might be a consequence of stagnant growth or insufficient basally low ribosomal activities of the HI-LR cell types, and that SCs might benefit from the high cellular ribosome/translation/ growth rate to counteract aging-induced chronic inflammation. In contrast, rather than inducing an inflammatory state, the basally low-level inflammatory cytokines might be a consequence of the normal growth of the LI-HR cells (Fig. 4F). Similar relationships can be consistently observed in BN models inferred for each cell type, and based on the interactions in these BNs, the HI-LR cell types automatically aggregate together and segregate from other cell types by unsupervised hierarchical clustering (Fig. 4G).

Finally, to confirm the inferred suppression of inflammation by ribosomal activity (translation) in HI-LR cells, we temporally inhibited translation using a translation inhibitor cycloheximide (CHX), which binds to the 60 S ribosomal unit and inhibits translation elongation ( 31 ), in human blood primary monocyte cells and THP-1 monocyte cell line, which represents a HI-LR cell type and was found to secrete senescence-associated cytokines in our previous study ( 32 ). As expected from the BN model, inhibiting translation caused a significant up-regulation of commonly observed inflammatory cytokines, interleukin-6 (IL-6) and IL-8, in a time-dependent manner in primary monocyte cells (Fig. 4H) and THP-1 (fig. S8C).

## DISCUSSION

Together, by generating a reference 28-sample aging cohort, we developed single-cell level single-cell aging clocks based on single-cell type compositions of our reference 17-individual aging cohort, and placed the SCs at a biological age between 80.43 and 102.67 years (Fig. 2D and fig. S2H). Among the independent validation cohorts, both the Chinese WHCT and CYCT and Japanese JOCT are well aligned to their actual ages. Although the single-cell aging clocks we constructed have already reached saturation within the framework of linear models, a larger size of training data, in the hundreds or even thousands, may further enable building AI models to further increase accuracy. Furthermore, we generated single-cell type transcriptome clocks for each cell type and placed SCs to an even younger cross cell type median transcriptome age range of 65.01 to 75.83 years (Fig. 3C), much younger than their chronological ages. This also suggests a stronger slow aging of SCs at the single-cell transcriptome level than at the cell type composition level. It should be noted that all the age predictors we used (PLSR and EN) are linear models. Therefore, the existing models are sufficient to capture the linear age-related changes without necessarily the full age range. It is based on only these linear models we find the SCs have a significantly slower aging rate compared with expected linear changes during natural aging. In the future, it would be interesting to investigate the nonlinear changes during aging and in SCs, which will entail more samples from all age ranges for building nonlinear models. It would also be interesting to test the clock on progeroid diseases like Hutchinson-Gilford progeria syndrome.

To identify the cell types and molecular events underlying the slow aging of SCs, we used the clock models to generate the agematched expected profiles for the SCs and then compared them to the real values in SCs in terms of cell types and molecular features. We found that, at the cell type level, SCs display an increase in naive CD8 + T cells, together with a decrease in cytotoxic CD8 + T cells and memory CD4 + T cells. At the single-cell level in the HI-LR cell types, SCs contain more cells with high ribosome level, which is strongly negatively associated to the expression of inflammatory cytokine genes. Using a BN inference model, we inferred that high ribosome gene expression levels potentially contribute to a low inflammation state and delayed aging or slow aging of SCs in HILR cell types. Such a causal relationship was observed in cultured monocytes experimentally inhibiting ribosomal activity or translation induced the expression of inflammatory cytokines, such as IL-6 and IL-8. A recent study has found that these cytokines, generated by macrophages (tissue-infiltrated monocytes) if not rapidly dampened, can recruit cytotoxic CD8 + T cells ( 33 ), which initiate killing of the senescent cells. It would be interesting to see whether the reduced level of cytotoxic CD8 + T cells in SCs could be a further consequence of reduced inflammatory cytokine expression in the HI-LR cell types.

Ribosomopathy in humans results in Diamond Blackfan syndrome, which encompasses many age-related degenerative symptoms, including glaucoma and cataracts, at least through the G0G1 cell cycle arrest ( 34 ). In yeast, ribosomal mutations result in shortened replicative life span ( 35 ). In contrast, improving translation efficiency by specific ribosome variant increases life span in multiple species ( 36 ). Our previous study has found ribosome gene expression being the most significant pathway positively correlated with life spans among mice subjected to different lifestyles ( 37 ). Here, we identified that single-cell ribosomal gene expression levels decrease with age in nearly all cell types and are significantly delayed in the HI-LR cell types in SCs (Fig. 4D). We also found that in the HI-LR cells, a further decrease in their basally low ribosomal gene expression could lead to enhanced inflammatory cytokine expression. Since ribosomal levels are a robust measure of growth rate for single cells ( 33 , 38 ) and enhanced inflammatory cytokine is a senescence-associated secretory phenotype (SASP) ( 39 ) accelerating aging rates in human ( 8 ), the high proportion of cells in high ribosome states of the HI-LR cell types in the blood from SCs and the ribosome/translation against inflammation balance identified by our single-cell aging clocks reveal an unexpected aging hallmark at the single-cell level and fuel the philosophical assertation ' When you are not growing, you are not living. '

## MATERIALS AND METHODS

## Ethics approval and participant consent

Human peripheral blood samples were collected from Shanghai East Hospital with informed consent and were approved by the hospital ethics committee in accordance with the Helsinki Declaration (registered number of approval: 2019.032).

## Human blood sample collection and storage

Peripheral venous blood samples were collected at the Shanghai East Hospital Health Peer Center. Clinical parameters and medical histories were available for all volunteers. All volunteers were evaluated as generally healthy at the time of blood collection based on evaluation of their medical history and assessment of their vital signs. In our present study, 17 healthy volunteers were enrolled in scRNA-seq.

## Isolation of PBMCs and monocyte

PBMCs were isolated from human peripheral blood samples as previously described ( 40 ). In brief, 5 ml of fresh peripheral blood was drawn per individual, mixed with 5 ml of phosphate-buffered saline (PBS), and then gently layered over 5 ml of Ficoll (GIBCO, USA) in a Falcon tube. Samples were centrifuged at 400 g for 30 min without brake. After centrifugation, PBMCs in the white layer were carefully transferred to a new tube. After being washed with PBS twice, samples were then centrifuged at 400 g for 10 min, and the pellets were resuspended with cryopreservation medium [1 ml of fetal bovine serum supplemented with 120 μl of dimethyl sulfoxide (DMSO)] for long-term storage in liquid nitrogen.

Humanperipheral blood primary monocytes were isolated using the EasySep Human Monocyte Isolation Kit (STEMCELL Technologies, USA). PBMCs (5 × 10 7 ) were resuspended in 2 ml of isolation buffer (PBS containing 2% fetal bovine serum, 1 mM EDTA). Cells were incubated with 100 μl of isolation cocktail for 10 min under 4°C, and then 100 μl of magnetic particles was added, top up to

2.5 ml with isolation buffer, and incubated for 2.5 min at room temperature. Finally, the enriched cell suspension was harvested and centrifuged at 400 g for 10 min.

## 10X genomics scRNA-seq library preparation

To perform scRNA-seq, thawed PBMCs were captured using the 10X Chromium System (10X Genomics, USA) according to the manufacturer ' s protocol. Cells were loaded onto the Chromium Controller Instrument to generate gel bead-in-emulsions (GEM). Afterward, barcoding and complementary DNA (cDNA) libraries were prepared using the Single-Cell 3 ′ Library and Gel Bead Kit (catalog no. 120237, 10X Genomics). GEM -reverse transcription (RT) was performed in a Mastercycler Nexus Thermal cycler (Eppendorf), and then GEMs were broken. cDNA was cleaned up with Dynabeads MyOne Silane Beads (Thermo Fisher Scientific, USA) and amplified. Amplified cDNA product was then cleaned up using the SPRIselect Reagent Kit (0.6× SPRI, Beckman Coulter). Indexed sequencing libraries were constructed using the Chromium Single-Cell 3 ′ Library Kit and subsequently sequenced using Illumina HiSeq PE150 at Novogene (Beijing, China) with average sequencing depth of &gt;15 million reads per cell.

## scRNA-seq data processing

Reads for each gene from the 17 scRNA-seq samples were quantified using the Cell Ranger Single-Cell Software Suite (version 3.0.0, 10X Genomics) against the GRCh38 human reference genome. Independent health cohorts of Chinese were downloaded from Beijing Institute of Genomics (BIG) Data Center: CYCT ( 41 ), HRA000150; WHCT ( 24 ), HRA000069. JOCT dataset was obtained from the authors of the original publication, and the SC dataset was downloaded from https://humandbs.biosciencedbc.jp/en/hum0229v1 ( 21 ).

SCANPY ( 42 ) (single-cell analysis python toolkit, version 1.7.1) was used to read a combined gene-barcode matrix of these samples. Quality of cells was assessed based on three metrics step by step: (i) Genes expressed in &lt;3 cells were removed. (ii) Cells with &lt;200 or &gt;5000 genes detected were removed. (iii) Cells with &gt;15% mitochondrial gene counts were removed.

## Batch effect correction and cell subtype annotations

To integrate cells into a shared space from different datasets for unsupervised clustering, we used the harmony algorithm to do batch effect correction ( 43 ). First, total count matrix was normalized by library size correction using default size factor 10,000. Highly variable genes (HVGs) were calculated by function pp.highly\_varia-ble\_genes with default parameters. All ribosomal and mitochondrial genes were then removed from the list, which were identified as input to perform PCA analysis. Then, this PCA matrix was fed into pp.harmony-integrate function with default parameters, in which each sample was set as different batch. The resulting batch-corrected matrix was used to build nearest neighbor graph. Such nearest neighbor graph was then used to find clusters by Louvain algorithm with parameter ' resolution ' = 0.9 to identify the cell clusters. The cluster-specific marker genes were identified using the rank\_genes\_groups function by Wilcoxon rank-sum test. The top-ranked genes from the respective test statistic are regarded as marker genes. We annotated the clusters into eight major cell types based on the known marker genes. For gene expression

To identify subclusters within each major cell type, we performed a second round of clustering on T/NK, B, and myeloid cells separately. The procedure of the second round of clustering is the same as the first round on the HVGs chosen as described above, with resolution ranging from 0.1 to 0.9. Annotation of the resulting clusters to cell types was based on the known markers and top highly expressed genes compared with other cells.

## Clock model generation

Partial least-squares regression (PLSR) clock model based on cell type compositions was established using the ' plsr ' R packages with only the first three PLSR components to avoid overfitting. The leave-one-out (LOO) method (predictors were trained using all but one samples and then used to predict the age of the leftout samples) was applied to obtain the predicted age of each sample. Variable Influence on Projection (VIP) is a weighted sum of squares of the PLSR loadings that takes into account the amount of explained Y variance of each component. VIP scores give information about how the variables combine to form the quantitative relation between X and Y , thereby providing a better assessment of their relative importance in the model, and a VIP score ≥ 1.0 is considered as highly influential ( 45 ). VIP scores are calculated with the formula defined by Chong and Jun ( 46 ).

EN clock model was trained on all subtype compositions of 28 SE + GM samples with α = 5 × 10 -4 and L1 ratio = 0.08. Then, the cell types with high weight (|weight| &gt; 50) in the model were considered to be important for age prediction.

We used the sum of gene expressions of all cells belonging to each cell type for each individual to train PLSR models as the pseudo-bulk transcriptome clock for the cell type.

## Saturation analysis

To test the sufficiency of our sample size, we conducted a saturation analysis. We randomly selected one sample at a time to add to the training dataset, ranging from 5 to 28 samples. For each point, we repeated the process 100 times and calculated the average MAD and PCC.

## Cell type annotation transfer by TOSICA

To transfer cell type annotations on additional datasets including GM old samples and Cord samples (GSE157007), COVID-19 (GSE158055), and SLE (GSE174188), we used the cell type annotation transfer tool TOSICA that we recently developed ( 26 ), which can achieve interpretable, accurate, and batch immune cell type transfer from reference dataset to query dataset. Here, we used 17 SEs as reference dataset to train a REACTOME pathway masked TOSICA model and applied the model to predict the cell type of each cell in the disease datasets.

## Delayed age-up and age-down cell types

For PLSR-based procedures, the predict cell type compositions at 110 years were generated by calculating the solution of the reversed PLSR clock models at 110 years. ½ Pre compositions � F n � 1 ¼ 110 �½ PLS1 þ PLS2 � GLYPH&lt;0&gt; 1 1 � n , where n is the 29 cell types used in the model. For LR model -based procedures, the delayed age-up and delayed age-down cell types are identified by regressing the cell type compositions of each individual to age using the function lm(Composition~Age) in R for each gene with P &lt; 0.05 (corrected by Benjamini-Hochberg adjustment). Then, the predict gene expression levels at 110 years were considered as the expected compositions, and z score of the expected value on the distribution of real composition levels in SCs was used to determine whether the gene expression was significantly delayed with P &lt; 0.05 (corrected by Benjamini-Hochberg adjustment).

## Delayed age-up and age-down genes, Gene Ontology terms, and TF enrichment analysis

For PLSR-based procedures, the predict gene expression levels at 110 years were generated by calculating the solution of the reversed PLSR clock models at 110 years. ½ Pre expression levels � n � 1 ¼ 110 �½ PLS1 þ PLS2 � GLYPH&lt;0&gt; 1 1 � n , where n is the genes with VIP &gt; 1. For LR model -based procedures, the delayed age-up and delayed age-down genes in each cell type are identified by regressing the median gene expression levels of these genes in each cell type of each individual to age using the function lm(Expression~Age) in R for each gene with P &lt; 0.05. Then, the predict gene expression levels at 110y were considered as the expected expression, and z score of the expected value on the distribution of real expression levels in SCs was used to determine whether the gene expression was significantly delayed with P &lt; 0.05.

Enriched KEGG pathway was determined for each group of cells using R package clusterProfiler ( 47 ) with default parameters. Annotation Dbi R package ' org.Hs.eg.db ' was used to map gene identifiers. Terms enriched in at least two cell types were visualized with the ggplot2 R package as dot plot. Metascape (http://metascape.org/ gp/index.html#/main/step1) was used for TF enrichment analysis.

## Inflammatory score

For a given cell or cell subset X , the inflammatory score was computed as the sum of all unique molecular identifier (UMI) for all the genes of ' HALLMARK\_INFLAMMATORY\_RESPONSE ' label from MsigDB ( 48 ), expressed in X, divided by the sum of all UMI expressed by X ( 49 ).

## Cell-cell communication analysis

Cell-cell communication was predicted based on the scRNA-seq data by using CellPhoneDB software (version 1.1.0) ( 28 ). For each individual, only receptors and ligands expressed in at least 10% of cells of a given type were further analyzed. The significance of agerelated difference was determined by RCC. The delayed age-up and down pairs were determined as delayed genes based on average expression of each ligand-receptor pair across different cell types, and only the top 100 pairs were used for visualization by LRPlot function from R package iTalk ( 50 ).

## Age correction of AgeDiff and predicted age

To study chronological age-independent associations of AgeDiff (the difference between predicted and actual age based on PLS/ EN model), we corrected AgeDiff for age by fitting to a polynomial model to age as follows, with span 0.5 and degree 2:

The following formula is used to correct the predicted age for the bias against age:

## BN inference

Weused the WinMine package (https://www.microsoft.com/en-us/ research/publication/the-winmine-toolkit-2) to calculate joint conditional probability and build the preliminary potential BNs. The BN analysis method is as described previously ( 51 ). Here, BN is inferred on the mean ribosome protein gene expression level [derived from scanpy.pp.calculate\_qc\_metrics() function], inflammation score, age, and age-corrected AgeDiff of each single cell. To avoid any signal loss by batch effect correction among four different datasets, the precorrection SE gene counts given by SCANPY were used. Only edges with PCC slope P &lt; 0.05 are kept.

## Translational inhibition assay

CHX (100 μg/ml final concentration) or equal volume of vehicle solvent (DMSO) was added to the THP-1 and human blood primary monocyte cells 15, 30, 60, 180, and 360 min before harvesting. Total RNA was extracted from cells using TRIzol Universal reagent (Tiangen, Beijing) according to the manufacturer ' s protocol. cDNA was synthesized using HiScript III RT SuperMix for qPCR (+gDNA wiper) (Vazyme, Nanjing), gene expression level was quantified by reverse transcription polymerase chain reaction (RT-PCR) using ChamQ Universal SYBR qPCR Master Mix (Vazyme, Nanjing), and relative gene expression levels were calculated using the 2 -△△ Ct method. Glyceraldehyde-3-phosphate dehydrogenase (GAPDH) was used as the endogenous control. The following primers were used for the quantitative PCR: GAPDH, 5 ′ -TTGAGGTCAATGAAGGGGTC-3 ′ (forward) and 5 ′ -GAAGG TGAAGGTCGGAGTCA-3 ′ (reverse); IL-6, 5 ′ -ACTCACCTCTT CAGAACGAATTG-3 ′ (forward) and 5 ′ -CCATCTTTGGAA GGTTCAGGTTG-3 ′ (reverse); IL-8, 5 ′ -GTTTTTGAAGAGGGCT GAGAATTC-3 ′ (forward) and 5 ′ -CCCTACAACAGACCCACA CAATAC-3 ′ (reverse).

## Supplementary Materials

## REFERENCES AND NOTES

Acknowledgments: We thank C. V. Cannistraci for insightful suggestions. Funding: This work was supported by grants from the National Natural Science Foundation of China (92049302 and 32088101), China Ministry of Science and Technology (2020YFA0804000), and Shanghai Municipal Science and Technology Major Project (2017SHZDZX01) to J.-D.J.H. and Major Program of Development Fund for Shanghai Zhangjiang National Innovation Demonstration Zone &lt;Stem Cell Strategic Biobank and Stem Cell Clinical Technology Transformation Platform&gt; (ZJ2018-ZD-004) to H.Z. and Z.L. Author contributions: J.-D.J.H. and H.Z. conceived and designed the project. Z.L. provided resources. L.G., J.Z., and L.M. collected clinical samples. L.G. and H.W. performed the experiments. J.C. and K.L. performed the computational analyses. J.D.J.H. and J.C. designed the analyses and wrote the paper with help from others.

Competing interests: A Chinese patent (202210687262.4), named ' Senescence detection method and system based on immune cell typing, ' has been applied by Shanghai East Hospital on 17 June 2022. The patent is still pending (2 March 2023), and H.Z., J.-D.J.H., and J.C. are also authors on the patent. The authors declare no other competing interests. Data and materials availability: All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. Processed gene expression data are deposited in Gene Expression Omnibus (accession no. GSE213516).