## E3 ligase autoinhibition by C-degron mimicry maintains C-degron substrate fidelity

## Graphical abstract

## Highlights

## Authors

## Correspondence

## In brief

E3 ligases must recruit proteins needing ubiquitylation while ignoring related nonsubstrate proteins. Scott et al. show that this depends on interconversion between a C-degron-mimic-mediated autoinhibited self-assembly and an active monomerfor the CUL2 receptor KLHDC2. Both forms are deactivated without substrate. True substrates drive forming the active neddylated CRL2 KLHDC2 E3.

## E3 ligase autoinhibition by C-degron mimicry maintains C-degron substrate fidelity

Daniel C. Scott, 1,8, * Moeko T. King, 1,7 Kheewoong Baek, 2,7 Clifford T. Gee, 3 Ravi Kalathur, 1,4 Jerry Li, 5 Nicholas Purser, 5 Amanda Nourse, 1,4 Sergio C. Chai, 3 Sivaraja Vaithiyalingam, 1,4 Taosheng Chen, 3 Richard E. Lee, 3 Stephen J. Elledge, 6 Gary Kleiger, 5 and Brenda A. Schulman 1,2

## SUMMARY

E3ligaserecruitment of proteins containing terminal destabilizing motifs (degrons) is emerging as a major form of regulation. How those E3s discriminate bona fide substrates from other proteins with terminal degron-like sequences remains unclear. Here, we report that human KLHDC2, a CRL2 substrate receptor targeting C-terminal Gly-Gly degrons, is regulated through interconversion between two assemblies. In the self-inactivated homotetramer, KLHDC2's C-terminal Gly-Ser motif mimics a degron and engages the substrate-binding domain of another protomer. True substrates capture the monomeric CRL2 KLHDC2 , driving E3 activation by neddylation and subsequent substrate ubiquitylation. Non-substrates such as NEDD8 bind KLHDC2 with high affinity, but its slow on rate prevents productive association with CRL2 KLHDC2 . Without substrate, neddylated CRL2 KLHDC2 assemblies are deactivated via distinct mechanisms: the monomer by deneddylation and the tetramer by auto-ubiquitylation. Thus, substrate specificity is amplified by KLHDC2 self-assembly acting like a molecular timer, where only bona fide substrates may bind before E3 ligase inactivation.

## INTRODUCTION

The ubiquitin-proteasome system (UPS) is a major pathway regulating eukaryotic protein levels. Selectivity for UPS protein substrates is largely governed by E3 ligases, which modify proteins with ubiquitin (UB) to target them for degradation. 1 Given the importance of the UPS in regulating protein turnover, a major question is how E3 ligases selectively target bona fide substrates while sparing other proteins. 2

E3 ligases possess unique protein interaction domains that recruit substrates by recognizing a molecular motif termed a 'degron. 3 ' Many E3 ligases maintain proteostasis in quality control pathways, and such E3s typically detect degrons within native sequences aberrantly exposed upon errors in protein synthesis, folding, or complex assembly. 4-9 Other E3 ligases function in regulatory cascades, selectively binding degrons modified post-translationally by phosphorylation or prolyl hydroxylation. 10

Yet, another theme in the field of E3 specificity is the recognition of degrons at protein termini. 11-13 The first degrons identified-more than 30 years ago-are hydrophobic or basic amino acids at protein N termini. 14 Such sequences bind distinct domains within a family ''N-degron'' E3 ligases, as well as bacterial Clp-family proteases. 15-17 Recently, several other E3 ligase substrate-binding domains (SBDs) were shown to recognize degrons at either N or C termini. Some of these terminal degrons were identified serendipitously, 18-20 whereas others were identified by global protein stability (GPS) technology, a highthroughput method for profiling UPS substrates. 21-23

Howisspecificity for terminal degrons established by E3s? For some, regulation is achieved through the controlled formation of terminal degrons, such as proteolytic cleavage yielding fragments with C- and/or N-degrons, or by enzymatic addition or modification of amino acids at protein termini. 11,24 Alternatively, E3 ligases can be regulated by the metabolically controlled incorporation of SBDs into multiprotein complexes. 25-28

Nonetheless, for many E3 ligases recognizing terminal degrons, the question remains as to how true substrates are distinguished from those naturally bearing degron-like amino acidsequences. 17,29-35 In this regard, a particularly vexing terminal degron E3 ligase is CRL2 KLHDC2 , a member of the CRL2 family of

multiprotein cullin-RING ligases (CRLs). 21,22,29 Prior GPS screens and crystal structures revealed the Gly-Gly C terminus as the primary feature of CRL2 KLHDC2 substrate degrons. 21,22,29,36,37 Identification of CRL2 KLHDC2 substrates by GPS profiling revealed roles in selenoprotein quality control and signal-induced regulation by targeting the deubiquitylating enzyme USP1 during DNA repair. 18,21,22,29,37,38 Because KLHDC2's SBD binds peptide-like sequences with a C-terminal Gly-Gly sequence, it remains perplexing how CRL2 KLHDC2 avoids targeting UB and Ub-like proteins (UBLs) that contain flexible C termini terminating with Gly-Gly. Crystal structures of the KLHDC2 SBD in complex with C-terminal degron peptides revealed an extensive constellation of KHLDC2 basic and aromatic residues anchoring the substrates C-terminal diGly motif. 29 KHLDC2'spropensity to tolerate diverse sequences upstream of the diGly motif is explained by interactions with only backbonecarbonyls of three of four upstream residues of the substrate. Although structural studies raised the possibility that a UBL protein might bind the KLHDC2SBD,prior GPS studiessuggested that GFP-UB had the negligible ability to trigger its KLHDC2dependent degradation. 22,37

Here, by investigating how CRL2 KLHDC2 excludes the UBL NEDD8 as a substrate, we report biochemical, structural, and cell biological studies that collectively reveal a unique substrate mimicry mediating self-assembly of KHLDC2-EloBC as a specificity amplification mechanism that restricts ubiquitylation to bona fide substrates.

## RESULTS

## KLHDC2 recognizes the diGly C terminus of NEDD8

We confirmed how KLHDC2 binds C-degrons by determining the crystal structure of its SBD in complex with a peptide derived from EPHB2 (Figure 1A, Table 1). KLHDC2 sidechains cradle the substrate C-terminal Gly-Gly residues, making limited contact with the sidechains of preceding residues (Figure 1A). Thus, consistent with prior structures, we observed an apparent lack of sequence requirements for residues preceding the diGly motif. 29

To determine if KLHDC2 binds to ubiquitin (UB) or the UB-like protein NEDD8, we developed a competitive binding assay. Briefly, time-resolved fluorescence resonance energy transfer (TR-FRET) occurs upon binding of a fluorescently labeled substrate peptide from Ric8B to the SBD of KLHDC2-proteins are assayed for binding to KLHDC2 based on their inhibition of the TR-FRET signal. The titration of two CRL2 KLHDC2 substrates (SELK or USP1) resulted in the loss of the TR-FRET signal

(Figure 1B). Consistent with prior studies, UB was not an effective competitor, 22,37 but unexpectedly, NEDD8 inhibited the KLHDC2-Ric8B interaction. A NEDD8 mutant whose C-terminal Gly residue was mutated to Asp could not compete (Figure 1B), consistent with the essential nature of the diGly motif in binding to KLHDC2.

To determine whether NEDD8 can serve as a CRL2 KLHDC2 substrate, we assayed ubiquitylation employing a ''pulse-chase'' format that detects neddylated CRL2 KLHDC2 -dependent transfer (CRL2-family members are activated by post-translational modification with the UBL NEDD8 through a process called neddylation) of labeled UB (*UB) from UBE2R2. In the absence of substrate, KLHDC2 is rapidly ubiquitylated. However, instead of auto-ubiquitylation of KLHDC2, neddylated CRL2 KLHDC2 preferentially promotes *UB ligation to SELK or USP1 when present in the reaction mixtures (Figures 1C and 1D). Surprisingly, despite the ability of free NEDD8 to bind KLHDC2's SBD, little *UB transfer to NEDD8 was observed. Instead, KLHDC2 was still auto-ubiquitylated to the extent observed in assays lacking substrate (Figure 1C). The results suggest that (1) complex formation with KLHDC2's SBD is necessary but not sufficient for substrate ubiquitylation, (2) the CRL2 KLHDC2 complex has the means to distinguish between SBD protein interactors and bona fide substrates, and (3) the latter protect KLHDC2 from autoubiquitylation.

## KLHDC2 C terminus is a C-degron mimic

To gain insights into how the KLHDC2-EloBC substrate receptor complex is regulated, we determined its crystal structure at 2.6 A Ëš resolution (Table 1). Unexpectedly, the SBD of each KLHDC2 protomer was occupied by the C terminus from the adjacent KLHDC2 in the crystal. The C-terminal Ser, penultimate Gly, and preceding 4 residues superimpose with true substrate degrons bound to KLHDC2's isolated SBD (Figures 2A, S1A, and S1B). 29 To rule out the possibility that the interaction was artifactually imposed by crystal packing, the observed binding mode was confirmed with another crystal structure of the isolated SBD bound to a peptide corresponding to the KLHDC2 C terminus (Figure 2B, Table 1).

Measuring affinities of interactions between the KLHDC2 SBD and peptides by surface plasmon resonance (SPR) showed that the wild-type (WT) KLHDC2-derived peptide bound to the KLHDC2 SBD with a KD of 1.4 m M (Figure 2C). This is only 3-5fold lower affinity than KD values for peptides from physiological substrates, USP1's N-terminal UBL domain ( KD of 0.26 uM) or GFP-RIC8B ( KD of 0.46 uM, Figure S1C). Interestingly, a single amino acid replacement of the KLHDC2 C-terminal Ser with Gly increased the affinity by more than three orders of magnitude, owing to a much slower off rate (Figure 2C). As predicted from the structure, mutation of the Gly-Ser motif to Lys-Lys completely abolished binding (Figure 2C). Thus, the KLHDC2 C

terminus mimics a degron, but with a sequence motif of intermediate affinity.

## KLHDC2-EloBC forms a substrate mimic-induced tetramer

The ability of the KLHDC2 degron mimicking C terminus (hereafter referred to as C-degron-mimic) to engage another molecule of KHLDC2's SBD suggested that such interactions could avidly stabilize the self-assembly of the substrate receptor complex. To test for oligomerization in solution, we employed size-exclusion chromatography coupled with multi-angle light scattering. The data revealed an estimated mass of 280.8 kDa for the complex, which matches the calculated mass of a tetrameric KLHDC2EloBC complex (282.1kDa, Figure 3A, hereafter we refer to

''tetramer'' as four KLHDC2-EloBC complexes in a singular assembly, and ''monomer'' as one KLHDC2-EloBC complex). Performing sedimentation velocity analytical ultracentrifugation (SV-AUC) at various concentrations revealed a dilution-dependent decrease in tetramer population concomitant with increasing levels of monomer (Figure 3B). The data yielded an estimated KD value for tetramer dissociation of /C24 150 pM. Thus, the KLHDC2-EloBC substrate receptor complex forms an exceptionally stable tetrameric assembly.

The KLHDC2-EloBC X-ray structure revealed the molecular mechanism of tetramer formation: the asymmetric unit contained two KLHDC2-EloBC complexes that together with two flanking complexes in the crystallographic lattice formed a tetramer (Figures 2A, S1A, and S1B). We term the non-equivalent

KLHDC2-EloBC complexes as protomers A or B and symmetryrelated protomers A' and B 0 (Figure 2A). Each protomer is connected to two others by interactions between SBDs and KLHDC2 C-degron-mimics. For instance, the C-degron-mimic of protomer A occupies the SBD of protomer B, and the SBD from A is occupied by the C-degron-mimic of protomer B 0 . As such, the tetrameric arrangement is closed since all SBDs are engaged (Figure 2A).

Although each copy of the KLHDC2 SBD and the BC-boxEloBC complex superimposed with the prior structures of these domains (Figures S1D and S1E), 29,39 the helices emanating from the BC-box and preceding the C-degron mimic approach protomersAandB(orA 0 and B 0 ) from different orientations (Figure 3C). The distinct trajectories depend on inter-protomer interactions that establish the specific closed tetrameric arrangement (Figure 3D). The 6 C-terminal residues of KLHDC2 A are guided into the substrate-binding pocket of KLHDC2 B by two unique sets of interactions. First, the backside of the BC-box helix of KLHDC2 A, opposite the face bound to EloBC, interacts with the b -propeller SBD from KLHDC2 B (Figure 3E, left panel). Also, the CUL2-box from subunit A forms an a -helical structure that is buttressed on both sides by KLHDC2 B (Figure 3E, left panel). Second, the b -propellers from protomers B and A 0 are arranged in a head-to-tail fashion that creates an extensive interaction surface between solvent-exposed residues from Kelch-repeat 5 (KR5) and KR6 from protomer B, with those from KR1 and KR2 from protomer A 0 (Figure 3E, right panel). As a result, instead of extensive interactions with a nearby protomer, the backside of the BC-box of KLHDC2 B interacts with the extreme C-terminal sequence of its own CUL2-box, thereby stabilizing its C-degron-mimic proximal to the substrate-binding pocket of KLHDC2 A 0 (Figures 3C and 3E, right panel). The KLHDC2 B-A' subunit interaction is further stabilized by loop elements from KLHDC2 A' interacting with elements from the BC-box and EloC from protomer B (Figure 3E, right panel).

Within the interactions described above, two tetramerization elements (TEs) define the specific geometry. The formation of the closed tetramer would prevent the assembly of filament structures by unchecked head-to-tail binding between KLHDC2 SBDs and C-degronmimics(FigureS2A).Thetetramericstructuresuggested that a loop within KLHDC2 KR4 serves as one TE (TE1), displaying several residues including Phe177, Phe182, and Trp183 that collectively interact with TE2 (Ile373 and the backside of the BC-box) of an adjacent KHLDC2 subunit (Figure 3E, left panel).

The roles of KLHDC2 sequences predicted to stabilize the tetramer were characterized through mutagenesis and monitoring the migration pattern of these mutants by non-denaturing gel electrophoresis. Deleting KLHDC2's C-terminal 6 residues

(C D 6 mutant, Figure 3E), either alone or together with the CUL2-binding 'CUL2-box' ( D C2B), converted KLHDC2-EloBC to a monomeric species (Figure 3F). Notably, the C D 6 mutant also terminates with Gly-Ser, suggesting the length of the KLHDC2 C terminus has evolved to engage the adjacent subunit (Figure 3E). A mutant with the KLHDC2 C-terminal Gly-Ser dipeptide replaced with Lys-Lys (-KK) also migrated as a monomer (Figure 3F, confirmed by AUC, Figure S2B), as does a mutant in the SBD (S269E) previously demonstrated to disrupt C-degron binding (Figure 3F). 29

Mutation of the C-degron mimics to the optimal degron sequence Gly-Gly (-GG) resulted in multiple species whose gel migrations were indicative of a range of oligomers from dimers to assemblies larger than tetramers (Figure 3F). Importantly, the mutation of TE1 or TE2 was destabilizing leading to the appearance of trimeric and dimeric species (Figure 3F). These results demonstrate that KLHDC2's C-terminal degron mimic is indispensable for oligomerization, but additional elements constrain the assembly to the closed tetramer thwarting the formation of varying oligomeric species.

The tetrameric KLHDC2-EloBC assembly is shared with at least one closely related CRL2-based C-degron substrate receptor, KLHDC3 (Figure S3). As predicted from its C-terminal -His-Gly sequence, purified KLHDC3-EloBC migrated with a mass consistent with a tetrameric assembly (Figures S3B and S3C). On the other hand, KLHDC10-EloBC was monomeric, consistent with the divergence of its C-terminal sequence from its cognate degron (Figures S3B, S3I, and S3J).

## The KLHDC2-EloBC tetramer is autoinhibited for substrate binding

Several experiments indicated that the KLHDC2-EloBC self-assembly is autoinhibited for substrate binding. First, the potential for KLHDC2-EloBC to interact with a fluorescein-labeled fragment fromthesubstrateSELK(FAM-SELK)wasexaminedbynon-denaturing gel electrophoresis. FAM-SELK alone was not observed in the gel, presumably due to its small size ( /C24 5,700Da) and high chargedensity (pI = 11.5). Upon mixing with KLHDC2-EloBC a single band appeared (Figures 4A and S2C). A substantially greater signal was observed with monomeric -KK mutant. Similar migration in the gel raised the possibility that the WT KLHDC2-EloBC complex was also monomeric when bound to FAM-SELK (Figures 4A and S2C). Interestingly, a FAM-SELK band was not observed with the stabilized -GG mutant or with the negative control degron-binding-deficient S269E mutant (Figures 4A and S2C-S2E).

The data raised the possibility that FAM-SELK could capture monomericKLHDC2-EloBCpresentfromslowtetramer/monomer

interconversion at equilibrium. This model predicts that the KLHDC2-SELKinteractionshouldincreasewithbothgreater incubation time and increasing substrate concentration, which were both confirmed in experiments with WT KLHDC2-EloBC (Figures 4A, 4B, and S2F). However, neither an 18-h pre-incubation nor a 4-molar excess of FAM-SELK was sufficient to achieve the level of interaction observed with KLHDC2-EloBC -KK. The identity of the substrate and its affinity for isolated KLHDC2 SBD also appeared to be important (Figures 1B and S1C). Compared with FAM-SELK, for which a 4-fold molar excess allowed substantial complex formation with KLHDC2-EloBC, a 20-fold molar excess of FAM-USP1 was required for only minimal interaction (Figures 4B and S2F).

To further explore the parameters of conversion from tetrameric KLHDC2-EloBC to substrate-bound monomer, we performed analytical size-exclusion chromatography after various incubation periods for KLHDC2-EloBC and substrate. Without incubation, the tetrameric form of KLHDC2-EloBC and MBPfused SELK fragments migrate in discrete fractions across an SD200 column. Mixing the two for extended time periods led to the migration of a stoichiometric MBP-SELK:KLHDC2EloBC complex (Figure S2G). Modeling the distribution of tetramer and MBP-SELK-KLHDC2-EloBC species under various incubation times enabled the computational estimation of the rates of tetramer dissociation and association (Figures S2G and S2H). Interestingly, the estimated KD (180-260 pM) agrees with the experimentally derived value from SV-AUC experiments performed in the absence of substrate ( KD = 158 pM, Figure 3B), suggesting that substrate does not directly stimulate tetramer dissociation. Furthermore, the range of estimated kinetic rates for tetramer dissociation (2-2.5 3 10 5 s -1 ) suggests KLHDC2EloBC tetramers dissociate with a half-life of &gt;12 h, consistent with the time course of substrate binding observed in non-denaturing gel electrophoresis (Figures 4A and S2C).

The prolonged incubation times required for substrate binding to KLHDC2-EloBC seemed to exceed a biologically relevant time scale. Thus, we considered if substrate binding may synergize with other facets of an active CRL2 KHLDC2 E3 ligase. Indeed, in the non-denaturing gel mobility shift assay, adding CUL2-RBX1 reduced the required pre-incubation time for KLHDC2-FAMSELK complex formation to &lt;3 h (Figure 4C, left panel), whereas a monomeric neddylated CRL-KLHDC2-EloBC-substrate complex required no pre-incubation (Figure 4C, right panel). In summary, although KLHDC2-EloBC on its own slowly dissociates into a monomer capable of binding to substrates, dissociation was greatly accelerated in the neddylated CRL KLHDC2 complex that is activated for ubiquitylation.

Structural modeling provided a rationale for the accelerated dissociation. First, we superimposed EloBC-CUL2-RBX1 from the prior structure of CRL2 VHL onto the corresponding region of the KLHDC2-EloBC tetramer, which showed the CUL2 binding sites are only accessible in the B and B' protomers (Figures S4A and S4B). 40,41 Second, we confirmed this arrangement by SEC-MALS (multi-angle light-scattering) (Figures S4C and S4D). For the KLHDC2 -KK monomer bound to either CUL2-RBX1 or NEDD8 /C24 CUL2-RBX1, the observed masses of 170.3 and 182.7 kDa agrees with the calculated mass of a 1:1 complex (Figures S4C and S4D). For WT KLHDC2-EloBC-CUL2-RBX1, the peak was broad and deconvoluted into at least two distinct complexes. The leading edge of the peak with an estimated mass of 456.7 kDa is within experimental error of the calculated molecular mass for a KLHDC2-EloBC tetramer bound to two copies of CUL2-RBX1 ( /C24 480 kDa). The lagging edge of the peak corresponded to an estimated mass of 427.5 kDa. Although we cannotunequivocallydefinethiscomplexassembly,theestimated mass is consistent with a trimeric KLHDC2-EloBC ( /C24 210 kDa) bound to two CUL2-RBX1 subcomplexes ( /C24 200 kDa). For WT KLHDC2-EloBC-NEDD8 /C24 CUL2-RBX1, the estimated mass was388kDa,consistentwithatetramericKLHDC2-EloBCassembly ( /C24 280 kDa) bound to one NEDD8 /C24 CUL2-RBX1 moiety ( /C24 110 kDa). In summary, the data are consistent with dynamic assembly between KLHDC2-EloBC and CUL2-RBX1, resulting in a sample that contained both two CUL2-RBX1 bound to a KLHDC2-EloBC tetrameric assembly and another sub-complex, seemingly of this where one KLHDC2-EloBC entity has dissociated. These results are consistent with accelerated dissociation of the KLHDC2-EloBC tetramer in the presence of CUL2-RBX1. In addition, the predominant complex of KLHDC2-EloBC tetramer with one NEDD8 /C24 CUL2-RBX1 is consistent with the notion that clashing between the tetrameric assembly and both NEDD8 and the CUL2 and RBX1 domains (that become mobilized by neddylation) reduces the overall stability relative to the 4:2 KLHDC2EloBC:CUL2-RBX1 assembly. Indeed, analytical sizing runs of WT KLHDC2-EloBC NEDD8 /C24 CUL2-RBX1 with or without the SELK fragment confirm that NEDD8 /C24 CUL2-RBX1 accelerates KLHDC2-EloBC tetrameric disassembly (Figures S4E and S4F), although peak overlap between oligomeric KLHDC2EloBC-NEDD8 /C24 CUL2-RBX1 and substrate-bound monomeric KLHDC2-EloBC-NEDD8 /C24 CUL2-RBX1 prevented accurate quantification of these species precluding kinetic modeling as in Figures S2G and S2H.

To further corroborate these ideas, we obtained an 8.2 A Ëš resolution cryo electron microscopy (EM) map of CRL2 KLHDC2 (Figures 4D and S5A). The EM density was readily fit with the

crystallographically determined KLHDC2-EloBC tetramer, and CUL2 bound to protomers B and B 0 . As is typical for CRL structures, density for the catalytic regions-CUL2's C-terminal ''WHB'' domain and RBX1's RING domain-was visible only at low contour and resolution, precluding the unambiguous orientation of these domains. Nonetheless, they are adjacent to EloBC and KLHDC2, respectively, from protomer A or A 0 (Figure 4D). By analogy to other CRLs in which the cullin WHB and RBX1RINGdomainsareflexibly tethered, it seems likely that positions accessible to these domains would sterically clash with the tetramer. 42-44 This would be further exacerbated for the neddylated complex in which NEDD8, the CUL2 WHB domain to which it is linked, and RBX1's RING domain would all be expected to sample multiple conformations. 42-46

To gain insights into potential catalytic functions of tetramerization, we superimposed a prior structure of a CRL2 substrate receptor bound to a protein substrate (a PROTAC-mediated complex between VHL-EloBC and second bromodomain from BRD4) onto protomer B, which can bind CUL2. 47 This comparison revealed that KLHDC2 A' recruited to KLHDC2 B is similarly positioned as a substrate (Figure 4E), suggesting that KLHDC2 tetramerization could promote its auto-ubiquitylation. Indeed, ubiquitylation assays showed that only versions of KLHDC2 that oligomerize (WT without substrate and -GG with or without substrate) are subject to auto-ubiquitylation (Figure 4F), whereas the obligate monomer KLHDC2 -KK is only marginally auto-ubiquitylated even in the absence of substrate.

## A slow on rate protects NEDD8 from ubiquitylation by CRL2 KHLDC2

The structural and biochemical results hint at the intricate interplay between the degron-mimic-mediated KLHDC2 tetramer and monomer capture by the substrate. To further explore why NEDD8 was a relatively poor ubiquitylation substrate in pulsechase assays despite its comparable affinity with USP1 for the KLHDC2 SBD, rapid quench flow was employed to quantify pre-steady-state kinetic parameters. In reactions with KLHDC2 -KK and following a 1-h equilibration of the substrate with E3, the rate of NEDD8 modification was surprisingly faster than for the bona fide substrates USP1 and SELK (Figures S6A-S6D, kobs 0.30s-1 NEDD8, 0.21s-1 USP1, and 0.10s-1 SELK). Thus, NEDD8 has the potential to serve as a substrate if sufficient time is provided for the formation of a KLHDC2-substrate complex (Figures S6A-S6D).

Next, qualitative ubiquitylation reactions were performed comparing *UB transfer to USP1 or NEDD8 after varying pre-incubation periods between substrate and NEDD8 /C24 CRL2 KLHDC2 . Maximal ubiquitylation of USP1 was achieved after a 5-min preincubation, whereas under identical conditions with NEDD8, the majority of *UB partitioned to KLHDC2 (Figures 5A and 5B). Even a 30-min pre-incubation with NEDD8 was not sufficient to fully direct *UB conjugation to NEDD8 (Figure S6E). With KLHDC2-KK,maximalubiquitylation of UPS1 required no pre-incubation, whereasthepre-incubationrequiredforNEDD8ubiquitylation was reduced to &lt;5 min (Figures 5A and 5B).

The extended pre-incubation requirement suggested that NEDD8 associates with KLHDC2 relatively slowly. Indeed, the association rate of NEDD8, as measured by SPR, was an order-of-magnitude slower (Figure 5C, 5.5 3 10 3 M /C0 1 s /C0 1 ) than that of USP1 or SELK (4.9 3 10 4 and 3 3 10 4 , respectively). Despite NEDD8's slow rate of association, its equilibrium binding affinity is comparable with USP1 (211 versus 264 nM), due to an order-of-magnitude slower rate of dissociation (Figure 5C).

Structural analysis suggested that NEDD8's C-terminal tail might be too short to enable a rapid association with KLHDC2. 22,37 This was tested for with a panel of NEDD8 variants extending the C-terminal tail by inserting 2, 4, or 6 residues (1, 2, or 3 GlySer repeats, respectively) upstream of the -GG. SPR measurements confirmed that the NEDD8 mutants rescued the rate of association with KLHDC2 (Figure 5D). Consistent with this, the insertion variantswerereadilyubiquitylatedwithoutpre-incubation (Figures 5E and 5F). Taken together, the data indicate that C-degron mimicry-induced KLHDC2 tetramerization excludes NEDD8 as a substrate due to its slow on rate, whereas bona fide substrates possess on rates that are sufficient to capture the monomeric form of neddylated CRL2 KLHDC2 within the monomer-tetramer equilibrium.

## Multimodal effects of C-degron mimic-mediated tetramerization on CRL2 KLHDC2 neddylation

Given the importance of neddylation in activating CRL-dependent ubiquitylation, we tested the effects of mutations impacting the C-degron mimic-mediated tetramerization on NEDD8 transfer to CRL2 KLHDC2 , the inhibition of this reaction by CAND1, and deneddylation mediated by the COP9 signalosome (CSN). Assays were performed with or without a C-degron peptide or a bona fide C-degron protein substrate, which both shift the tetramer-to-monomer equilibrium for WT CRL2 KLHDC2 , while also having complementary structural properties of being small (peptide) or large (protein).

The results showed a striking trend: monomeric versions of KLHDC2-EloBC, in some cases bound to a substrate protein, were preferentially neddylated. For example, the rate of CUL2RBX1 neddylation was greatest with the -KK monomeric mutant of KLHDC2-EloBC, with or without a bound substrate, and for the tetramer supplied with a substrate that captures the monomer (Figures 6A, left panel, and S7A). The same trend was also observed in the presence of CAND1, whose inhibition of neddylation of other CRLs has been shown to be overcome by their binding to substrate receptors (Figures 6A, right panel, and S7B).

The effects of tetramerization and substrate binding on deneddylation of NEDD8 /C24 CRL2 KLHDC2 were more intricate but agreed with structural modeling. Briefly, superimposing the EloBC-CUL2 portion of the complex on a prior cryo-EM map of CSN-bound to neddylated CRL2 VHL showed substantial clashing between CSN with the tetramer, explaining its inhibition of deneddylation (Figures 6B and S7C). 45 By contrast, less clashing is observed with a monomeric NEDD8 /C24 CRL2 KLHDC2 , explaining increased deneddylation rates of the monomeric -KK mutant, and of this mutant and WT NEDD8 /C24 CRL2 KLHDC2 in the presence of a small C-degron peptide (Figures 6B and S7C). As with other CRLs, the binding of a larger protein substrate would substantially obstruct CSN, thus explaining why a substrate protein (MBP-SELK) maintains NEDD8 on monomeric versions of CRL2 KLHDC2 (Figures 6B and S7C). 48,49

The increased rate of CUL2 neddylation in the presence of monomeric versions of KLHDC2-EloBC was unexpected as cullin neddylation is not typically influenced in vitro by substrate receptors. However, modeling a CRL2 KLHDC2 neddylation complex by superimposing the prior structure of the NEDD8 ligation machinery onto monomeric KLHDC2-EloBC showed that residues corresponding to KLHDC2 TE1, which are buried in the tetramer but exposed in the monomer, are proximal to UBE2M

(which catalyzes NEDD8 transfer to cullin, Figure 6C). 50 Michaelis-Menten kinetics on the neddylation reaction revealed that, in comparison to the values for CUL2-RBX1 alone, adding monomeric KLHDC2-EloBC -KK resulted in a /C24 17-fold increase in the maximal rate of neddylation (Vmax.) with little impact on the Michaelis constant (Km) (Figure 6D). Furthermore, although similar rates of neddylation were achieved by adding substrate to WT CRL2 KLHDC2 (Figure 6A, left panel), this was largely

(A) Normalized rates of CUL2 Neddylation by NEDD8 /C24 UBE2M in the absence (left panel) or presence (right panel) of CAND1 and with WT or the indicated KLHDC2-EloBC C-degron mutants and with or without SELK peptide.

(D) Velocity of CUL2 Neddylation versus NEDD8 /C24 UBE2M concentration. Rates of NEDD8 /C24 CUL2 formation were measured in the presence of CUL2 alone (circles) or CUL2 KLHDC2 (-KK). Data were fit to the Michaelis-Menten model using GraphPad Prism software.

(E) Same as in (A) but monitoring the stimulation of CUL2 Neddylation in the presence of WT or the indicated KLHDC2EloBC mutants with or without SELK. All results represent replicates of n = 3.

abrogated by mutation of KLHDC2 TE1 (Figures 6E and S7D). Importantly, the activation of neddylation by monomeric KLHDC2-EloBC required binding to CUL2, as a truncated version of CUL2-RBX1 that is competent for neddylation but cannot bind substrate receptors did not show KLHDC2-dependent stimulation of CUL2 neddylation (Figure S7E). Thus, the monomeric CRL2 KLHDC2 complex seems specifically arranged in a manner to promote the formation of the catalytically competent conformation of neddylation enzymes, although future studies will be required to determine the precise structural mechanism.

## Cellular roles for KLHDC2 C-degron mimicry

We sought to gain insights into the potential roles of KLHDC2's C-degron mimic mediating self-assembly in cells. Analysis of whole-cell lysates from transiently transfected 293T cells by gel filtration chromatography showed KLHDC2 migrating across numerous fractions (from &lt;660,000 to 75,000 Da, Figure S8A), presumably due to its bewildering number of interacting partners. We thus tested whether mutations affecting tetramer formation in vitro resulted in similar outcomes in cells.

We first examined the impact on CUL2 deneddylation. After confirming that CSN activity is maintained in lysis buffer conditions (Figure S8B), we performed transient transfections of 293T cells with FLAG-tagged WT or mutant KLHDC2, lysates were prepared in either the absence or presence of excess of SELK peptide, and with or without pharmacological inhibitors of neddylation (MLN4924) or deneddylation (CSN5i). Anti-FLAG immunoprecipitants were analyzed by immunoblotting for associated proteins. In the absence of any treatment, WT KLHDC2 co-precipitated a mixture of unmodified and neddylated CUL2. However, the monomeric -KK KLHDC2 mutant co-immunoprecipitated unneddylated CUL2, whereas the stabilized -GG KLHDC2 mutant preferentially associated with fully neddylated CUL2 (Figure 7A). Notably, as with experiments performed in vitro with purified components (Figure 6B), the addition of SELK peptide-stimulated CUL2 deneddylation in complexes with WT KLHDC2 and the inclusion of CSNi5 confirmed CSN as responsible for CUL2 deneddylation (Figure 7A).

We also examined the effects of tetramerization mutants on KLHDC2 protein levels in U2OS cells. After the CRISPR-mediated knockout of endogenous KLHDC2, stable cell lines were generated via lentiviral transduction of FLAG-tagged WT or mutant KLHDC2. Although the endogenous expression levels of KLHDC2 in U2OS cells are low, such that the exogenous

FLAG-tagged WT KLHDC2 protein is expressed at roughly 10-fold higher, the amount of FLAG-KLHDC2 matched endogenous levels seen in other cell lines such as PC-3 and HepG2 (Figure S8C). Notably, the normalized protein levels for the KLHDC2 variants differed, where both WT and -GG KLHDC2 protein levels were significantly lower than that of C D 6 and -KK monomers. The D C2B KLHDC2 variant displayed the highest protein levels, presumably due to its failure to interact with CUL2 preventing auto-ubiquitylation. Consistent with this notion, the treatment of the cell lines with MLN4924 stabilized both tetrameric and monomeric KLHDC2 variants to levels comparable with that of D C2B KLHDC2 (Figure 7B). Cell lines containing KLHDC2 variants were also treated with the proteasome inhibitor bortezomib, followed by FLAG immunoprecipitation and the detection of poly-ubiquitylated KLHDC2 by anti-K48 specific ubiquitin antibodies (Figure S8D). The KLHDC2 tetramer and stabilized -GG mutant was indeed more ubiquitylated than their monomeric counterparts, although surprisingly, D C2B KLHDC2 was also ubiquitylated, presumably in a CUL2-independent manner as MLN4924 treatment fails to stabilize this mutant (Figure 7B). Taken together, the cellular data are consistent with the in vitro biochemical data indicating that the C-degron mimicry-mediated tetramerization of KLHDC2 inhibits CSN-mediated deneddylation of CUL2 and promotes its auto-ubiquitylation.

Next, we tested the mutants for effects on substrate degradation. Steady-state levels of the USP1 N-fragment were marginally but consistently increased in the KLHDC2 knockout cell line, whereas full-length USP1 levels were unchanged (Figures 7C and 7D). The exogenous rescue of WT KLHDC2 resulted in an approximately 4-fold reduction of USP1 N-fragment levels, whereas the levels were unchanged upon the expression of D C2B KLHDC2, which neither tetramerizes nor binds CUL2. Importantly, expressing monomeric C D 6 and -KK versions of KLHDC2 further reduced levels of USP1 N-fragment (Figures 7C and7D).Theseresults are consistent with the tetrameric assembly downregulating CRL2 KLHDC2 activity.

Experiments with mutant cell lines also suggested that the KLHDC2 oligomeric assembly affects NEDD8 levels in cells. Unlike USP1, the deletion of KLHDC2 in U2OS cells had no significant effect on NEDD8 protein levels, suggesting KLHDC2 levels are finely tuned such that NEDD8 is unperturbed by its presence in cells. However, ectopic expression of WT KLHDC2 reduced the levels of unconjugated NEDD8 (Figures 7E and 7F). Expression of monomeric -KK KLHDC2 resulted in the further decrease of NEDD8 protein levels, whereas both -GG

and D C2B KLHDC2 expression restored NEDD8 to levels observed in WT U2OS cells (Figures 7E and 7F). In summary, these results support the notion of a synergistic relationship between the capacity of KLHDC2 to form a C-degron mimicmediated self-restrained tetramer and the absolute levels of both KLHDC2 and NEDD8 in cells.

## DISCUSSION

Here, we show that the C-degron-recognizing CRL2 substrate receptor KLHDC2 terminates with a sequence mimicking a C-degron. The C-degron mimic mediates E3 ligase self-assembly into an inactive homo-tetramer controlling the selection of protein substrates for ubiquitylation while protecting NEDD8, whose degron-like sequence paradoxically can bind the SBD with high affinity.

The regulation of C-degron recognition by KLHDC2 relies on the interconversion of an inactive KLHDC2-EloBC tetramer and an active monomeric form. The C-degron mimic, in combination with additional elements, stabilizes the tetramer by occupying the KLHDC2 SBD of an adjacent protomer. As such, KLHDC2EloBC tetramers cannot engage C-degron substrates and require dissociation to the monomeric form for substrate binding. However, it seems that the equilibrium between these two species is controlled by both CUL2-RBX1 and substrate (Figures 4A, 4B, and 4C, left panel), as well as the neddylation state of CUL2 (Figure 4C, right panel). We propose that protein substrates capture the monomeric KLHDC2-EloBC-CUL2RBX1 complex (CRL2 KLHDC2 ) preventing the reformation of the tetramer. In this way, substrate-binding also activates the CRL-substrate complex since a KLHDC2 surface mediating tetramerization becomes exposed and potentiates neddylation (Figures 6C and 6E). In the absence of substrate, the monomer is subjected to CSN-mediated deneddylation and inactivation (Figures 6B and 7A), whereas the tetramer is subject to autoubiquitylation (Figures 4F and 7B).

Adefining feature emerging from this work is that KLHDC2 tetramerization introduces an additional barrier for substrate recruitment that includes the rates of tetramer formation and dissolution. The combination of the kinetics of tetramerization with the rate of substrate association results in a phenomenon knownas kinetic amplification. Our model shares similar aspects with kinetic proofreading, a mechanism where non-substrates display increased rates of dissociation from enzymes relative to real substrates and are coupled to pseudo-irreversible energy-consuming steps in the reaction scheme. 51,52

Here, specificity for CRL2 KLHDC2 appears to be determined by the association rate of the substrate relative to the rates that control the KLHDC2-EloBC tetramer-monomer equilibrium. Without the self-assembly/disassembly steps and kinetic amplification, non-substrates such as NEDD8 would eventually gain access to the ligase even with their slow on rates (Figures 7G, S6AS6D, and 5B). Thus, it seems that only substrates that associate with CRL2 KLHDC2 monomers with kinetics more rapid than the rate of KLHDC2-EloBC tetramer formation become ubiquitylated. Non-physiological substrates like NEDD8 that associate more slowly with CRL2 KLHDC2 than the rate of tetramer reformation miss their window for poly-ubiquitin chain formation due to

KLHDC2 auto-ubiquitylation. The inclusion of a time barrier for substrate selection by CRL2 KLHDC2 , via auto-ubiquitylation, is like kinetic proofreading in that it will further amplify substrate selectivity beyond the differences in the association rates of substrates. Although additional selection strategies may also contribute, such as an evolutionary pressure for non-substrates to lack Lys residues on their surfaces that may serve as aberrant acceptors of UB conjugation, 53,54 we propose that the KLHDC2dependent molecular timer has an important role in protecting non-substrate proteins from degradation (Figures 7G and S8E).

For many CRLs, substrate binding also leads to CRL activation in cells. In these cases, the presence of substrates bound to CRLs controls neddylation, and thus E3 ligase activity, by virtue of sterically blocking CSN-mediated deneddylation. 48,49,55 In support of this, we found that a bulky substrate (MBP-fused SELK) inhibits CRL2 KLHDC2 deneddylation. However, the tetrameric KLHDC2-EloBC self-assembly also inhibits deneddylation. Thus, these two distinct E3 ligase assemblies mediate either substrate ubiquitylation or KLHDC2 auto-ubiquitylation.

With CRLs now firmly established as primary E3 ligases chosen for targeted protein degradation (TPD) efforts, we suggest that the TPD field consider the SR's of the C-degron pathway as intriguing candidates for promoting TPD due to their welldefined substrate-binding pockets. 56 Consistent with this, KLHDC2 has recently been proposed as a binding site for the highly in vogue proteolysis targeting chimeric (PROTAC) drugs. 57-59 Our insights into the regulation of KLHDC2 activity suggest that specific attention should be made to the association rates of KLHDC2 ligands to permit effective CRL2 KLHDC2 monomer capture for target degradation. Finally, our finding that C-degron-mimic-mediated self-assembly is conserved for the KLHDC2 paralog KLHDC3-EloBC suggests that such selfassociation could be a robust and evolutionarily successful means for kinetic amplification of E3 ligase terminal degron substrate selection.

## Limitations of the study

Given the number of potential KLHDC2 interacting partners of varying sizes in cells (&gt;80 identified in Biogrid), we were unable to directly demonstrate the existence of free tetrameric KLHDC2-EloBC in cells, presumably due to an extensive range of size distributions of KLHDC2-containing molecular complexes. Instead, the evidence of tetrameric assemblies comes from the expression of WT and mutant KLHDC2 variants and the comparison of their effects on myriad CRL-dependent assays.

## STAR + METHODS

Detailed methods are provided in the online version of this paper and include the following:

## SUPPLEMENTAL INFORMATION

Supplemental information can be found online at https://doi.org/10.1016/j. molcel.2023.01.019.

## ACKNOWLEDGMENTS

Wethank Ine Ë† s Chen, Daniel Bollschweiler, and Tillman Sch â‚¬ afer for their advice and assistance. This research was supported by ALSAC, St. Jude Children's Research Hospital, NIH P30CA021765 to St. Jude, NIH R01GM125885 (to D.C.S. and B.A.S.), NIH R01GM141409 (to J.L., N.P., and G.K.), NIH R01AG11085 and Howard Hughes Medical Institute (to S.J.E.), and MaxPlanck-Gesellschaft and the EU H2020 ERC Advanced Grant Nedd8Activate (to B.A.S. and K.B.).

## AUTHOR CONTRIBUTIONS

D.C.S., S.J.E., G.K., and B.A.S. conceptualized the study, directed the study, and wrote the manuscript. D.C.S. and M.T.K. performed and analyzed the crystallography, biochemical, and cellular experiments. K.B. performed the cryo-EM. C.T.G. and R.E.L. performed the SPR experiments. S.C. and T.C. performed the TR-FRET assays. A.N., S.V., and R.K. performed the SECMALS and SV-AUC experiments. J.L., N.P., and G.K. performed the presteady-state kinetic experiments.

## DECLARATION OF INTERESTS

B.A.S. is a member of the scientific advisory boards of Interline Therapeutics and BioTheryX. B.A.S. and D.C.S. are co-inventors of intellectual property licensed to Cinsano.

## REFERENCES

## STAR + METHODS

## RESOURCE AVAILABILITY

## Lead contact

Further information and requests for reagents may be directed to and will be fulfilled by the lead contact Daniel Scott (Danny.Scott@ stjude.org).

## Materials availability

Reagents generated in This study are listed in key resources table and are available upon request.

## Data and code availability

## EXPERIMENTAL MODEL AND SUBJECT DETAILS

Cells were maintained in DMEM (High glucose, L-glutamine, Pyruvate) with 10% fetal bovine serum.

## U2OS and SK-BR-3 Cells

Cells were maintained in McCoys 5A supplemented with 10% fetal bovine serum at.

## PC-3 Cells

Cells were maintained in F-12K supplemented with 10% fetal bovine serum at.

## HepG2

Cells were maintained at 37 /C14 C and 5 % CO2. All cell lines were routinely checked for mycoplasma contamination with LookOut mycoplasma PCR detection kit (Sigma).

## METHOD DETAILS

## Constructs

Expression constructs generated for This study were prepared by standard molecular biology techniques and coding sequences entirely verified. Mutant versions used in This study were generated by QuickChange (Stratagene). For expression of KLHDC2 B-propellor only domain, residues 15-361 of KLHDC2 were cloned into a pRSF DUET 6X His-MBP based vector to generate HisMBP-KLHDC2. For biotinylation of the B-propellor only domain, an AviTag sequence (GSMSGLNDIFEAQKIEWHEGS) and flanking GlySer linkers between the TEV cleavage site and KLHDC2 B-propellor ORF were inserted into the pRSF DUET based MBP-TEVKLHDC2 vector. For SPR experiments a N-terminal 10X Histidine tag was placed on the KLHDC2 B-propellor only domain by PCR and cloned into a pRSF DUET based vector. For expression of SELK and USP1 we cloned fragments, residues 43-C and residues 592-671 respectively, into a pGEX-TEV based vector. To express a GFP-RIC8B fusion protein we appended the C-terminal 23 amino-acids of RIC8B to the C terminus of GFP by PCR and cloned this fragment into a pGEX-TEV based vector. Clones for insect cell expression of full-length KLHDC2-EloB/C, mutant variants therof, full-length KLHDC3-EloB/C, or KLHDC10-EloB/C were first cloned into pLib via Gibson assembly. Cassettes were then generated via PCR as described (Weissman) and Gibson assembled into pBig1a to generate a single vector for co-expression of all components. Proper assembly into pBig1a was confirmed by PmeI and SwaI restriction digestion. To generate clones for transient transfection into 293T cells, a N-terminal FLAG tag was added by PCR to KLHDC2, or mutants, using the pLib vectors as templates and cloned into a pCDNA5 FRT based vector. For production of lentiviral particles to rescue KLHDC2 knockout in U2OS cells, we first clone FLAG-KLHDC2 variants into pDONR223 BP clonase gateway cloning and subsequently transferred to pHAGE DEST vector via LR clonase.

## Protein expression and purification

UBE2D2, UBE2M, UBE2R2, ARIH1, the NEDD8 E1 APPBP1-UBA3, and GFP-RIC8B (C-terminal 23 residues) were expressed in E. coli BL21 Gold (DE3) cells as GST fusion proteins. Fusion proteins were purified from cell lysates by glutathione affinity chromatography and liberated from GST by thrombin or TEV cleavage overnight at 4 /C14 C. Cleavage reactions were further purified by ion-exchange and size exclusion chromatography in 25mM HEPES, 200mM NaCl, 1mM DTT pH = 7.5 (Buffer A). NEDD8, UB, SELK

(residues 43-C), and USP1 (residues 592-671), (residues 5-73) were expressed in E. coli BL21 Gold (DE3) cells as GST- fusion proteins. Fusion proteins were purified from cell lysates by glutathione affinity chromatography and liberated from GST by thrombin or TEV cleavage during extensive dialysis overnight in Buffer A at 4 /C14 C. Cleavage reactions were passed back over a glutathione affinity resin to remove free GST and any remaining uncleaved GST-fusion protein. Protein collected in the flow fraction was concentrated with an Amicon Ultra filtration unit and further purified by size exclusion chromatography in Buffer A.

KLHDC2 b-propellor fold only domain (15-361), its Avi-tagged counterpart were, expressed in E. coli Rosetta2 cells, grown in Terrific broth, with a N-terminal 6XHis-MBP-TEV tag.Tagged protein was purified from cell lysates by Ni affinity chromatography. Bound proteins were eluted from the Ni resin with 250mM Immidazole. The 6XHis-MBP tag was liberated by cleavage with TEV overnight at 4 /C14 C. Cleavage products were further purified by ion-exchange and size exclusion chromatography in Buffer A. KLHDC2s b-propellor fold only domain harboring a N-terminal 10X Histidine tag was expressed and purified as described above for the HisMBP verions.

Full-length CUL1-RBX1 full-length was expressed in insect cells as GST-TEV fusion proteins. Fusion proteins were purified from cell lysates by glutathione affinity chromatography and GST liberated by TEV cleavage overnight at 4 C. Cleavage reactions were further purified by ion exchange and size exclusion chromatography in Buffer A.

Full-length KLHDC2-EloB/C, KLHDC3-EloB/C, KLHDC10-EloB/C and mutant variants therof wer expressed in insect cells with a N-terminal 6XHis tag on EloC. Tagged proteins were purified from cell lysates by Ni affinity chromatography and further purified by ion exchange and size exclusion chromatography in Buffer A.

Neddylation and purification of CUL2-RBX1, were prepared as described previously for CUL1-RBX1. 50 Briefly, the final concentrations of components in the neddylation reactions were as follows: 12 m MCUL2-RBX1, 1 m MUBE2M, 0.1 m MAPPBP1-UBA3, and 20 m MNEDD8in25mMHEPES,200mMNaCl,10mMMgCl2,1mMATP,pH=7.5.Reactionswereinitiatedatroomtemperaturebythe addition of NEDD8 and incubated for ten minutes prior to quenching by the addition of DTT to 10mM. Quenched reactions were spun at 13K rpm for 10 minutes and immediately applied to a Superdex SD200 column to purify NEDD8-CUL2-RBX away from reaction components.

## Protein modifications

Reaction conditions for biotinylation of Avi-KLHDC2 b-propellor fold only were initially established in small scale pilot experiments and the efficiency of Avi-KLHDC2 biotinylation was assessed by super shift of biotin-KLHDC2 in SDS-PAGE after the addition of a stoichiometric excess of avidin (Sigma Aldrich A9275). Biotinylation reactions contained 50 m M Avi-KLHDC2 and 1.5 m M BIRA in 10 mM Tris, 10 mM ATP, 10 mM magnesium acetate, 2 mM Biotin, pH 8.0 and were incubated at room temperature for 1 hours and continued overnight at 4C. The reaction mixture was diluted four-fold into 25 mM Tris, 1 mM DTT, pH 8.0 and purified over a HiTrap Q ion-exchange column. Fractions containing biotin-KLHDC2 were pooled, concentrated, and further purified by size exclusion chromatography in 25 mM HEPES, 200 mM NaCl, 1 mM DTT, pH 7.5.

Reaction conditions for the sortase mediated protein ligation of fluorescent peptides were initially established in small scale pilot experiments with the efficiency of labeling determined by fluorescent scanning of gels. Reactions to label the amino-terminus of SELK and USP1 with fluorescein contained 50 m M SELK or USP1, 2uM sortase, 200 m M FAM-(PEG)5-LPETGG peptide in 50 mM Tris, 150 mM NaCl, 10 mM CaCl2. After a thirty-minute incubation samples were buffer exchanged over a NAP-5 column into 25mMHEPES,200mMNaCl,1mMDTT,pH7.5(BufferA)toremoveunconjugatedfluorescentpeptide. Samples were concentrated and further purified over a Superdex SD75 sizing column in Buffer A. To label the amino-terminus of CUL2 with TAMRA reactions contained 25 m MCUL2-RBX1, 1uM sortase, 150 m MFAM-(PEG)5-LPETGG peptide in 50 mM Tris, 150 mM NaCl, 10 mM CaCl2. After two-hour incubation samples were buffer exchanged over a NAP-5 column into Buffer A to remove unconjugated fluorescent peptide. Samples were concentrated and further purified over a Superdex SD200 sizing column in Buffer A. Extent of labeling was calculated through the ratio of protein concentration by absorbance at 280 nm to the concentration of fluorescent probe. In all cases labeling was estimated to be &gt;85%.

To introduce a cysteine for fluorescent labeling of NEDD8, UB and K0UB we mutated the protein kinase a site in the pGEX2TK backbone converting the PKA site from RRASV to RRACV 50 . NEDD8, UB or UBK0 purified from this expression construct were labeled with Fluorescein-5-Maleimide , AlexaFluor 488 Maleimide, or Fluorescein-5-Maleimide respectively as previously described 50 . Briefly, DTT was added to UB or UBK0 at a final concentration of 10mM and incubated on ice for 20 minutes to completely reduce cysteines for labeling. DTT was removed by buffer exchange over a NAP-5 column (GE Healthcare) in labeling buffer (25mM HEPES, 200mM NaCl). Labeling reactions consisted of UB or UBK0 at 150 m Mfinal concentration and were initiated by the addition of 600 m M Fluorescein-5-Maleimide or Tetramethylrhodamine-5-Maleimide (4X excess over labeling target and &lt;5% final DMSO concentration). Reactions were incubated at room temperature for 2 hours and quenched by the addition of DTT to 10mM. Quenched reactions were desalted over a PD-10 column in labeling buffer containing 1mM DTT to remove unreacted probe. Desalted protein was concentrated in an Amicon Ultra filtration unit and further purified over a Sephadex SD75 column.

## Cell biological experiments

To monitor the existence of the C-degron mimic KLHDC2-EloB/C tetrameric assembly of cells we took advantage of the knowledge that the COP9 signalosome remains active and can deneddylated NEDD8 /C24 CULinnon-denaturing lysates. We transfected 293T cells with Lipofectamine 3000 per the manufacture's recommendations with the indicated pDCNA5-FRT based vectors. Forty-eight hours

later, cells were harvested and lysed in 50 mM Tris, 150 mM NaCl, 0.5% NP-50, 1X HALT protease and phosphatase inhibitor cocktail, 250 U universal nuclease, pH 7.5 with or without the addition of 10 m MCSN5i, 10 m MMLN4924,and/or 100 m MSELK Peptide . Cells were lysed for 5 minutes on ice with repeated pipetting, spun at 14,000 rpm for 5 minutes to remove cell debris, and the supernatant subjected to a rapid FLAG immunoprecipitation (15 minutes). FLAG resin was washed four times with lysis buffer and proteins eluted with 2X SDS-PAGE sample buffer. Elution's were separated on 4-12 % Bis-Tris gels, transferred to PVDF membranes, and probed with the indicated antibodies. Western blots were developed with SuperSignal West Pico Plus substrate and chemiluminescent signal read on an ImageQuant LAS4000.

The KLHDC2 knockout cell line was generated by transfecting a mixture of KLHDC2 directed TrueGuide sgRNA (ThermoFisher, CRISPR988733\_SGM, TGATGGCAACGAGGATCTGC) and TrueCut Cas9 protein v2 with Lipofectamine CRISPRMAX transfection reagent according to the manufacture's recommendations. Forty-eight hours later cells were harvested and single sorted into 96-well plates. Plates were monitored over the course of /C24 3 weeks for single cell growth and expansion. Individual clones were subjected to Next Gen sequencing to identify correct knockouts. Stable cells lines for re-expressing FLAG-tagged WT or mutant KLHDC2 variants were obtained by lentiviral transduction of the KLHDC2 knockout line with lentivirus generated from pHAGE DEST vectors. Forty-eight hours after transduction cells were selected in complete media supplemented with 3 m g/ml puromycin for two weeks.

Samples for western blots were prepared by resuspension of cell pellets in lysis buffer consisting of 50 mM Tris, 150 mM NaCl, 0.5% NP-50, 6 M Urea, 10 mM Iodoacetamide, 0.1 % SDS, 1X HALT protease and phosphatase inhibitor cocktail, 250 U universal nuclease, pH 7.5. Samples were lysed on ice for 20 minutes and cellular debris cleared by a 10-minute spin at 14,000 rpm. The protein concentration of supernatants were determined by BCA protein assay. Equivalent total amounts of protein lysate were mixed with 2X SDS-PAGE sample buffer, were separated on 4-12 % Bis-Tris gels, transferred to PVDF membranes, and probed with the indicated antibodies. Western blots were developed with SuperSignal West Pico Plus substrate and chemiluminescent signal read on an ImageQuant LAS4000.

Toimmunoprecipiate UB conjugated variants of KLHDC2, the indicated U2OS WT, knockout, or KLHDC2 rescue lines were treated with bortezomib for (1 m M) for five hours. Cell pellets were harvested and lysed by resuspension in 50 mM Tris, 150 mM NaCl, 0.5% NP-50, 10 mM Iodoacetamide, 1X HALT protease and phosphatase inhibitor cocktail, 250 U universal nuclease, pH 7.5. Cells were lysed on ice for 20 minutes and cellular debris cleared by a 10-minute spin at 14,000 rpm. Supernatants were subjected to FLAG immunoprecipitation for 2 hours, washed four times with lysis buffer, and eluted with 2X SDS-PAGE sample buffer. Elutions were separated on 4-12 % Bis-Tris gels, transferred to PVDF membranes, and probed with the indicated antibodies. Western blots were developed with SuperSignal West Pico Plus substrate and chemiluminescent signal read on an ImageQuant LAS4000.

## TR-FRET Assay

TR-FRET assays were carried out in black 384-well microtiter plates at a final volume of 20 m L per well. An assay cocktail was prepared as a mixture of 55.5 nM Biotin-KLHDC2 SBD, 22.2 nM AlexaFluor488-RIC8B peptide, 2.77 nM Tb-Streptavidin (ThermoFisher) in assay buffer (25 mM HEPES, 100 mM NaCl, 0.1% Triton X-100, 0.5 mM DTT, pH 7.5). The assay cocktail was incubated for thirty-minutes at room temperature and 18 m l distributed to assay plates. Appropriate dilutions of SELK, USP1, NEDD8, NEDD8 G76D, UB, or buffer (2 m l) were added to assay plates. The assay mixture was incubated for 1 hour at room temperature prior to measuring the TR-FRET signal with a PHERAstar FS plate reader (BMG Labtech) equipped with modules for excitation at 337 nm and emissions at 490 and 520 nm. The integration start was set to 100 m s and the integration time to 200 m s. The number of flashes was fixed at 100. The ratio of 520/490 was used as TR-FRET signal in calculations. Assay endpoints were normalized to 100% (buffer only) and used for curve fitting in Prisim. Data curves in Figure 1B are fits to the average of n=3 independent experiments.

## Crystallography

For the structure of KLHDC2's SBD bound to a diGly consensus peptide from EPHB2, KLHDC2 (250 m M final concentration) was mixed with a 1.5 fold excess of peptide, and incubated on ice for 1 hour to equilibrate binding prior to setting crystal trays. Crystals grew at room temperature in 10-15% PEG3350, 0.2M NaSCN, 0.1 M Bis-Tris Propane, pH = 7.5. Crystals were harvested in mother liquor supplemented with 25% Glycerol prior to flash-freezing in liquid nitrogen. Reflection data were collected at Sercat 24-ID at the Advanced Photon Source. The crystals belong to space group P21 with two KLHDC2-peptide complexes in the asymmetric unit. Phases were obtained by molecular replacement using PHASER 60 searching for two copies of KLHDC2 (with peptide omitted) from 6DO2.pdb. Manual building was performed in COOT and refinement was performed using Phenix. 61 Additional details of the refinement are provided in Table 1.

Mixtures for crystals trays of the structure of KLHDC2's SBD bound to a 12mer peptide derived from the C-terminus of KLHDC2 were prepared as described above. Crystals grew at 4 C in 8-12% PEG3350, 0.2M NaSCN, 0.1 M Bis-Tris Propane, 5 nM Hexamine cobalt (III) chloride, pH = 7.5. Crystals were harvested in mother liquor supplemented with 25% MPD prior to flash-freezing in liquid nitrogen. Reflection data were collected at NECAT 24-ID-E at the Advanced Photon Source. The crystals belong to space group P21 with two KLHDC2-peptide complexes in the asymmetric unit. Phases were obtained by molecular replacement using PHASER 60 searching for two copies of KLHDC2 (with peptide omitted) from 6DO2.pdb. Manual building was performed in COOT and refinement was performed using Phenix. 61 Additional details of the refinement are provided in Table 1.

Crystals of the full-length KLHDC2-EloB/C (10 mg/ml) complex initially grew in 11-14% PEG3350, 0.2 M KSCN, 0.1 M Tris, pH =6.8 but diffracted poorly &gt;10A. We noticed that diffraction substantially improved ( /C24 5-6A) upon cryopreservation of the crystals in mother liquor supplemented with 25% MPD. Ultimately, diffraction quality crystals were grown by streak seeding into 9% PEG3350, 0.2 M KSCN, 0.1 M Tris, 0.1M Na/K Tartrate, 2.5% MPD, pH =6.8. Crystals were harvested in mother liquor supplemented with 25% MPD prior to flash-freezing in liquid nitrogen. Reflection data were collected at NECAT 24-ID-C at the Advanced Photon Source. The crystals belong to space group P3112 with two KLHDC2-Elo/B complexes in the asymmetric unit. Phases were obtained by molecular replacement using PHASER 60 with (1) two copies of KLHDC2 (with peptide omitted) from 6DO2.pdb (2) Two copies of EloB/C from 1VCB.pdb. Manual building was performed in COOT and refinement was performed using Phenix. 61 Additional details of the refinement are provided in Table 1. The final Ramachandran statistics are as follows: 99% Favored, 0%.

## Biochemical assays

The use of pulse-chase assays allowed comparing the paths of UB transfer starting from either UBE2D2 or UBE2R2. First, the indicated E2 was pulse-labeled by incubating a mixture of UBA1 (0.3 m M), E2 (10 m M), and *UB (15 m M) in 25 mM HEPES, 100 mM NaCl, 100mM MgCl2, ATP (2 mM), pH 7.5 at room temperature for 13 minutes. Pulse-loading reactions were quenched by the addition of EDTAto50mMandincubatedonicefor5minutes.Chasereactions consisted of mixing the E2 /C24 *UBthioester conjugate (0.3 m Mfinal concentration) with the indicated pre-equilibrated NEDD8 /C24 CUL2-RBX1-SR complexes (0.3 m M final concentration) in 25 mM HEPES, 100 mM NaCl, 50 mM EDTA, 0.5 mg/ml BSA, pH 7.5 at room temperature. Where indicated test substrates were pre-incubated (0.6 m Mfinal concentration) with NEDD8 /C24 CUL2-RBX1-SR complexes for the indicated times prior to initiating reactions by the addition of E2 /C24 *UB. Reactions were quenched at the indicated times with 2X SDS-PAGE sample buffer and separated on 4-12% BisTris gradient gels, scanned for fluorescence on a Typhoon imager.

Neddylation of CUL2-RBX1 was also monitored by pulse-chase assays. First, UBE2M was pulse-labeled by incubating a mixture of APPBP1/UBA3(0.3 m M), UBE2M(10 m M), and *NEDD8 (15 m M)in 25 mM HEPES, 100 mM NaCl, 100mM MgCl2, ATP (2 mM), pH 7.5 at room temperature for 13 minutes. Pulse-loading reactions were quenched by the addition of EDTA to 50 mM and incubated on ice for 5 minutes. Chase reactions consisted of mixing the UBE2M /C24 *UB thioester conjugate (75 nM final concentration) with CUL2-RBX1 (100 nM final concentration) pre-equilibrated with the indicated fold excess of SR complexes with or without the indicated fold excess of diGly substrate in 25 mM HEPES, 100 mM NaCl, 50 mM EDTA, 0.5 mg/ml BSA, pH 7.5 at room temperature. Reactions monitoring the displacement of CAND1 were performed as described above, except CUL2-RBX1 (100 nM final concentration) was pre-equilibrated with equimolar CAND1, prior to initiating neddylation reactions by the simultaneous addition of UBE2M /C24 *NEDD8andtheindicated fold excess of SR with or without the indicated fold excess diGly substrate. Reactions were quenched at the indicated times with 2X SDS-PAGE sample buffer and separated on 4-12% Bis-Tris gradient gels, scanned for fluorescence on a Typhoon imager

## Time-course reactions

Reactions were pre-assembled into two separate mixtures with proteins diluted into reaction buffer containing 30 mM Tris-HCl (pH 7.5), 100 mM NaCl, 5 mM MgCl2, 2 mM DTT and 2 mM ATP. CRL components and 32 P-labeled substrate protein were sequentially added (tube 1) and incubated for 5, 10, 20, 40 and 60 minutes. Meanwhile, E1, ubiquitin and UBE2R2 proteins were sequentially added and incubated for a period of approximately 15 minutes (tubes 2). Ubiquitylation reactions were initiated by mixing equal volumes of tube 2 mix with the tube 1 incubations and time points were collected at 10-second and 5-minute intervals. Reactions were quenched in reducing 2X SDS-PAGE loading buffer (100 mM Tris-HCl (pH 6.8), 20% glycerol, 30 mM EDTA, 4% SDS and 4% beta-mercaptoethanol). Each reaction was performed in triplicate and was resolved on 18% polyacrylamide SDS-PAGE gels. Autoradiography was performed using an Amersham Typhoon 5 imager and quantification performed with ImageQuant software (Cytiva). The fraction of substrate ubiquitylated was calculated as the fraction of products over the total signal of each respective lane.

## Pre-steady-state quench flow kinetics

Reactions were pre-assembled into two separate mixtures with proteins diluted into reaction buffer as described in the previous section. CRL components and 32 P-labeled substrate protein were sequentially added (tube 1) and incubated for an hour. Afterwards, E1, ubiquitin and UBE2R2 proteins were sequentially added and incubated for a further 15 minutes (tubes 2). Each mix was loaded into separate sample loops on a KinTek RQF-3 Quench-Flow instrument. Reactions were initiated by bringing the two mixes together in drive buffer (30 mM Tris-HCl (pH 7.5) and 100 mM NaCl), and then quenched at various time points in reducing 2X SDS-PAGE loading buffer. Each reaction was performed in triplicate, and time points were resolved on 18% polyacrylamide SDS-PAGE gels and processed as described above. The rates of ubiquitin transfer were determined by fitting to analytical closed-form solutions (Pierce et al., 2009) using Mathematica.

## Analytical size exclusion chromatography

The distribution of KLHDC2-EloB/C species in the absence or presence of substate was monitored through migration over size exclusion chromatography. KLHDC2-EloB/C (6 m Mfinal) was mixed with MBP-SelK (12 m Mfinal) in 25 mM HEPES, 150 mM NaCl, 1 mM DTT, pH = 7.5 and incubated for the indicated times on ice. Mixtures were spun in a microfuge for 5 minutes and immediately applied to a Superdex SD200 column equilibrated in the same buffer. Fraction aliquots were removed and mixed with an equal volume of 2X SDS sample buffer and separated through 4-12% Bis-Tris gels. Gels were stained with Sypro Ruby according to the manufactures instructions and scanned on a Typhoon imager.

## Native gel electrophoresis

Native gel electrophoresis was used monitor the migration of KLHDC2-EloB/C species in the absence or presence of substate with or without CUL2 or N8 /C24 CUL2. Briefly, 2.5 m MKLHDC2, in the absence or presence of 2.5 m MTAMRAlabeled CUL2 or N8 /C24 CUL2, was mixed with the indicated concentrations of Fluorescein labeled SELK or USP1 in 25 mM HEPES, 150 mM NaCl, 5 % glycerol, 1 mM DTT, pH =7.5. Mixtures were allowed to equilibrate on ide for the indicated times and immediately loaded onto a 4-20% Tris-Glycine gel. Samples were separated at 130V for 180 minutes at 4 C. We note that, as we have no way of quenching binding reactions, continued binding and dissociation between protein complexes can occur during the 180 minute time period in which the gel is running.

Sedimentation velocity analytical ultracentrifugation (SV-AUC) experiments were conducted in a ProteomeLab XL-I analytical ultracentrifuge (Beckman Coulter, Indianapolis, IN) with the fluorescence detective system (AVIV Biomedical) following standard protocols unless mentioned otherwise. 62,63 All samples were prepared in a buffer containing 25 mM HEPES pH 7.5, 200 mM NaCl, 1 mM DTT at 20 C. AF488-DC2 and AF488-DC3 samples were loaded into cell assemblies comprised of double sector charcoal-filled centerpieces with a 12 mm path length and quartz windows.

Buffer density and viscosity were determined in a DMA 5000 M density meter and an AMVn automated micro-viscometer (both Anton Paar, Inc., Ashland, VA) respectively. A dilution series of 5,000, 800, 170, 30, 5 &amp; 0.5 nM were examined. In order to avoid surface adsorption of AF488-DC2 and AF488-DC3 0.1 mg/mL bovine serum albumin (BSA) was added in all protein samples as a carrier protein. The cell assemblies, containing sample volumes of approximately 390 m L, were placed in a rotor. For the FDS-SV experiments radial calibration and the set-up of focusing depth and PMT voltage were performed at 3,000 rpm. The focal depth was set at 4.696 mm, and, in order to obtain sufficient fluorescent signal for the samples with different concentrations of the fluorescent labelled protein a range of PMT settings were used (40% and gain 8 for 5 and 0.5 nM and 40 % and gain 1 for 5,000 nM). Then temperature equilibrium was carried out at rest at 20 /C14 C for 2 hours followed by acceleration from 0 to 45,000 rpm. Fluorescence scans were collected continuously for 12 hours. The velocity data were modeled with diffusion-deconvoluted sedimentation coefficient distributions c(s) in SEDFIT (https://sedfitsedphat.nibib.nih.gov/software), using algebraic noise decomposition and with signal-average frictional ratio and meniscus position refined with non-linear regression. 64 The s-values were corrected for time and finite acceleration of the rotor was accounted for in the evaluation of Lamm equation solutions. Maximum entropy regularization was applied at a confidence level of P-0.68.

The FDS sedimentation velocity data were subjected to standard c ( s ) analysis in SEDFIT as described above. 64 Isotherm data were obtained by the integration between 3 and 12 S of these c(s) distributions at concentrations of 800, 170, 30, 5 and 0.5 nM to generate signal weight-average sedimentation coefficient values, sw (or expressed as sw(c) ) for the isotherm analysis. Again the 2 S peak was excluded from the analysis because the mass was smaller than the monomer mass of AF488-DC2/DC3 and thus not part of the selfassociation process. It is possibly a breakdown product or contaminant protein. The resulting sw isotherm was then loaded into SEDPHAT(https://sedfitsedphat.nibib.nih.gov/software) and fitted with the monomer-dimer-tetramer self -association model. 62 The peaks centered at /C24 4 and /C24 10 S are the monomer and tetramer species, respectively. The association scheme used in this analysis was A + A 4 [A2] + [A2] 4 [A4] with A the AF488-DC2 and AF488-DC3 monomer (MW = 72,990 and 69,977 Da) and K12 the equilibrium association constant of the first step and K24 the second step. 62 Since K = 1/ Kd these association constants are expressed as dissociation constants Kd12, Kd14 and Kd24 and were determined by nonlinear regression analysis of the sw isotherms. Kd12 and Kd14 as well as s1, s2 and s4 were optimized in the fit, and the error intervals of the best-fit Kd14 values were determined using error surface projection at a 68.3 % confidence level. 65

## Surface plasmon resonance

Surface plasmon resonance (SPR) was utilized to determine kinetic parameters and binding affinity of KLHDC2 substrates using a PioneerFE (Sartorius). Experiments were carried out in 20 mM Tris, 200 mM NaCl, 1 mM TCEP, 0.01% Brij35, 1% DMSO, at pH

7.4 as a running buffer with a flow rate of 100 m L/min at 20 /C14 C. WT Nedd8 was tested at a slower flow rate (50 m L/min) due to slow association kinetics. 10xHis-KLHDC2 was diluted in running buffer and immobilized onto a Ni-NTA-coated biosensor chip (HisCap). The chip surface was conditioned using 10 mM HCl, 50 mM NaOH, and 0.1% SDS, (50 m L at 100 m L/min) twice followed by a 100 mM Tris buffer wash (100 m L at 100 m L/min). The surface was then activated using an injection of 0.5M EDTA (125 m L at 25 m L/ min) followed by an injection of 5 mM NiCl2 (125 m L at 25 m L/min). Protein was then immobilized onto channel 3 around 300 RU for most analytes, around 1500 RU for SelK and WT Nedd8, and around 2200 RU for the weak-binding substrates. Channel 2 was treated identically to channel 3 with the exception of protein loading to be used as a reference channel. Protein analytes were diluted to their designated concentrations in the running buffer and injected using the OneStep /C210 injection method on the Pioneer system which utilizes Taylor dispersion to obtain binding kinetics data from a single injection. 100% of the loop was injected with a 300 s dissociation time. All samples were run in triplicate. Data was analyzed using Qdat software and fit to a simple 1-site model.

## Cryo-EM sample preparation and microscopy

The protein complex of CUL2-RBX1-EloB-EloC-KLHDC2 was prepared at 1.7mg/ml in 25mM HEPES, 150mM NaCl, 1mM DTT, pH 7.5. Sample was vitrified by applying 3.5 m L of the protein complex onto Quantifoil R1.2/1.3 holey carbon grids and immediately plunge frozen into liquid ethane with a Vitrobot Mark IV.

Dataset was collected on a 200kV Glacios equipped with a K2 Summit direct detector. 2609 movies were gained on counting mode at a nominal magnification of 22kx with a 1.885A Ëš /pixel. 60 e-/A Ëš 2 were fractionated over 40 frames, using a defocal range of -1.5 m m /C24 -3.0 m m.

## Cryo-EM data processing

Collected movies were motion-corrected using RELION-3.1, 67 andCTFestimated by CTFFIND-4.1 68 and particles were picked using Gautomatch-0.56 (K. Zhang, MRC Laboratory of Molecular Biology, Cambridge). Picked particles were subject to several rounds of 2D classification, followed by 3D classification and 3D refinement. While the 3D reconstruction covered all angular distributions, it contained a significant number of particles with one particular view. The dominant view was downsized by removing views with iterative 2D classification without performing alignment and 3D refinement. Final reconstruction yielded a 8.2A Ëš map with a more homogeneous reconstruction. Processing schematic is shown in Figure S4.

SEC-MALS experiments were carried out using WTC-030S5 (MW range 5000-1, 250,000 Da) size-exclusion column (Wyatt Technologies, Santa Barbara, CA, USA) with three detectors connected in series: an Agilent 1200 ultraviolet (UV) detector (Agilent Technologies, Santa Clara, CA), a Wyatt DAWN-HELEOS-II multi-angle light-scattering (MALS) and a Wyatt Optilab T-rEX differential refractive index (RI) detector (Wyatt Technologies, Santa Barbara, CA, USA). The column was equilibrated with 25 mM HEPES (pH 7.5), 200 mM NaCl, and 1 mM TCEP. All the data were collected at 25 /C14 C. A 100 m L sample in volume was injected into the column using an auto-sample injection method with a flow rate of 0.5 ml/min. Protein in the eluent was detected via UV absorbance at 280 nm, light scattering, and refractive index detectors. The data were recorded and analysed with the Wyatt Astra software (version 6.1). The refractive index increment, dn/dc, was assumed to be 0.185 ml/g for measuring the concentration of the protein samples. EASI Graphs (Astra software) were exported and plotted as a molar mass distribution superimposed on a chromatogram of absorbance at 280 nm versus elution volume.

## QUANTIFICATION AND STATISTICAL ANALYSIS

## Statistical analysis

For CUL2-RBX1 neddylation kinetics N=3 and the values reported in the figures are the mean Â± one SD as determined by fitting in Prisim.

## Data and software availability

The following PDB files were used to generate structural models as described in the figures utilizing the molecular visualization software PyMOL (https://www.pymol.org/). PDB:4KBL, PDB:4LJO, PDB:5EDV, PDB:4UN2, and PDB:4P5O